

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Chapter 3: Microbial Growth in the Lab and in Nature &#8212; MICB 211 Foundations in Microbiology eTextbook</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'unit1-chap3';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="&lt;no title&gt;" href="unit1-chap4.html" />
    <link rel="prev" title="Chapter 2: Microbial Cell Structures" href="unit1-chap2.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="intro.html">
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="intro.html">
                    MICB 211 E-Textbook Acknowledgements
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 1</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit1-chap1.html">Chapter 1: Introduction to Microbiology and Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap2.html">Chapter 2: Microbial Cell Structures</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Chapter 3: Microbial Growth in the Lab and in Nature</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 2</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit2-chap1.html">Unit 2: Chapter 1</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Funit1-chap3.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/unit1-chap3.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Chapter 3: Microbial Growth in the Lab and in Nature</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prokaryotic-cell-division">3.1 Prokaryotic Cell Division</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#individual-cell-growth-binary-fission">3.1.1 Individual Cell Growth: Binary Fission</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#batch-cell-growth-growth-curves">3.1.2 Batch cell growth: Growth curves</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microbes-growing-in-the-lab">3.2 Microbes Growing in the Lab</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#colony-forming-units">3.2.1 Colony Forming Units</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#optical-density">3.2.2 Optical density</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biomass-weight">3.2.3 Biomass weight</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#direct-microscopic-count">3.2.4 Direct microscopic count</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-stevia-effect-on-growth">Case Study: Stevia effect on growth</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#growth-rate">3.3 Growth rate</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microbes-growing-in-the-wild">3.4 Microbes Growing in the Wild</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#temperature">3.4.1 Temperature</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#solute-concentration">3.4.2 Solute concentration</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#ph-h-concentration">3.4.3 pH (H+ concentration)</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#oxygen-concentration">3.4.4 Oxygen concentration</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#nutritional-status">3.4.5 Nutritional Status</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-assays">3.5 Protein Assays</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#sds-page">3.5. 1 SDS-PAGE</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#western-blot">3.5.2 Western Blot</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-salt-and-starvation-dont-mix">Case Study: Salt and starvation don’t mix</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <div class="tex2jax_ignore mathjax_ignore section" id="chapter-3-microbial-growth-in-the-lab-and-in-nature">
<h1>Chapter 3: Microbial Growth in the Lab and in Nature<a class="headerlink" href="#chapter-3-microbial-growth-in-the-lab-and-in-nature" title="Permalink to this heading">#</a></h1>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Learning Objectives</strong></p>
<p class="sd-card-text">By the end of this chapter, students will be expected to:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">define “growth” in the context of Bacteria</p></li>
<li><p class="sd-card-text">describe the different phases of a growth curve</p></li>
<li><p class="sd-card-text">distinguish between growth in single and multiple substrate cultures</p></li>
<li><p class="sd-card-text">estimate growth rate and calculate growth rate variables</p></li>
<li><p class="sd-card-text">interpret data collected using different methods of measuring microbial growth in the lab</p></li>
<li><p class="sd-card-text">correlate the effects of different environmental pressures on a microorganism and strategies to overcome them</p></li>
<li><p class="sd-card-text">interpret and analyze data on microbial growth under different environmental pressures</p></li>
<li><p class="sd-card-text">interpret data collected using protein assays</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="prokaryotic-cell-division">
<h2>3.1 Prokaryotic Cell Division<a class="headerlink" href="#prokaryotic-cell-division" title="Permalink to this heading">#</a></h2>
<p>Unicellular prokaryotes do not grow in the sense that they continually “get bigger”. Granted these organisms do get bigger as part of their reproductive cycle, but the end result of this cycle is to produce two organisms similar in size and mass to the starting organism. Thus, when microbiologists say that unicellular prokaryotes “grow”, what they usually mean is a population of organisms increasing in total number of cells and mass, i.e., growth rate = population growth rate.</p>
<div class="section" id="individual-cell-growth-binary-fission">
<h3>3.1.1 Individual Cell Growth: Binary Fission<a class="headerlink" href="#individual-cell-growth-binary-fission" title="Permalink to this heading">#</a></h3>
<p>Prokaryotic cells divide through a process known as <strong>binary fission</strong>. Before dividing, the cell replicates its DNA and segregates it at the polar ends of as it elongates. A ring-like structure called the FtsZ ring forms in the middle of the two cells, “pinching” the cells apart (Fig 33).</p>
<div class="figure align-default" id="fig33">
<a class="reference internal image-reference" href="_images/3-1.png"><img alt="_images/3-1.png" src="_images/3-1.png" style="height: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 33 </span><span class="caption-text">Binary fission.</span><a class="headerlink" href="#fig33" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="batch-cell-growth-growth-curves">
<h3>3.1.2 Batch cell growth: Growth curves<a class="headerlink" href="#batch-cell-growth-growth-curves" title="Permalink to this heading">#</a></h3>
<p>As individual cells divide, they contribute to the expansion of the overall culture population or <strong>batch growth</strong>. Prokaryotic batch cultures undergo 4 phases (Fig 34):</p>
<div class="figure align-default" id="fig34">
<a class="reference internal image-reference" href="_images/3-2.png"><img alt="_images/3-2.png" src="_images/3-2.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 34 </span><span class="caption-text">Growth curve of a batch culture.</span><a class="headerlink" href="#fig34" title="Permalink to this image">#</a></p>
</div>
<ol class="arabic simple">
<li><p>The initial lag phase does not involve growth/division or death. At this point where the cells have just been inoculated (introduced into the culture), there is a period of acclimatization to their new</p></li>
<li><p>The exponential phase involves a consistent, exponential growth of the population over time as nutrients are readily abundant and sufficient to support growth. At this phase, you can estimate a species’ doubling time. The doubling time is the time it takes for the population to double. Every species has a characteristic doubling time depending on the types of nutrients it is growing in.</p></li>
<li><p>Once nutrients have been exhausted, the population reaches a stationary phase where the number of cells dividing is equal to the number of cells dying.</p></li>
<li><p>Waste begins to accumulate and the number of cells dying off outnumber the number of cells growing and dividing resulting in the death phase.</p></li>
</ol>
<p>Just to make it interesting. Imagine that a bacterial cell was growing in two substrates (ie. two nutrients). Interestingly, it would not consume both at the same time, but more likely, it would actually consume the preferred substrate first. When this happens, we observe <strong>diauxic growth</strong> or we say it results in <strong>diauxia</strong> (Fig 35). A diauxic growth curve shows two growth cycles where the first cycle leading to the first stationary phase involves the metabolism of the preferred substrate. Once consumed, the bacteria start to metabolize the secondary substrate and undergoes a second exponential phase leading into a second stationary phase when that second substrate is also consumed.</p>
<div class="figure align-default" id="fig35">
<a class="reference internal image-reference" href="_images/3-3.png"><img alt="_images/3-3.png" src="_images/3-3.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 35 </span><span class="caption-text">Diauxic growth curve.</span><a class="headerlink" href="#fig35" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="microbes-growing-in-the-lab">
<h2>3.2 Microbes Growing in the Lab<a class="headerlink" href="#microbes-growing-in-the-lab" title="Permalink to this heading">#</a></h2>
<p><strong>Nutrient media</strong>
A nutrient, growth or culture medium is an aqueous (=water-based) mixture of nutrient chemicals that support cell division. Different prokaryotes use/consume different nutrients, so there are many different kinds of nutrient media. However, all media possess one common component: water. All prokaryotes are aquatic organisms, they may be able to survive without liquid water, but it is essential for cell division. Even when bacterial and archaeal cells seem to be dividing on a dry surface, they are really doing so in a thin film of water.</p>
<p><strong>Sterilizing nutrient media</strong>
Microorganisms are ubiquitous (= present everywhere), even in the chemicals used to make-up nutrient media. Thus, before a nutrient medium can be used to obtain a <strong>pure culture</strong>, it must first be rendered free of living microorganisms (= sterilized) before use. One way to accomplish this is through the use of steam under pressure by placing the liquid medium a pressure cooker (the scientific version is called an <strong>autoclave</strong>). Historically, the practical conditions for steam sterilization have been a temperature of 121C and a pressure of approximately 15 psi (over normal atmospheric pressure).</p>
<p>In this section, we will cover 4 different ways of measuring microbial growth in the lab using sterilized nutrient media:</p>
<ul class="simple">
<li><p>Colony forming units</p></li>
<li><p>Optical density</p></li>
<li><p>Biomass</p></li>
<li><p>Direct microscopic count</p></li>
</ul>
<div class="section" id="colony-forming-units">
<h3>3.2.1 Colony Forming Units<a class="headerlink" href="#colony-forming-units" title="Permalink to this heading">#</a></h3>
<p>One of the more classical methods for growing a prokaryotic culture involves spreading organisms on the surface of a nutrient medium that has been solidified with <strong>agar</strong>, a polysaccharide isolated from marine algae. The preparation of solid nutrient medium using agar is similar to making Jello^TM. First agar powder is dissolved in hot liquid medium to form molten agar or molten medium which is then poured into petri plates (Fig 36). Upon cooling of the molten medium to room temperature, a solid, clear gel is formed. Agar has useful properties that have not been easily duplicated by other materials:</p>
<ul class="simple">
<li><p>Agar is not used as a nutrient by most prokaryotes.</p></li>
<li><p>Agar gels are transparent.</p></li>
<li><p>Only a low concentration agar (1.5 g/100 mL) is required for solidification purposes. Molten agar is not viscous, so it is easy to pour but gives a stiff gel when solidified.</p></li>
<li><p>Agar gels can be used in culturing over a wide temperature range ~1C - 60C (agar gels liquefy 85C).</p></li>
</ul>
<div class="figure align-default" id="fig36">
<a class="reference internal image-reference" href="_images/3-4.png"><img alt="_images/3-4.png" src="_images/3-4.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 36 </span><span class="caption-text">Preparing agar nutrient media.</span><a class="headerlink" href="#fig36" title="Permalink to this image">#</a></p>
</div>
<p>Once the plates have been made, bacteria can be <strong>inoculated</strong> (transferred) onto the surface using several different methods not covered in this course. When it comes to quantifying bacterial load, the goal is to inoculate the plate in a way that separates the individual prokaryotic cells enough so that they grow to be distinguishable colonies (Fig 37). A colony is a clonal clump of cells that derived from the single cell that was inoculated on the surface. Each colony represents a <strong>colony forming unit (CFU)</strong> and are visible to the naked eye allowing a researcher to count the CFUs as a measure of how many cells were initially inoculated on the plate. The higher the CFU, the greater the initial bacterial load that was inoculated.</p>
<div class="figure align-default" id="fig37">
<a class="reference internal image-reference" href="_images/3-5.png"><img alt="_images/3-5.png" src="_images/3-5.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 37 </span><span class="caption-text">Inoculating an agar plate to count colonies.</span><a class="headerlink" href="#fig37" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="optical-density">
<h3>3.2.2 Optical density<a class="headerlink" href="#optical-density" title="Permalink to this heading">#</a></h3>
<p>In contrast to CFUs which are measured based on cells growing on a plate (i.e. solid surface), <strong>optical density (OD)</strong> is used to measure cell density of prokaryotes grown in liquid cultures. When prokaryotes grow in liquid medium, the medium itself which starts off clear becomes turbid (Fig 38). This turbidity can be used to measure cell density.</p>
<div class="figure align-default" id="fig38">
<a class="reference internal image-reference" href="_images/3-6.png"><img alt="_images/3-6.png" src="_images/3-6.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 38 </span><span class="caption-text">Clear media (left) and bacteria-inoculated media (right).</span><a class="headerlink" href="#fig38" title="Permalink to this image">#</a></p>
</div>
<p>A <strong>spectrophotometer</strong> is a large instrument which allows us to measure the amount of light that is scattered by a sample. It starts by passing a light through the sample. Clear media with no turbidity would allow light to easily pass through, therefore, very little light is being scattered away (Fig 39). The OD reading would be very low or 0. When media is turbid due to high bacterial load, the bacterial cells would cause light passing through to scatter away and very little light would reach the sensor (Fig 39). The amount of scattered light would result in an OD reading above 0. The higher the OD reading indicates that less light has passed through and more was scattered away, also indicating higher cell density.</p>
<div class="figure align-default" id="fig39">
<img alt="_images/3-7.png" src="_images/3-7.png" />
<p class="caption"><span class="caption-number">Fig. 39 </span><span class="caption-text">OD through clear and turbid media.</span><a class="headerlink" href="#fig39" title="Permalink to this image">#</a></p>
</div>
<div class="margin sidebar">
<p class="sidebar-title"></p>
<p><strong>OD600</strong>
When you see OD data for bacterial cultures, you will often see it represented as OD600. The “600” refers to 600nm which is the wavelength of light used to measure cell density. This wavelength has been determined as the most optimal in getting a more accurate measure of cell density. How it works is beyond the scope of this course.</p>
</div>
</div>
<div class="section" id="biomass-weight">
<h3>3.2.3 Biomass weight<a class="headerlink" href="#biomass-weight" title="Permalink to this heading">#</a></h3>
<p>Another way that you can measure batch growth is by determining the weight of biomass. This can be done by filtering a culture through a filter disc and weighing it to compare it to an empty filter disc (Fig 40).</p>
<div class="figure align-default" id="fig40">
<a class="reference internal image-reference" href="_images/3-8.png"><img alt="_images/3-8.png" src="_images/3-8.png" style="height: 200px;" /></a>
<p class="caption"><span class="caption-number">Fig. 40 </span><span class="caption-text">Biomass measure (left) and direct microscopic count (right).</span><a class="headerlink" href="#fig40" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="direct-microscopic-count">
<h3>3.2.4 Direct microscopic count<a class="headerlink" href="#direct-microscopic-count" title="Permalink to this heading">#</a></h3>
<p>Direct microscopic count involves looking a subset of your sample under the microscopy and literally counting the absolute number of cells then using that number to estimate the population size in your original culture flask (Fig 40).</p>
<p></p>
</div>
</div>
<div class="section" id="case-study-stevia-effect-on-growth">
<h2>Case Study: Stevia effect on growth<a class="headerlink" href="#case-study-stevia-effect-on-growth" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Soroski et al., 2020</strong></p>
<p class="sd-card-text">Soroski T, Tsankov BK, Hossain AR, and Lee D. 2020. Commercial stevia does not inhibit growth or biofilm formation of Escherichia coli strain MG1655. UJEMI 6:1-8</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p>This study was also conducted by 4th year UBC students in MICB 421! In this study, the students wanted to know if Stevia, a type of artificial sweetener, had any potential effects on the growth of <em>E. coli</em>. They hypothesized that, being a type of artificial sugar, it would be metabolized like natural sugar and therefore promote growth of <em>E. coli</em>.</p>
<p>In order to test this hypothesis, Soroski et al. collected the following data in the form of growth curves:</p>
<div class="figure align-default" id="fig41">
<a class="reference internal image-reference" href="_images/3-9.png"><img alt="_images/3-9.png" src="_images/3-9.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 41 </span><span class="caption-text">Growth curve comparisons of <em>E. coli</em> MG1655 grown in M9 minimal medium with 0% (control), 1%, 10%, 20%, and 40% (v/v) concentrations of stevia was performed using three biological replicates. Error bars represent standard deviation.</span><a class="headerlink" href="#fig41" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">Questions to consider:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">What technique was used to collect the data?
a.	What is the advantage of the technique used to collect this data?
b.	What are alternative ways to measure this?</p></li>
<li><p class="sd-card-text">What are the treatment(s) and what is the control group?</p></li>
<li><p class="sd-card-text">What is your initial impression of the data?</p></li>
<li><p class="sd-card-text">Which phases of the growth curve can you resolve? Which conditions have reached stationary phase?</p></li>
<li><p class="sd-card-text">Which condition has the fastest growing culture?</p></li>
<li><p class="sd-card-text">Which condition has the lowest growing culture?</p></li>
<li><p class="sd-card-text">Which culture has reached the highest cell density after 16 hours?</p></li>
<li><p class="sd-card-text">What can you conclude about high and low levels of stevia on <em>E. coli</em> growth?</p></li>
<li><p class="sd-card-text">Do you agree with the authors’ original hypothesis?</p></li>
</ol>
</div>
</div>
</div>
</div>
<div class="section" id="growth-rate">
<h2>3.3 Growth rate<a class="headerlink" href="#growth-rate" title="Permalink to this heading">#</a></h2>
<p>Growth rate is usually estimated in laboratory cultures. Such an approach allows microbiologists to remove prokaryotes from the myriad of influences they are normally subjected to in their complex environmental milieu which in turn allows the factors affecting their growth to be analyzed one-by-one in a “controlled” manner (this can only be done for those prokaryotes that can be cultured in the laboratory).</p>
<p><strong>Measuring Growth Rate</strong></p>
<p>We can estimate the population size using the following formula:</p>
<div class="math notranslate nohighlight">
\[N_t=N_o × 2^{(t/n)}\]</div>
<p>Nt = Final population size
No = Initial population size or number of cells inoculated
t = time of incubation
n = doubling time</p>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>Example: <em>E. coli</em> is known to have a doubling time of approximately 17 minutes. If 100 cells were originally added to the medium and grown for 2 hours (120 minutes), the final population size would be approximately 13,332 cells.</p>
<p>Nt = 100 x 2^(120/17) = 13,332</p>
</div>
</div>
<div class="section" id="microbes-growing-in-the-wild">
<h2>3.4 Microbes Growing in the Wild<a class="headerlink" href="#microbes-growing-in-the-wild" title="Permalink to this heading">#</a></h2>
<p>Prokaryotes exhibit enormous diversity with respect to the environmental conditions under which they thrive. Further, many types of prokaryotes live under environmental regimes considered quite harsh from the standpoint of human biology. At the extremes of livability on the planet, one finds Archaea and Bacteria.</p>
<p>The growth and survival of all organisms is controlled by a variety of physical and chemical environmental factors. Most organisms are adapted to grow optimally under a particular set of physiochemical conditions and subjecting them to conditions outside their normal range is considered a stress which may result in death. Different organisms vary in their ability to tolerate environmental stresses. In this section, we want you to focus on how each of these variables can act as a “stress” factor and how bacteria have adapted to cope with these stressors.</p>
<p>Using the human-centred, chauvinistic criteria, microbiologists have grouped prokaryotes in relation to environmental variables in an attempt to “organize” the prokaryotic world for study. Consider the examples below. While the language used to name these groups might seem obscure and confusing, a closer examination reveals that most terms involve just a few suffixes and adjectives:</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Group</p></th>
<th class="head"><p>Definition</p></th>
<th class="head"><p>Suffix/adjective</p></th>
<th class="head"><p>Definition</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Thermophiles</p></td>
<td><p>Loves high temperatures</p></td>
<td><p>-phile</p></td>
<td><p>lover</p></td>
</tr>
<tr class="row-odd"><td><p>Phototrophs</p></td>
<td><p>Feed on (consume) light</p></td>
<td><p>-troph</p></td>
<td><p>feeder</p></td>
</tr>
<tr class="row-even"><td><p>Obligate aerobe</p></td>
<td><p>Requires oxygen</p></td>
<td><p>obligate</p></td>
<td><p>restricted</p></td>
</tr>
<tr class="row-odd"><td><p>Facultative anaerobe</p></td>
<td><p>Can live with or without oxygen</p></td>
<td><p>facultative</p></td>
<td><p>flexible</p></td>
</tr>
</tbody>
</table>
<div class="section" id="temperature">
<h3>3.4.1 Temperature<a class="headerlink" href="#temperature" title="Permalink to this heading">#</a></h3>
<p>Temperature has numerous effects on microbial cells including:</p>
<ul class="simple">
<li><p>the rate of enzyme-catalyzed reactions.</p></li>
<li><p>protein 3D structure (conformation) required for function (Fig 42)</p></li>
<li><p>the secondary and tertiary structure of nucleic acids</p></li>
<li><p>the fluidity of membranes.</p></li>
</ul>
<div class="figure align-default" id="fig42">
<img alt="_images/3-10.png" src="_images/3-10.png" />
<p class="caption"><span class="caption-number">Fig. 42 </span><span class="caption-text">Protein denaturation under extreme temperatures.</span><a class="headerlink" href="#fig42" title="Permalink to this image">#</a></p>
</div>
<p>Because temperature regulates so many critical aspects of the cell, changing temperatures can significantly affect cell physiology. Despite that, we do observe microorganisms living in rather extreme temperature environments as shown in Fig 43.</p>
<div class="figure align-default" id="fig43">
<a class="reference internal image-reference" href="_images/3-11.png"><img alt="_images/3-11.png" src="_images/3-11.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 43 </span><span class="caption-text">Prokaryotic diversity in growth at different temperatures.</span><a class="headerlink" href="#fig43" title="Permalink to this image">#</a></p>
</div>
<p><strong>Psychrophiles</strong></p>
<p>At low temperatures, molecular motion is dampened. In order to remain flexible at low temperatures, psychrophiles make proteins with weaker intramolecular interactions and less hydrophobic cores (leading to a weakened hydrophobic effect). A consequence of this built-in flexibility is weak stability, and it for this reason the proteins of psychrophiles heat denature at lower temperatures than the proteins of mesophiles.</p>
<p><strong>Thermophiles</strong></p>
<p>At high temperatures molecular motion is intensified. In order to function properly at high temperature, thermophiles make proteins which are less flexible than proteins made by mesophiles. While it is known that the proteins of thermophiles require high temperature for optimal function and heat denature at much higher temperatures than those of mesophiles, after decades of study it is still not well understood what molecular features of thermophilic proteins are responsible for these properties.</p>
</div>
<div class="section" id="solute-concentration">
<h3>3.4.2 Solute concentration<a class="headerlink" href="#solute-concentration" title="Permalink to this heading">#</a></h3>
<p>High solute concentrations in the environment usually result from the presence of salt, NaCl. Accordingly, prokaryotes are grouped according to the effects of NaCl concentration on their growth rate (Fig 3.12). Different prokaryotes are adapted to grow optimally in environments with different NaCl concentrations. The fact that some organisms require high NaCl concentrations is partly due to the structure of their particular proteins and lipids. Moderate and extreme halophiles specifically require NaCl because their membranes require high concentrations of Na+ for stability and their proteins require high concentrations of Na+ for proper function.</p>
<div class="figure align-default" id="fig44">
<a class="reference internal image-reference" href="_images/3-12.png"><img alt="_images/3-12.png" src="_images/3-12.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 44 </span><span class="caption-text">Prokaryotic diversity in growth at different salt concentrations.</span><a class="headerlink" href="#fig44" title="Permalink to this image">#</a></p>
</div>
<p>When the environmental solute concentration is higher than that of the cytoplasm (i.e., the environment is <strong>hypertonic</strong>), the net direction of water movement is out of cells. Under hypertonic conditions, the volume of the cytoplasm decreases and the membrane collapses. This is called <strong>plasmolysis</strong> (Fig 45). Depending on its severity, plasmolysis may or may not lead to death of an organism. Unlike protection from osmotic lysis, a rigid wall component cannot protect a cell from plasmolysis. One way to prevent plasmolysis is to make the cytoplasm at least <strong>isotonic</strong> with environment and some halophiles and halotolerant organisms do exhibit this adaptation to high salt environments.</p>
<div class="figure align-default" id="fig45">
<a class="reference internal image-reference" href="_images/3-13.png"><img alt="_images/3-13.png" src="_images/3-13.png" style="height: 200px;" /></a>
<p class="caption"><span class="caption-number">Fig. 45 </span><span class="caption-text">Prokaryotic diversity in growth at different salt concentrations.</span><a class="headerlink" href="#fig45" title="Permalink to this image">#</a></p>
</div>
<p><strong>Halophiles</strong></p>
<p>Two fundamentally different strategies exist within the microbial world that enable microorganisms to cope with the osmotic stress associated with high salt concentrations (Fig 46):</p>
<ul class="simple">
<li><p>The <strong>“salt-in” strategy</strong>: This strategy involves transporting salt into the cytoplasm from the environment in order to balance the external concentration. For this to work, all intracellular systems have to be adapted to the presence of a high salt concentration</p></li>
<li><p>The <strong>“compatible solute” strategy</strong>: This strategy involves synthesizing organic solutes which accumulate to high concentrations in the cytoplasm in order to balance the external salt concentration. These organic solutes are called compatible solutes because their accumulation to high concentrations in the cytoplasm is compatible with cell function; no special adaptation of intracellular systems is required. The compatible solute strategy is very energetically expensive because it requires the synthesis of large amounts of organic molecules particularly in very high salt environments.</p></li>
</ul>
<div class="figure align-default" id="fig46">
<a class="reference internal image-reference" href="_images/3-14.png"><img alt="_images/3-14.png" src="_images/3-14.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 46 </span><span class="caption-text">Strategies to cope with osmotic shock, red: “salt-in”; yellow: “compatible solute”.</span><a class="headerlink" href="#fig46" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="ph-h-concentration">
<h3>3.4.3 pH (H+ concentration)<a class="headerlink" href="#ph-h-concentration" title="Permalink to this heading">#</a></h3>
<p>The folding of proteins and the ability of membrane phospho- and glycolipids to form a bilayer structure is sensitive to pH. This is because pH affects whether certain chemical groups possessed by these molecules gain or lose H+ which in turn affects the charge carried by the molecule. Consider the amino acid glycine below (Fig 47).</p>
<div class="figure align-default" id="fig47">
<a class="reference internal image-reference" href="_images/3-15.png"><img alt="_images/3-15.png" src="_images/3-15.png" style="height: 250px;" /></a>
<p class="caption"><span class="caption-number">Fig. 47 </span><span class="caption-text">Glycine under different pH conditions.</span><a class="headerlink" href="#fig47" title="Permalink to this image">#</a></p>
</div>
<p>Accumulation of positive charges under acidic conditions or negative charges under alkaline conditions can lead to repulsive interactions disrupting protein folding and lipid bilayer structure.</p>
<p>Prokaryotes grow over a wide range of pH and they have been grouped according to the effect of environmental pH on their growth rate (Fig 48).</p>
<div class="figure align-default" id="fig48">
<a class="reference internal image-reference" href="_images/3-16.png"><img alt="_images/3-16.png" src="_images/3-16.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 48 </span><span class="caption-text">pH range tolerance of microbial organisms.</span><a class="headerlink" href="#fig48" title="Permalink to this image">#</a></p>
</div>
<p><strong>Coping Strategies</strong></p>
<p>Presumably one way to control cytoplasmic pH would be to modulate the permeability of the cytoplasmic membrane lipid bilayer to H+ and indeed, many studies have documented compositional and structural differences between the membrane phospholipids of organisms growing under highly acidic or alkaline conditions. This said, it is often not clear at the molecular level why these differences should affect lipid bilayer permeability to H+.</p>
<p><strong>H+ transport</strong></p>
<p>Another way prokaryotes control cytoplasmic pH is though membrane protein-based passive or active transport of H+ in or out of the cytoplasm as required.</p>
</div>
<div class="section" id="oxygen-concentration">
<h3>3.4.4 Oxygen concentration<a class="headerlink" href="#oxygen-concentration" title="Permalink to this heading">#</a></h3>
<p>Because the two outer orbital electrons of O2 are unpaired, O2 is a very reactive molecule. Indeed, when O2 enters cells, it reacts with a variety of cellular molecules becoming reduced to the hydroxyl radical (•OH) in a series of steps:</p>
<div class="figure align-default" id="fig49">
<img alt="_images/3-17.png" src="_images/3-17.png" />
<p class="caption"><span class="caption-number">Fig. 49 </span><span class="caption-text">Reactive oxygen species.</span><a class="headerlink" href="#fig49" title="Permalink to this image">#</a></p>
</div>
<p>The products of the reactions shown in Fig 49, superoxide anion, hydrogen peroxide, and hydroxyl radicals, can all be toxic to bacterial cells. However, we know that there are several bacterial species that can grow in the presence of oxygen and some relay heavily on oxygen as a key part of their metabolism. Microbial organisms have been classified based on (1) whether an organism requires O2 for growth and (2) whether it is killed by atmospheric O2 concentrations.</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Group</p></th>
<th class="head"><p>Requires O2 for growth?</p></th>
<th class="head"><p>Do not require O2 but can use it enhancing growth</p></th>
<th class="head"><p>Killed by atmospheric [O2] = 20%?</p></th>
<th class="head"><p>Enzymes</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Obligate aerobe</p></td>
<td><p>Yes</p></td>
<td><p></p></td>
<td><p>No</p></td>
<td><p>SOD, catalase</p></td>
</tr>
<tr class="row-odd"><td><p>Microaerophilic aerobe</p></td>
<td><p>Yes</p></td>
<td><p></p></td>
<td><p>Yes</p></td>
<td><p>Low levels of SOD or superoxide reductase; catalase</p></td>
</tr>
<tr class="row-even"><td><p>Facultative (an)aerobe</p></td>
<td><p>No</p></td>
<td><p>Yes</p></td>
<td><p>No</p></td>
<td><p>SOD, catalase</p></td>
</tr>
<tr class="row-odd"><td><p>Aerotolerant anaerobe</p></td>
<td><p>No</p></td>
<td><p>No</p></td>
<td><p>No</p></td>
<td><p>Mn+2 ions, peroxidase</p></td>
</tr>
<tr class="row-even"><td><p>Obligate anaerobe</p></td>
<td><p>No</p></td>
<td><p>No</p></td>
<td><p>Yes</p></td>
<td><p>Some: superoxide reductase</p></td>
</tr>
</tbody>
</table>
<p>The main coping strategy for dealing with reactive oxygen species is to synthesize enzymes capable of converting those species to a non-toxic form. Examples are listed in the table above. For example:</p>
<ul class="simple">
<li><p><strong>Obligate and facultative aerobes</strong> typically synthesize both superoxide dismutase and catalase allowing them to grow in 20% O2.</p></li>
<li><p><strong>Microaerophiles</strong> require O2 but are sensitive to 20% O2. This seems to be because they synthesize insufficient amounts of the enzymes necessary to protect them from atmospheric O2 concentrations.</p></li>
<li><p><strong>Aerotolerant anaerobes</strong> do not use or require O2 but can grow in the presence of 20% O2. Interestingly, these organisms lack superoxide dismutase and appear to somehow use the Mn+2 ions as an electron donor to reduce the superoxide anion. Such a system is not as efficient as superoxide dismutase and may have originated as the first way ancient anaerobic organisms dealt with the appearance of O2 in the atmosphere.</p></li>
</ul>
</div>
<div class="section" id="nutritional-status">
<h3>3.4.5 Nutritional Status<a class="headerlink" href="#nutritional-status" title="Permalink to this heading">#</a></h3>
<p>Beyond temperature, pH, solute concentration, O2 concentration, another important aspect of the environment is its nutritional status. One way that some prokaryotes deal with a poor nutritional environment is to differentiate into a resting cell.</p>
<ul class="simple">
<li><p>Vegetative cell: An active cellular form capable of reproduction.</p></li>
<li><p>Resting cell: A dormant cellular form, not “dead” but not capable of reproduction. The purpose of resting cell formation is to weather adverse nutritional conditions. Once suitable nutritional conditions arise, resting cells differentiate into vegetative cells.</p></li>
</ul>
<p>Different prokaryotes elaborate different resting forms including <strong>exospores, and endospores</strong>. Compared to their vegetative forms, resting cells have low to undetectable metabolic rates with the ability to remain dormant for months to years. They are also more resistant to desiccation (drying), UV light damage and heat stress. Below, one type of resting cell, the endospore, is discussed in more detail.</p>
<p><strong>Sporulation</strong></p>
<p>Depending on the organism, endospores are round to oval in shape and derive their name from the fact they are produced inside a vegetative cell and released when this cell ruptures. Endospore formation has been studied mostly in bacteria with a Gram-positive cell wall. It begins with copying of the genome and pseudo cell division yielding two membrane-bound compartments within one cell wall unit: a larger mother “cell” and a forespore, which ultimately becomes the endospore (Fig 50).</p>
<p>Following forespore formation, the mother cell membrane engulfs it, creating an additional membrane around the forespore. The outermost forespore membrane, derived from the mother cell’s membrane, has an opposite orientation to that of the forespore’s membrane, so the outside surfaces of both membranes face the space between them. This allows both the forespore and the mother cell to use their membrane transport proteins to transport materials into this space.</p>
<p>The forespore and mother cell “cooperate” to synthesize a cell wall made a thick cortex of modified PG, a spore coat made of covalently X-linked protein molecules and an exosporium. Simultaneously with cortex formation, large amounts of small positively charged proteins are synthesized which bind loosely to the DNA of the nucleoid, saturating and condensing it into a tightly packed, donut shaped structure
In addition to the nucleoid, the central core (which is surrounded by the core membrane) contains ribosomes. In the final act of sporulation, the cell wall of the mother cell (now called the sporangium) is degraded by autolytic enzymes releasing the mature spore.</p>
<div class="figure align-default" id="fig50">
<a class="reference internal image-reference" href="_images/3-18.png"><img alt="_images/3-18.png" src="_images/3-18.png" style="height: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 50 </span><span class="caption-text">Stages of bacterial sporulation.</span><a class="headerlink" href="#fig50" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="protein-assays">
<h2>3.5 Protein Assays<a class="headerlink" href="#protein-assays" title="Permalink to this heading">#</a></h2>
<p>Many of the different coping mechanisms described in the previous section rely on the expression of proteins. This can include enzymes that convert toxic oxygen species to non-toxic forms or protein transporters involved in regulating solute concentration. In order to study these coping mechanisms, researchers often measure the level of the proteins produced through protein assays such as SDS-PAGE and Western blotting.</p>
<div class="section" id="sds-page">
<h3>3.5. 1 SDS-PAGE<a class="headerlink" href="#sds-page" title="Permalink to this heading">#</a></h3>
<p>One technique that can be used to visualize protein is called <strong>SDS-PAGE (sodium dodecyl-sulfate polyacrylamide gel electrophoresis)</strong>. This technique involves separating proteins by size then visualizing them as bands on a gel. The gel material has a similar viscosity and texture as Jello^TM. It consists of many asymmetrical pores. In order to pass through, proteins have to navigate their way through the pores.</p>
<p>This lab technique follows these general steps (Fig 51):</p>
<ol class="arabic simple">
<li><p>Bacterial cells are grown in large amounts in a batch culture</p></li>
<li><p>The cells need to be lysed in order to release their intracellular content</p></li>
<li><p>The content is then purified and the proteins are denatured. Folded proteins are difficult to pass through a gel membrane.</p></li>
<li><p>The protein solution is loaded onto wells in the gel. These wells are opening at the top of the gel where solutions can be added into.</p></li>
<li><p>Once the proteins have been extracted, they are loaded onto the SDS-PAGE protein gel (Fig 51 (4)). This involves adding the samples to the top of the gel where receiving wells have been created. An electric current runs through the gel pulling the proteins through the gel matrix. The gel itself is like a sponge, containing various sized pores. Proteins that are relatively large take longer to move through the pores and therefore migrate very slowly down the gel. Smaller proteins, on the other hand, move relatively quickly and can make more distance in the same amount of time. This process allows us to separate the proteins based on size.</p></li>
<li><p>The gel is subsequently stained to allow us to visualize the proteins as bands. The position of the band indicates the protein size and the intensity of the band indicates the concertation (Fig 52). The lower the band, the smaller the protein. The more intense the band, the more concentrated the protein.</p></li>
</ol>
<div class="figure align-default" id="fig51">
<a class="reference internal image-reference" href="_images/3-19.png"><img alt="_images/3-19.png" src="_images/3-19.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 51 </span><span class="caption-text">Process of preparing an SDS-PAGE experiment.</span><a class="headerlink" href="#fig51" title="Permalink to this image">#</a></p>
</div>
<div class="figure align-default" id="fig52">
<a class="reference internal image-reference" href="_images/3-20.png"><img alt="_images/3-20.png" src="_images/3-20.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 52 </span><span class="caption-text">Example of SDS-PAGE gel where the molecular ruler is an indicator of what each band size is. Moving down the gel, protein size decreases. The intensity of the band (in this case, purple in color) represents more concentrated proteins.</span><a class="headerlink" href="#fig52" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="western-blot">
<h3>3.5.2 Western Blot<a class="headerlink" href="#western-blot" title="Permalink to this heading">#</a></h3>
<p>Although SDS-PAGE is efficient at separating all the proteins and visualizing them, it is not very efficient at highlighting any proteins of interest. Consider that bacterial cells have hundreds and thousands of proteins and many of them are the same size. Therefore, a single band on an SDS-PAGE gel could represent a mixture of multiple different proteins. A work-around is the <strong>Western Blot</strong>. The Western Blot allows you to visualize only proteins of interest by using specific antibodies that bind to your target protein and omit light so that you can create a photograph of the same SDS-PAGE gel but only with your target proteins. The intensity of the band still represents the concentration of your target protein.</p>
<p>Western blots generally follow four key steps (Fig 53):</p>
<ol class="arabic simple">
<li><p>You run your protein mixture in an SDS-PAGE assay</p></li>
<li><p>You transfer all the protein bands from the gel onto a membrane (this membrane often looks similar to a thin sheet of filter paper)</p></li>
<li><p>You add antibodies (specific to your protein(s) of interest) onto the membrane allowing them to bind. Once they bind, they omit a light signature that can be photographed.</p></li>
<li><p>A photo of the membrane can be taken (often black and white) that only shows bands containing your target protein.</p></li>
</ol>
<div class="figure align-default" id="fig53">
<a class="reference internal image-reference" href="_images/3-21.png"><img alt="_images/3-21.png" src="_images/3-21.png" style="height: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 53 </span><span class="caption-text">General protocol for a Western Blot.</span><a class="headerlink" href="#fig53" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="case-study-salt-and-starvation-dont-mix">
<h2>Case Study: Salt and starvation don’t mix<a class="headerlink" href="#case-study-salt-and-starvation-dont-mix" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Widderich et al., 2016</strong></p>
<p class="sd-card-text">Widderich N, Rodriguez CDA, Commichau FM, Fischer KE, Ramirez-Guadiana FH, Rudner DZ, Bremer E. 2016. Salt-sensitivity of σH and Spo0A prevents sporulation of Bacillus subtilis at high osmolarity avoiding death during cellular differentiation. Mol Microbiol. 100(1):108-124</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p>In this study, Widderich et al. wanted to study two environmental pressures, high salt and starvation conditions, on Bacillus subtilis, a model organism for studying sporulation.  They hypothesized that high salt conditions would inhibit B. subtilis sporulation under starvation conditions by inhibiting the expression of protein regulators involved in activating sporulation. In essence, they wanted to know if one type of stress factor would override the mechanism for coping by another stress factor like starvation</p>
<p>In order to test this hypothesis, Widderich et al. collected the following data.</p>
<div class="figure align-default" id="fig54">
<a class="reference internal image-reference" href="_images/3-22.png"><img alt="_images/3-22.png" src="_images/3-22.png" style="height: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 54 </span><span class="caption-text">(A) Representative phase-contrast images (type of light microscopy) of cells of wild-type <em>B. subtilis</em> cells (JH642) sporulated by nutrient exhaustion for 36 hours in the presence of the indicated concentrations of NaCl. (B) Quantitative assessment of increasing salt concentration on the formation of spores. (C) Bar graph showing the negative effects of 0.6 M and 1.2 M NaCl on sporulation of the indicated strains of <em>B. subtilis</em> strains and of <em>B. mojavensis</em> (a different species of Bacillus).</span><a class="headerlink" href="#fig54" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<p class="sd-card-text">Consider for each panel?</p>
<ol class="arabic simple">
<li><p class="sd-card-text">What technique was used to collect the data?</p></li>
<li><p class="sd-card-text">What are the treatment(s) and what is the control group?</p></li>
<li><p class="sd-card-text">Does the treatment affect the phenotype you are studying?</p></li>
<li><p class="sd-card-text">How does it correlate to the hypothesis?</p></li>
</ol>
<p class="sd-card-text">Panel B
5.	Does the data compliment the data in panel A?</p>
<p class="sd-card-text">Panel C
6.	Which strain shows the most sensitivity to salt?
7.	Why include a different species?</p>
</div>
</div>
</div>
<p>Next the researchers wanted to know what regulatory proteins could potentially be playing a role in mediating the effects observed in Fig 54. They needed a way to measure expression of those regulators.</p>
<div class="figure align-default" id="id1">
<img alt="_images/3-23.png" src="_images/3-23.png" />
<p class="caption"><span class="caption-number">Fig. 55 </span><span class="caption-text">Western blot data looking at the general expression levels (intensity of the band) of 4 different regulators treated with different concentrations of NaCl for up to 3 hours. All cultures were grown under starvation conditions.</span><a class="headerlink" href="#id1" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What is the control and what are the treatment groups?</p></li>
<li><p class="sd-card-text">For each regulator, looking at the control, consider if expression is increasing or decreasing as cells begin to sporulate over time.</p></li>
<li><p class="sd-card-text">What regulator(s) does not appear to be involved in regulating sporulation? How can you tell?</p></li>
<li><p class="sd-card-text">What regulator(s) appear to be involved in regulating sporulation?</p></li>
<li><p class="sd-card-text">Can you determine if there is one regulator who is more essential are regulating sporulation than others?</p></li>
<li><p class="sd-card-text">Consider that the process of sporulation is temporal (ie. it happens in stages over time), can you tell which regulators mediate the early, mid and late stages of sporulation?</p></li>
</ol>
</div>
</div>
</div>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="unit1-chap2.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Chapter 2: Microbial Cell Structures</p>
      </div>
    </a>
    <a class="right-next"
       href="unit1-chap4.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">&lt;no title&gt;</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#prokaryotic-cell-division">3.1 Prokaryotic Cell Division</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#individual-cell-growth-binary-fission">3.1.1 Individual Cell Growth: Binary Fission</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#batch-cell-growth-growth-curves">3.1.2 Batch cell growth: Growth curves</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microbes-growing-in-the-lab">3.2 Microbes Growing in the Lab</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#colony-forming-units">3.2.1 Colony Forming Units</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#optical-density">3.2.2 Optical density</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biomass-weight">3.2.3 Biomass weight</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#direct-microscopic-count">3.2.4 Direct microscopic count</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-stevia-effect-on-growth">Case Study: Stevia effect on growth</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#growth-rate">3.3 Growth rate</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microbes-growing-in-the-wild">3.4 Microbes Growing in the Wild</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#temperature">3.4.1 Temperature</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#solute-concentration">3.4.2 Solute concentration</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#ph-h-concentration">3.4.3 pH (H+ concentration)</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#oxygen-concentration">3.4.4 Oxygen concentration</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#nutritional-status">3.4.5 Nutritional Status</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-assays">3.5 Protein Assays</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#sds-page">3.5. 1 SDS-PAGE</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#western-blot">3.5.2 Western Blot</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-salt-and-starvation-dont-mix">Case Study: Salt and starvation don’t mix</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By MICB 211 Teaching Team led by Evelyn Sun
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2022.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>