

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Chapter 4: Antimicrobials and Resistance &#8212; MICB 211 Foundations in Microbiology eTextbook</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'unit2-chap4';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Chapter 3: Bacterial Pathogenesis and Virulence" href="unit2-chap3.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="intro.html">
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="intro.html">
                    MICB 211 E-Textbook Acknowledgements
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 1</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit1-chap1.html">Chapter 1: Introduction to Microbiology and Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap2.html">Chapter 2: Microbial Cell Structures</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap3.html">Chapter 3: Microbial Growth in the Lab and in Nature</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap4.html">Chapter 4: Gene Expression and Regulation</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap5.html">Chapter 5: Sensing and Responding to the Environment</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 2</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit2-chap1.html">Chapter 1: The Human Microbiome</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap2.html">Chapter 2: Community Lifestyle of a Biofilm</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap3.html">Chapter 3: Bacterial Pathogenesis and Virulence</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Chapter 4: Antimicrobials and Resistance</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Funit2-chap4.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/unit2-chap4.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Chapter 4: Antimicrobials and Resistance</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#therapy-and-prevention-of-bacterial-diseases">4.1 Therapy and Prevention of Bacterial Diseases</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotherapeutic-agents">4.1.1 Chemotherapeutic agents</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#how-antibiotics-generally-work">4.1.2 How antibiotics generally work</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#testing-for-antibiotic-sensitivity">4.2 Testing for Antibiotic Sensitivity</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#disk-diffusion-assay">4.2.1 Disk Diffusion Assay</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#minimal-inhibitory-concentration-mic">4.2.2 Minimal Inhibitory Concentration (MIC)</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-antibiotic-susceptibility-in-mycobacterium-tuberculosis">Case Study: Antibiotic susceptibility in <em>Mycobacterium tuberculosis</em></a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#antibiotic-resistance">4.3 Antibiotic Resistance</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#acquiring-resistance-bacteria">4.3.1 Acquiring Resistance – Bacteria</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#acquiring-resistance-human-contributions">4.3.2 Acquiring Resistance – Human Contributions</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#a-global-crisis">4.3.3 A global crisis</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-antibiotic-resistance-in-mycobacteroides-abscessus">Case Study: Antibiotic resistance in <em>Mycobacteroides abscessus</em></a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <div class="tex2jax_ignore mathjax_ignore section" id="chapter-4-antimicrobials-and-resistance">
<h1>Chapter 4: Antimicrobials and Resistance<a class="headerlink" href="#chapter-4-antimicrobials-and-resistance" title="Permalink to this heading">#</a></h1>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Learning Objectives</strong></p>
<p class="sd-card-text">By the end of this chapter, students will be expected to:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">define key terms related to antibacterial agents and resistance.</p></li>
<li><p class="sd-card-text">differentiate between bactericidal, bacteriostatic, broad-spectrum and narrow-spectrum antibiotics.</p></li>
<li><p class="sd-card-text">identify features that make a successful antibiotic.</p></li>
<li><p class="sd-card-text">describe common modes of antibiotic action against bacteria and why they are “safe” for host cells.</p></li>
<li><p class="sd-card-text">describe the ways bacteria are innately resistant to antibiotics and how resistance can be acquired.</p></li>
<li><p class="sd-card-text">understand how human practices contribute to antibiotic resistance and why it is now a global crisis.</p></li>
<li><p class="sd-card-text">interpret data using antibiotic assays like MIC and disc diffusion.</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="therapy-and-prevention-of-bacterial-diseases">
<h2>4.1 Therapy and Prevention of Bacterial Diseases<a class="headerlink" href="#therapy-and-prevention-of-bacterial-diseases" title="Permalink to this heading">#</a></h2>
<div class="section" id="chemotherapeutic-agents">
<h3>4.1.1 Chemotherapeutic agents<a class="headerlink" href="#chemotherapeutic-agents" title="Permalink to this heading">#</a></h3>
<p>The most prominent mode of combating infectious diseases involves the use of chemical compounds known as chemotherapeutic agents. These compounds can be used internally. This is in contrast to <strong>antiseptics</strong> which are chemical agents that are applied to living tissues to kill or inhibit the growth of microorganisms (these compounds are used for washing hands or treating surface wounds), or <strong>disinfectants</strong> which are chemical agents used on inanimate objects to kill microorganisms (most of these agents are too toxic to be used inside the body). The key requirement of chemotherapeutic agents is their selective toxicity; these compounds inhibit or kill bacteria without causing serious harm to the host.  Chemotherapeutic agents that are able to kill bacteria or inhibit the growth of bacteria are usually called <strong>antibiotics</strong>.</p>
<p>Antibiotics were initially differentiated from synthetic compounds, like Sulfa drugs, because they were the natural products derived from microbial activity. Many antibiotics have been modified by chemical changes carried out in the laboratory and are described as semi-synthetic antibiotics. In the 20th century, scientists have discovered ways to make or modify antibiotics, thus making the current antibiotic field an array of natural, <strong>semi-synthetic and synthetic antibiotics</strong>. The most commonly used antibiotics in hospitals are the beta-lactams.  These consist of the naturally derived penicillin-like and cephalosporin antibiotics and their many semi-synthetic derivatives.  Notably, they contain a common 4-membered ring termed the beta-lactam ring  (Fig 122).</p>
<div class="figure align-default" id="fig122">
<img alt="_images/4-110.png" src="_images/4-110.png" />
<p class="caption"><span class="caption-number">Fig. 122 </span><span class="caption-text">The β-lactam ring structure (left) and penicillin (right).  A β -lactam ring is a four-member lactam (a cyclic amide). The β -lactam ring forms the core structure of several antibiotics, such as penicillin.</span><a class="headerlink" href="#fig122" title="Permalink to this image">#</a></p>
</div>
<p>The sensitivity of microorganisms to antibiotics and other chemotherapeutic agents varies.  Gram-positive bacteria are usually more sensitive to antibiotics than Gram-negative bacteria.  A <strong>broad-spectrum antibiotic</strong> (sometimes called a ‘chemotherapeutic agent’) is one that acts on both Gram-positive and Gram-negative bacteria and is more frequently used in medicine; a <strong>narrow-spectrum antibiotic</strong> is one that acts on only a single group of organisms (which may still be useful if a broad-spectrum antibiotic fails to resolve an infection, or for special organisms like <em>Mycobacterium tuberculosis</em>).</p>
<p><strong>Optimal attributes of chemotherapeutic agents are:</strong></p>
<ul class="simple">
<li><p>Solubility in body fluids with good penetration to the infection site</p></li>
<li><p>Effectiveness against bacteria at a low concentration, i.e., a low MIC (see section 4.2.2)</p></li>
<li><p>A broad spectrum of activity i.e., activity against a broad range of bacteria.  Individual antibiotics tend to vary in their activity spectrum</p></li>
<li><p>Low frequency of resistance development</p></li>
<li><p>Low toxicity, well-tolerated in the human body</p></li>
<li><p>Non-allergenic</p></li>
</ul>
<p>Antibiotics don’t always possess all of these optimal attributes, but they do tend to be safe and effective drugs. After having covered the microbiota in Unit 2 Chap 1, consider how antibiotic therapy could affect a person’s health through the microbiota.</p>
</div>
<div class="section" id="how-antibiotics-generally-work">
<h3>4.1.2 How antibiotics generally work<a class="headerlink" href="#how-antibiotics-generally-work" title="Permalink to this heading">#</a></h3>
<p>For an antibiotic to be useful in medicine, it must be able to inhibit bacterial growth or kill bacterial cells but not affect host cells.  For example, the antibiotic can target a metabolic process that is not found in the host cell, or the antibiotic can target a bacterial protein that is sufficiently different from the homologous protein in the host cell.</p>
<p>Chemotherapeutic agents kill or inhibit the growth of bacteria by a variety of different mechanisms. The primary way that most antibiotics work is by targeting and often inhibiting a key process in bacterial growth and/or survival (eg. transcription, translation, DNA replication) (Fig 123). The reason(s) they don’t also kill our cells at the prescribed dose is sometimes clear, sometimes not. Some examples of antibiotics and their activity are listed in Table 4.1</p>
<div class="figure align-default" id="fig123">
<img alt="_images/4-22.png" src="_images/4-22.png" />
<p class="caption"><span class="caption-number">Fig. 123 </span><span class="caption-text">Different antibiotics kill or inhibit the growth of bacteria by targeting different parts of bacterial metabolism.</span><a class="headerlink" href="#fig123" title="Permalink to this image">#</a></p>
</div>
<p><strong>Table 4.1</strong>: Common antibiotic classes and their targets.</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Antibiotic or antibiotic class</p></th>
<th class="head"><p>Mode of action against bacteria</p></th>
<th class="head"><p>Why it doesn’t affect host cells</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>β-lactam   antibiotics</p></td>
<td><p>Interfere with bacterial   cell wall biosynthesis by inhibiting transpeptidation reactions during   peptidoglycan biosynthesis</p></td>
<td><p>Eukaryotic cells do not have   peptidoglycan</p></td>
</tr>
<tr class="row-odd"><td><p>Polymyxins</p></td>
<td><p>Destabilize the cell   membrane</p></td>
<td><p>Specific against bacterial cell   membrane. These compounds are more toxic to humans than other antibiotics so   only used as a last resort</p></td>
</tr>
<tr class="row-even"><td><p>Erythromycin, Tetracyclines,   and the new Oxazolidinones (bacteriostatic drugs).</p></td>
<td><p>Interfere with protein synthesis   by binding to the ribosome and inhibiting its function.</p></td>
<td><p>Bacteria and eukaryotic cells have   distinct ribosome structures</p></td>
</tr>
<tr class="row-odd"><td><p>Nalidixic acid and     Ciprofloxacin</p></td>
<td><p>Interfere with DNA gyrase   activity (DNA unwinding during replication of the chromosome)</p></td>
<td><p>DNA replication is different   enough in bacteria vs. humans for these compounds to generally not be toxic   to us</p></td>
</tr>
<tr class="row-even"><td><p>Rifampin</p></td>
<td><p>Interfere with   DNA-directed RNA polymerase activity inhibiting transcription</p></td>
<td><p>Bacterial RNA polymerase is different   enough from ours not to be toxic</p></td>
</tr>
<tr class="row-odd"><td><p>Sulfonamides and Trimethoprim</p></td>
<td><p>Interfere with folic acid   metabolism</p></td>
<td><p>These processes are specific to   bacteria</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="testing-for-antibiotic-sensitivity">
<h2>4.2 Testing for Antibiotic Sensitivity<a class="headerlink" href="#testing-for-antibiotic-sensitivity" title="Permalink to this heading">#</a></h2>
<div class="section" id="disk-diffusion-assay">
<h3>4.2.1 Disk Diffusion Assay<a class="headerlink" href="#disk-diffusion-assay" title="Permalink to this heading">#</a></h3>
<p>Antibiotic therapy may be started immediately without actual knowledge of the identity of the pathogen or the pathogen’s sensitivity to antibiotics.  However, such information is critical if there is a subsequent lack of therapeutic response in the patient.  One simple method to determine the antibiotic sensitivity of a clinical isolate (ie. strain of bacteria isolated from a medical case) is the disk diffusion antibiotic susceptibility test (Fig 124).</p>
<div class="figure align-default" id="fig124">
<img alt="_images/4-31.png" src="_images/4-31.png" />
<p class="caption"><span class="caption-number">Fig. 124 </span><span class="caption-text">Antibiotic sensitivity testing. In the above image, the bacteria are sensitive to all antibiotics except #4 and #6.</span><a class="headerlink" href="#fig124" title="Permalink to this image">#</a></p>
</div>
<p>In this test, the bacterial isolate is spread evenly onto the surface of an agar plate so that when the bacteria grow, it will form a confluent lawn. Paper discs impregnated with various antibiotics are placed on the surface of the plate and the antibiotic is allowed to diffuse into the culture medium. As the bacteria on the lawn grow, they are inhibited to varying degrees by the antibiotic diffusing from the disk.  Inhibition of bacterial growth around a disk following incubation indicates sensitivity to the particular antibiotic contained in the disk.  The radius of the bacteria free circular zones is referred to as the “<strong>zone of inhibition</strong>”. The size of the zone of inhibition is dependent on the diffusion rate of the antibiotic, the degree of sensitivity of the microorganism, and the growth rate of the bacterium.  The test is performed under standardized conditions. The zones of inhibition under standard conditions have been established for each antibiotic.  If the zone of inhibition is equal to or greater than the standard, the organism is considered to be sensitive or <strong>susceptible</strong> to the antibiotic.  If the zone of inhibition is less than the standard, the organism is considered to be <strong>resistant</strong>.</p>
</div>
<div class="section" id="minimal-inhibitory-concentration-mic">
<h3>4.2.2 Minimal Inhibitory Concentration (MIC)<a class="headerlink" href="#minimal-inhibitory-concentration-mic" title="Permalink to this heading">#</a></h3>
<p>Antibiotics can be classified as <strong>bactericidal</strong> (kill the bacteria) or <strong>bacteriostatic</strong> (inhibit bacterial growth without directly killing) (1).  When the concentration of bacteriostatic antibiotics falls below a certain concentration (the minimal inhibitory concentration or MIC) the bacteria start to grow again, whereas with bactericidal antibiotics, the bacteria cannot grow when the concentration falls below the MIC (1).</p>
<p>The MIC of an antibiotic against a specific bacteria can be determined by finding the lowest concentration of antibiotic that prevents visible bacterial growth. MICs are the “gold standard” of determining bacterial susceptibility to an antibiotic, and they are utilized when other methods, for example disk diffusion, lack definitive results (2). To perform the MIC measurement, bacteria are exposed to a range of antibiotic concentrations on either liquid or solid growth media (Fig 125 shows an example using liquid media dispensed into a 96-well plate). The antibiotic and growth conditions will be case dependent.</p>
<div class="figure align-default" id="fig125">
<img alt="_images/4-41.png" src="_images/4-41.png" />
<p class="caption"><span class="caption-number">Fig. 125 </span><span class="caption-text">(Left plate) Layout of a typical 96-well plate MIC assay. This example tests eight different antibiotics. The gradient symbol along the top indicates the high to low concentration gradient of each antibiotic. Often consecutive concentrations are generated through two-fold serial dilution steps. The sterility control in column 1 is bacteria-free broth, while the growth control in column 12 contains antibiotic-free bacterial culture. (Right plate) Determining MICs based on bacterial growth. MIC is the lowest concentration of antibiotic in which no bacterial growth is observed. Note the expected phenotype in each of the control columns.</span><a class="headerlink" href="#fig125" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="case-study-antibiotic-susceptibility-in-mycobacterium-tuberculosis">
<h2>Case Study: Antibiotic susceptibility in <em>Mycobacterium tuberculosis</em><a class="headerlink" href="#case-study-antibiotic-susceptibility-in-mycobacterium-tuberculosis" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Shah et al., 2016</strong></p>
<p class="sd-card-text">Shah S, Dalecki AG, Malalasekera AP, Crawford CL, Michalek SM, Kutsch O, Sun J, Bossmann SH, Wolschendorf F. 8-Hydroxyquinolines Are Boosting Agents of Copper-Related Toxicity in Mycobacterium tuberculosis. Antimicrob Agents Chemother. 2016 ;60(10):5765-76. doi: 10.1128/AAC.00325-16.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p><em>Mycobacterium tuberculosis</em> is the causative agent of tuberculosis and historically the leading cause of death due to infectious disease globally. While that title has been replaced by COVID-19, it is still the leading cause of death due to bacterial infection. Due to drug resistance and difficult treatment regimes, new antimicrobials are in desperate need. This study aims to identify a compound whose antimicrobial activity against M. tuberculosis is enhanced in the presence of copper. The rational for this combination is the utilization of copper by the host immune system as a natural antimicrobial. The chemical compound investigated in this study is the broad-spectrum antimicrobial, 8-hydroxyquinoline (8HQ). Dalecki et al. hypothesized that 8HQ activity is enhanced in the presence of copper both <em>in vitro</em> and <em>in vivo</em>.</p>
<div class="margin sidebar">
<p class="sidebar-title"><em>in vitro and in vivo</em></p>
<p>Common terms used to describe studies of pathogens are <em>in vitro</em> and <em>in vivo</em>. <em>In vitro</em> is Latin for “in the glass” and refers to artificial conditions generated in a laboratory setting. <em>In vivo</em>, on the other hand, is Latin for “within the living” and refers to analysis of pathogens within living cells or organisms. In this case, macrophages are living immune cells and more closely model infection conditions than what the bacteria would experience in growth media (4).</p>
</div>
<p>In order to test this hypothesis, Dalecki et al. had to go through the following steps:</p>
<p><strong>Step 1: Determining MIC of 8HQ ± Copper</strong></p>
<p>The effect of copper on 8HQ activity was determined through calculating change in MIC. This was monitored in growing (G) and recovering (R) bacteria.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>Growing vs. Recovering M. tuberculosis</strong></p>
<p>One of the major features of disease caused by M. tuberculosis is the transition from actively replicating bacteria (growing) to a dormant, non-replicating cell (recovery) (3). This can make treating this pathogen with antibiotics difficult as they often target processes found in growing bacterial cells. In this study, dormant bacteria were induced “artificially” outside of the host and were therefore considered recovering once they were placed into the growth conditions used to measure MICs (3).</p>
</div>
<div class="figure align-default" id="fig126">
<img alt="_images/4-51.png" src="_images/4-51.png" />
<p class="caption"><span class="caption-number">Fig. 126 </span><span class="caption-text">(A) Fluorescence measure of bacterial cell count at different concentrations of 8HQ as growing (G) and recovering (R) cells in the presence or absence of copper (Cu). (B) MIC of growing and recovery <em>M. tuberculosis</em> cells with (+) and without (-) copper. The concentration differential of 8HQ between individual data points is 2-fold, with the maximum being 10 μM.</span><a class="headerlink" href="#fig126" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">Based on this figure, what are the MICs under each condition?</p></li>
<li><p class="sd-card-text">What is the purpose of showing both panels A and B? Are there differences in the data?</p></li>
<li><p class="sd-card-text">What is the purpose of looking at both growing and recovering <em>M. tuberculosis</em> cells? What do the results in both groups tell us about 8HQ - Cu activity, 8HQ + Cu activity?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that copper enhances the antimicrobial activity of 8HQ?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 2: Determining Effect of 8HQ ± Copper during infection</strong></p>
<p>To determine the efficacy of 8HQ <em>in vivo</em>, growth of <em>M. tuberculosis</em> in macrophages (a type of immune cell) was monitored.</p>
<div class="figure align-default" id="fig127">
<img alt="_images/4-61.png" src="_images/4-61.png" />
<p class="caption"><span class="caption-number">Fig. 127 </span><span class="caption-text"><em>M. tuberculosis</em>-infected macrophages (ie. immune cells) were treated for 48 h with 1 μM or 10 μM 8HQ. hpi = hours post infection. Panel A was purposely removed. Δ indicates significant differences.</span><a class="headerlink" href="#fig127" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What is the role of the input column? What about the 0 8HQ column?</p></li>
<li><p class="sd-card-text">Based on this figure. What can you conclude by comparing the 1 vs. 10 8HQ columns?</p></li>
<li><p class="sd-card-text">Why was this study of infection performed in macrophages?</p></li>
<li><p class="sd-card-text">Based on the MIC you determined in Fig. 126, why were concentrations of 1 and 10 8HQ utilized.</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that copper enhances the antimicrobial activity of 8HQ?</p></li>
<li><p class="sd-card-text">Looking at both Fig. 126 and 127, what can you conclude about the activity of 8HQ ± Cu <em>in vitro</em> and <em>in vivo</em>.</p></li>
</ol>
</div>
</div>
</div>
<p></p>
</div>
<div class="section" id="antibiotic-resistance">
<h2>4.3 Antibiotic Resistance<a class="headerlink" href="#antibiotic-resistance" title="Permalink to this heading">#</a></h2>
<div class="section" id="acquiring-resistance-bacteria">
<h3>4.3.1 Acquiring Resistance – Bacteria<a class="headerlink" href="#acquiring-resistance-bacteria" title="Permalink to this heading">#</a></h3>
<p>The acquired ability of microorganisms to resist the effects of an antibiotic to which it is normally sensitive is known as <strong>antibiotic resistance</strong>.  Antibiotic resistance in bacterial infections was first observed shortly after the first antibiotic was introduced into clinical use in the 1940s.</p>
<p>Not all antibiotics act against all microorganisms. Bacteria may be naturally resistant to an antibiotic because they lack the structures that the antibiotic inhibits. For example, mycoplasmas (a causative agent of pneumonia) are resistant to penicillin because they lack peptidoglycan. Another reason a bacterium may be resistant is because many organisms naturally possess rather non-specific efflux pumps that transport antibiotics out of the organism before the drug can be effective. Some Gram-negative bacteria are naturally resistant to some types of antibiotic due to a combination of poor permeability across the outer membrane combined with secondary mechanisms such as efflux and antibiotic inactivating enzymes that take advantage of the slow uptake of antibiotics into the bacterial cell. This is known as <strong>intrinsic resistance</strong>, a regular or natural feature of the bacteria that allows it to resist the antibiotic.</p>
<p>Bacteria can also be resistant to antibiotics by modifying the target of the antibiotic.  For example, bacteria can become resistant to a given antibiotic because of mutations that change the structure of the target site (common in bacteria that are producing the antibiotic), alter a metabolic pathway, or change the permeability of the outer membrane. These mutations typically occur naturally during DNA replication, when ‘mistakes’ are made that actually benefit the bacteria in the presence of an antibiotic. These mutations are then passed to the progeny of the cell (<strong>vertical gene transfer</strong>) during cell division to create a larger pool of antibiotic resistant bacteria. This is known as <strong>acquired resistance</strong>.</p>
<p>Alternatively, some bacteria carry specific genetic information (‘<strong>resistance genes</strong>’) that renders them resistant to the antibiotic they produce. For example, they may have a gene that encodes an enzyme that can modify the antibiotic by breaking a bond or by adding a phosphate group.  Sensitive bacteria can acquire resistance genes through horizontal gene transfer (Unit 1 Chapter 4).</p>
</div>
<div class="section" id="acquiring-resistance-human-contributions">
<h3>4.3.2 Acquiring Resistance – Human Contributions<a class="headerlink" href="#acquiring-resistance-human-contributions" title="Permalink to this heading">#</a></h3>
<p>When discussing antibiotic resistance, it is important to note that exposure to antibiotics does not cause bacteria to become drug resistant. Changes in the bacterium that enable it to resist the antibiotic occur naturally as a result of mutation or genetic exchange and recombination. Exposure to the antibiotic selects for outgrowth of the bacteria that have become resistant through natural selection.</p>
<p>Resistance may develop in bacteria through the natural process of mutation and genetic recombination, but antibiotic abuse encourages the emergence of antibiotic resistant forms. A number of surveys suggest that antibiotics are used in clinical practice far more often than is necessary. Data suggests that antibiotic treatment is warranted for only 20% of individuals seen for clinical infections, yet antibiotics are prescribed up to 80% of the time. Furthermore, in up to 50% of cases, the recommended doses or duration of treatment are not correct.</p>
<p>To add to this problem, many people fail to complete their prescriptions. Many patients stop taking their antibiotics as soon as they are ‘feeling better.’  Thus, the virulent pathogens are subjected to sub-lethal doses of antibiotics and remain alive to evolve into resistant organisms. Upon using the same antibiotic in the future, it will kill the sensitive bacteria (even those bacteria that are beneficial), leaving the resistant bacteria to thrive because they face reduced competition from susceptible organisms.</p>
<p>Hospitals are another environment driving the emergence of resistant bacteria. In many cases, large doses of antibiotics are used to prevent infection during and following surgery. Finally, antibiotics are often available in developing countries without prescription.</p>
<p>The largest area of antibiotic misuse is within the agricultural industries. Statistics from 2016 indicate that in North America, approximately 80% of the antibiotics produced are used in agriculture. Up to 90% of those are not used to treat sick animals, but rather as growth-promoting agents in livestock and for the prophylactic treatment of livestock, crops, and farmed fish.</p>
<p>Fortunately, government agencies in both Canada and the U.S. are now creating much stricter regulations which came into effect in 2017 and 2018, which should bring those numbers down considerably. There is strong evidence that the use of antibiotics in agriculture results in antibiotic resistance in bacteria that can then be transferred to humans through handling these animals and food consumption. Aquaculture is another area in which concerns have been raised regarding antibiotic use in the environment. An example of this is shown in Figure 128.</p>
<p><em>Campylobacter jejuni</em> is a major foodborne pathogen causing severe diarrheal disease. It can easily be acquired in North America through the consumption of mishandled poultry products and is often the leading cause of food poisoning in developed countries. However, it is much more likely to be acquired when traveling abroad. Ciprofloxacin, a fluoroquinolone, is the first-line antibiotic prescribed by travel doctors in case the traveler contracts a severe case of diarrhea. Unfortunately, as even this older graph shows, due to use in agriculture and over-the-counter availability, fluoroquinolones are not the best option at present, especially in certain parts of the world.</p>
<div class="figure align-default" id="fig128">
<img alt="_images/4-71.png" src="_images/4-71.png" />
<p class="caption"><span class="caption-number">Fig. 128 </span><span class="caption-text">Trends for fluoroquinolone resistance rates (in percentages) in <em>Campylobacter jejuni</em> from human sources around the world.</span><a class="headerlink" href="#fig128" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="a-global-crisis">
<h3>4.3.3 A global crisis<a class="headerlink" href="#a-global-crisis" title="Permalink to this heading">#</a></h3>
<p>The emergence and spread of multi-drug-resistant pathogens has increased substantially over the past twenty years. The most obvious consequence of antibiotic use has been the increased rates of mortality when physicians are not able to treat patients effectively and in a timely manner. In some cases, the only drugs available to treat a particular infection have been rendered ineffective because of antibiotic resistance.</p>
<p>New <strong>superbugs</strong> that are resistant to most or all types of antibiotic treatment have arisen (e.g., MRSA: methicillin-resistant <em>Staphylococcus aureus</em>, VRE: vancomycin-resistant Enterococcus; imipenem resistant <em>Pseudomonas aeruginosa</em>, penicillin resistant Streptococcus pneumoniae).  These superbugs are resistant to many different antibiotics.</p>
<p>A great concern is the rise of multidrug resistant and extensively drug resistant M. tuberculosis. Due to its long-standing status as the leading cause of death due to infectious disease (just recently out ranked by COVID-19), these drug resistant strains are severely increasing the global burden of this pathogen. These strains can be resistant to several if not all of the front-line drugs and some second-line drugs, greatly reducing number of effective antibiotics for treatment. Furthermore, antibiotic resistance is costly. For example, the total cost of treatment of antibiotic susceptible tuberculosis (in USD) is <code class="docutils literal notranslate"><span class="pre">$</span></code>18,000 whereas the treatment of the most antibiotic resistant tuberculosis can cost <code class="docutils literal notranslate"><span class="pre">$</span></code>513,000 per patient. There is also increased mortality (5).</p>
<p>In order to combat the increasing global occurrence of antimicrobial and antibiotic resistant bacteria, in 2015 the World Health Organization (WHO) endorsed a Global Action Plan. You can see more information on the details of this plan and the global strategies at the WHO Global Action Plan webpage: (<a class="reference external" href="https://ahpsr.who.int/publications/i/item/global-action-plan-on-antimicrobial-resistance">https://ahpsr.who.int/publications/i/item/global-action-plan-on-antimicrobial-resistance</a>)</p>
<p></p>
</div>
</div>
<div class="section" id="case-study-antibiotic-resistance-in-mycobacteroides-abscessus">
<h2>Case Study: Antibiotic resistance in <em>Mycobacteroides abscessus</em><a class="headerlink" href="#case-study-antibiotic-resistance-in-mycobacteroides-abscessus" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Dal Molin et al., 2017</strong></p>
<p class="sd-card-text">Dal Molin M, Gut M, Rominski A, Haldimann K, Becker K, Sander P. Molecular Mechanisms of Intrinsic Streptomycin Resistance in Mycobacterium abscessus. Antimicrob Agents Chemother. 2017 Dec 21;62(1):e01427-17. doi: 10.1128/AAC.01427-17.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p><em>Mycobacterium abscessus</em> is a closely related pathogen to <em>M. tuberculosis</em> and also a leading agent of lung disease in immunocompromised individuals. One of the struggles faced when treating <em>M. abscessus</em> infections is the highly drug resistant nature of this pathogen. In fact, <em>M. abscessus</em> is resistant to streptomycin, the first ever drug used in the treatment of tuberculosis. In order to determine a genetic determinant of resistance, the authors of this study identified the gene encoding the enzyme MAB_2385 as a potential resistance gene. Notice that this gene doesn’t even have a proper name. It is because it has not been well studied and described. Dal Molini et al. hypothesized that presence of MAB_2385 in the <em>M. abscessus</em> genome leads to streptomycin resistance.</p>
<p>In order to test this hypothesis, Dal Molin et al. underwent the following experiments:</p>
<p><strong>Step 1: Determining MIC of streptomycin in <em>M. abscessus</em> ± MAB_2385</strong></p>
<p>MICs were determined using a streptomycin Etest. This was performed on wild type M. abscessus, the MAB_2385 knockout, and a complement strain of this knockout</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>Etests</strong></p>
<p>An Etest combines both the concepts of a disk diffusion and MIC assay (7). Etests are a method of determining MICs by using a permeable strip (similar to the disk in disk diffusion) on a bacterial lawn. They contain a gradient of antibiotic along the length of the strip that allow you to determine the MIC based on the ability of a bacteria to grow around the strip. The antibiotic concentrations are indicated numerically along the length to allow you to estimate the MIC (7).</p>
</div>
<div class="figure align-default" id="fig129">
<img alt="_images/4-81.png" src="_images/4-81.png" />
<p class="caption"><span class="caption-number">Fig. 129 </span><span class="caption-text">Streptomycin Etest of <em>M. abscessus</em> ATCC 19977T, the MAB_2385 gene deletion mutant <em>M. abscessus</em> ΔMAB_2385, and the complementation vector pMV361-aac(3)IV-MAB_2385. Concentrations on the Etest strips are in mg/L.</span><a class="headerlink" href="#fig129" title="Permalink to this image">#</a></p>
</div>
<p><strong>Step 2: Heterologously expressing MAB_2385 in <em>Mycobacterium smegmatis</em></strong></p>
<p>To further test the ability of MAB_2385 to induce streptomycin resistance, an expression plasmid (pMV361-aac(3)IV-MAB_2385) containing the gene was transformed into another, closely related bacterium, <em>M. smegmatis</em> which known to not have this gene naturally in its genome. The MICs for the WT and transformed strains were measured  as shown in Table 4.2.</p>
<p>Table 4.2. MIC of streptomycin against M. smegmatis strains with and without the MAB_2385 gene transformed into it.</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p><strong>Strain</strong></p></th>
<th class="head"><p><strong>MIC (mg/L)</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><em>M.   smegmatis</em>      (empty   pMV361 plasmid)</p></td>
<td><p>0.5</p></td>
</tr>
<tr class="row-odd"><td><p>pMV361-aac(3)IV-MAB_2385 <em>M. smegmatis</em></p></td>
<td><p>4</p></td>
</tr>
</tbody>
</table>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What form of antibiotic resistance is presented by <em>M. abscessus</em>? Intrinsic or acquired (if it is acquired, describe the mode of acquisition)?</p></li>
<li><p class="sd-card-text">What is the role of the complement strain? What information does it provide?</p></li>
<li><p class="sd-card-text">Looking at this data, what are the MICs for all three strains.</p></li>
<li><p class="sd-card-text">This method of determining MICs is different than the dilution plating method mentioned in this chapter. Using your understanding of bacterial growth techniques from Unit 1, try to determine how this experiment was set up/performed.</p></li>
<li><p class="sd-card-text">Based on Figure 129. What can you conclude about the role of MAB_2385 in streptomycin resistance? Does this support the author’s hypothesis?</p></li>
<li><p class="sd-card-text">In Table 4.2, why would the authors transform WT <em>M. smegmatis</em> with an empty pMV361 plasmid?</p></li>
<li><p class="sd-card-text">Does the data presented in Table 4.2 support the hypothesis? Why or why not?</p></li>
<li><p class="sd-card-text">In the experiment described in Step 2 of the case study, what form of antibiotic resistance is presented? Intrinsic or acquired (if it is acquired, describe the mode of acquisition)?</p></li>
</ol>
</div>
</div>
</div>
</div>
<div class="section" id="references">
<h2>References:<a class="headerlink" href="#references" title="Permalink to this heading">#</a></h2>
<ol class="arabic simple">
<li><p>Wiegand, I., Hilpert, K. &amp; Hancock, R. E. W. Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances. Nat Protoc 3, 163–175 (2008).</p></li>
<li><p>Andrews, J. M. Determination of minimum inhibitory concentrations. J Antimicrob Chemother 48 Suppl 1, 5–16 (2001).</p></li>
<li><p>Shah, S. et al. 8-Hydroxyquinolines Are Boosting Agents of Copper-Related Toxicity in Mycobacterium tuberculosis. Antimicrob Agents Chemother 60, 5765–5776 (2016).</p></li>
<li><p>In Vivo vs. In Vitro: Definition, Examples, and More. Healthline <a class="reference external" href="https://www.healthline.com/health/in-vivo-vs-in-vitro">https://www.healthline.com/health/in-vivo-vs-in-vitro</a> (2019).</p></li>
<li><p>CDCTB. Tuberculosis (TB) - Drug-Resistant TB. Centers for Disease Control and Prevention <a class="reference external" href="https://www.cdc.gov/tb/topic/drtb/default.htm">https://www.cdc.gov/tb/topic/drtb/default.htm</a> (2022).</p></li>
<li><p>Dal Molin, M. et al. Molecular Mechanisms of Intrinsic Streptomycin Resistance in Mycobacterium abscessus. Antimicrob Agents Chemother 62, e01427-17 (2017).</p></li>
<li><p>ETEST®. bioMérieux <a class="reference external" href="https://www.biomerieux.ca/en/product/etestr">https://www.biomerieux.ca/en/product/etestr</a>.</p></li>
</ol>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="unit2-chap3.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Chapter 3: Bacterial Pathogenesis and Virulence</p>
      </div>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#therapy-and-prevention-of-bacterial-diseases">4.1 Therapy and Prevention of Bacterial Diseases</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotherapeutic-agents">4.1.1 Chemotherapeutic agents</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#how-antibiotics-generally-work">4.1.2 How antibiotics generally work</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#testing-for-antibiotic-sensitivity">4.2 Testing for Antibiotic Sensitivity</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#disk-diffusion-assay">4.2.1 Disk Diffusion Assay</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#minimal-inhibitory-concentration-mic">4.2.2 Minimal Inhibitory Concentration (MIC)</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-antibiotic-susceptibility-in-mycobacterium-tuberculosis">Case Study: Antibiotic susceptibility in <em>Mycobacterium tuberculosis</em></a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#antibiotic-resistance">4.3 Antibiotic Resistance</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#acquiring-resistance-bacteria">4.3.1 Acquiring Resistance – Bacteria</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#acquiring-resistance-human-contributions">4.3.2 Acquiring Resistance – Human Contributions</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#a-global-crisis">4.3.3 A global crisis</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-antibiotic-resistance-in-mycobacteroides-abscessus">Case Study: Antibiotic resistance in <em>Mycobacteroides abscessus</em></a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By MICB 211 Teaching Team led by Evelyn Sun
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2022.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>