

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Chapter 2: Microbial Cell Structures &#8212; MICB 211 Foundations in Microbiology eTextbook</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'unit1-chap2';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Chapter 3: Microbial Growth in the Lab and in Nature" href="unit1-chap3.html" />
    <link rel="prev" title="Chapter 1: Introduction to Microbiology and Data Analysis" href="unit1-chap1.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="intro.html">
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="intro.html">
                    MICB 211 E-Textbook Acknowledgements
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 1</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit1-chap1.html">Chapter 1: Introduction to Microbiology and Data Analysis</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Chapter 2: Microbial Cell Structures</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap3.html">Chapter 3: Microbial Growth in the Lab and in Nature</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap4.html">Chapter 4: Gene Expression and Regulation</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap5.html">Chapter 5: Sensing and Responding to the Environment</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 2</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit2-chap1.html">Chapter 1: The Human Microbiome</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap2.html">Chapter 2: Community Lifestyle of a Biofilm</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap3.html">Chapter 3: Bacterial Pathogenesis and Virulence</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap4.html">Chapter 4: Antimicrobials and Resistance</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Funit1-chap2.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/unit1-chap2.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Chapter 2: Microbial Cell Structures</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-shape">2.1 Cell Shape</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#individual-cell-shapes">2.1.1 Individual cell shapes</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#group-cell-shapes">2.1.2 Group cell shapes</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#extracellular-structures">2.2 Extracellular Structures</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#capsule">2.2.1 Capsule</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-wall">2.2.2 Cell Wall</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cytoplasmic-or-plasma-membrane">2.2.3 Cytoplasmic or Plasma Membrane</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#appendages">2.2.4 Appendages</a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#flagella">Flagella</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#pili">Pili</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-stain">2.3 Gram Stain</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-positive-cells">2.3.1 Gram Positive Cells</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-negative-cells">2.3.2 Gram Negative Cells</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#how-the-procedure-works">2.3.3 How the procedure works</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy">2.4 Microscopy</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#light-microscopy">2.4.1 Light Microscopy</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#electron-microscopy">2.4.2 Electron Microscopy</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intracellular-structures">2.5 Intracellular Structures</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cytosol">2.5.1 Cytosol</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#nucleoid">2.5.2 Nucleoid</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-denatured-lysozyme">Case Study: Denatured Lysozyme</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <div class="tex2jax_ignore mathjax_ignore section" id="chapter-2-microbial-cell-structures">
<h1>Chapter 2: Microbial Cell Structures<a class="headerlink" href="#chapter-2-microbial-cell-structures" title="Permalink to this heading">#</a></h1>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Learning Objectives</strong></p>
<p class="sd-card-text">By the end of this chapter, students will be expected to:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">use correct terminology to describe microbial cell structures</p></li>
<li><p class="sd-card-text">describe the cell surface structure of a model bacterium</p></li>
<li><p class="sd-card-text">distinguish between Gram-negative and Gram-positive bacteria</p></li>
<li><p class="sd-card-text">interpret microscopy and Gram stain data</p></li>
<li><p class="sd-card-text">use data to predict what type of cell it could be or what its surface structure consists of</p></li>
<li><p class="sd-card-text">describe how the chromosomal DNA is package in prokaryotic cells</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="cell-shape">
<h2>2.1 Cell Shape<a class="headerlink" href="#cell-shape" title="Permalink to this heading">#</a></h2>
<div class="section" id="individual-cell-shapes">
<h3>2.1.1 Individual cell shapes<a class="headerlink" href="#individual-cell-shapes" title="Permalink to this heading">#</a></h3>
<p>The shapes of most bacterial cells are described using terms derived from Greek or Latin words. In most cases, it is not clear why different prokaryotic cells have different shapes. The significance of cell shape is a very difficult question to address experimentally.</p>
<p>The two most common bacterial cell shapes include <strong>coccus (singular) or cocci (plural)</strong> which are spherical-shaped bacteria and <strong>bacillus (sing.) or bacilli (plur.)</strong> which are rod-shaped bacteria. <em>Escherichia coli</em>, being a model bacterial species in the field of microbiology, is bacillus shaped. Fig 13 shows some of the most common microbial shapes.</p>
<div class="figure align-default" id="fig13">
<a class="reference internal image-reference" href="_images/2-1.png"><img alt="_images/2-1.png" src="_images/2-1.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 13 </span><span class="caption-text">Common microbial shapes and their proper nomenclature.</span><a class="headerlink" href="#fig13" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="group-cell-shapes">
<h3>2.1.2 Group cell shapes<a class="headerlink" href="#group-cell-shapes" title="Permalink to this heading">#</a></h3>
<p>Almost all prokaryotes are unicellular organisms. What about multicellular prokaryotes? Deciding whether a prokaryote is a multicellular organism can be difficult. Depending on cell division planes and the incomplete separation of cells, chains (<strong>filaments</strong>) or <strong>clusters</strong> of cells result in phenotypes seen in genera such as Staphylococcus and Streptococcus (Fig 14). Biologists and microbiologists may disagree on the specific definition of <strong>multicellularity</strong> and little is known about why certain species grow into “multicellular” shapes.</p>
<div class="figure align-default" id="fig14">
<a class="reference internal image-reference" href="_images/2-2.png"><img alt="_images/2-2.png" src="_images/2-2.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 14 </span><span class="caption-text">Microbial growth of “multicellular” bacterial species.</span><a class="headerlink" href="#fig14" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="extracellular-structures">
<h2>2.2 Extracellular Structures<a class="headerlink" href="#extracellular-structures" title="Permalink to this heading">#</a></h2>
<p>In this next section, we want to cover the outer “leaflets” of Bacteria in terms of composition and function starting from the outer most layer to the inner most. Note that although we introduce common features of these structures, the diversity of species within the Bacteria domain is quite vast, and these structures may be very different or non-existent in some species. We want to unravel 4 leaflets (sec 2.2.1-2.2.3) as shown in Fig 15 not including exterior appendages like the flagella or pili (sec 2.2.4).</p>
<div class="figure align-default" id="fig15">
<a class="reference internal image-reference" href="_images/2-3.png"><img alt="_images/2-3.png" src="_images/2-3.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 15 </span><span class="caption-text">Exterior leaflets of a Gram-positive bacterial cell.</span><a class="headerlink" href="#fig15" title="Permalink to this image">#</a></p>
</div>
<div class="section" id="capsule">
<h3>2.2.1 Capsule<a class="headerlink" href="#capsule" title="Permalink to this heading">#</a></h3>
<p>If a bacterial species possesses a <strong>capsule</strong>, it covers all underlying surface structures except for protein  appendages such as flagella (tail structure) or pili (hair-like structures) (sec 2.2.4).</p>
<p>Chemically, bacterial capsules fall into two main types:</p>
<ul class="simple">
<li><p>Polysaccharide capsules: chains of sugars, often a single repeated sugar.</p></li>
<li><p>Polypeptide capsules: chains of amino acids, often a single repeated amino acid.</p></li>
</ul>
<p>This being said, there are also examples of capsules composed of both sugars and amino acids.</p>
<p><strong>Function(s)</strong>
Depending on the species, capsules can have a variety of functions.</p>
<p><em>Protection from desiccation (drying)</em>
Capsules are composed of sugars (which possess many polar -OH groups) or amino acids with side chains possessing polar groups. This means both types of molecules are capable of forming H-bonds with water. Indeed, 95% of the weight of isolated capsular material can be due to water.</p>
<p><em>Protection from phagocytosis</em>
Phagocytosis is the process by which immune cells engulf foreign particles. It is not clear why, but encapsulated organisms are not easily engulfed by phagocytic cells such as amoebae or white blood cells.</p>
<p><em>Attachment to surfaces</em>
Attachment to surfaces can be beneficial for a number of reasons.</p>
<ul class="simple">
<li><p>As chemicals, nutrients frequently concentrate at solid-liquid interfaces so attachment to a surface can help protect an organism from starvation.</p></li>
<li><p>Attachment maintains an organism “in place” and prevents it from being “washed away”. For example, attachment to a lumenal surface allows bacteria to resist being swept out of an animal body by flow of saliva, urine, intestinal contents etc. Similarly, attachment can prevent organisms from being washed away from the vicinity of other types of surfaces, eg. attachment to a rock in a fast-flowing stream.</p></li>
<li><p>Organisms attached to surfaces are more resistant to predation, eg. by phagocytic organisms.</p></li>
</ul>
<p><em>Protection from viruses</em>
The viruses that infect prokaryotes (like bacteriophages) have difficulty reaching the cell wall to cause an infection if the cell is covered by a capsule.</p>
</div>
<div class="section" id="cell-wall">
<h3>2.2.2 Cell Wall<a class="headerlink" href="#cell-wall" title="Permalink to this heading">#</a></h3>
<p>A major component in the bacterial cell wall is <strong>peptidoglycan (PG)</strong>.</p>
<p>Peptidoglycan	PG is one, huge, mesh-like molecule composed of two parts: a glycan chain and a tetrapeptide. The glycan chain is made of two amino sugars: N-acetylglucosamine (NAG or G) and N-acetylmuramic acid (NAM or M) joined by strong covalent bonds in O-glycosidic linkages (Fig 16). Peptidoglycan structure has been mostly studied in a few rod-shaped bacteria. In these organisms, the glycan chains are oriented approximately perpendicular to the long axis of the cell. Individual glycan chains do not run all the way around the cell circumference, rather they are present as short fragments of various lengths.</p>
<div class="figure align-default" id="fig16">
<a class="reference internal image-reference" href="Figures/Unit_1/2-4.png"><img alt="Figures/Unit_1/2-4.png" src="Figures/Unit_1/2-4.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 16 </span><span class="caption-text">Peptidoglycan as a key component of bacterial cell walls.</span><a class="headerlink" href="#fig16" title="Permalink to this image">#</a></p>
</div>
<p><strong>Function(s)</strong>
Most bacteria live in <strong>hypotonic</strong> environments, i.e. the solute concentration of the environment is lower than the solute concentration of the cytoplasm. Under these conditions, water spontaneously flows across the phospholipid bilayer from the environment into the cytoplasm (= <strong>osmosis</strong>). This situation combined with the fact that the molecules making-up the cytoplasmic membrane are not covalently bonded together means that a bacterial cell in a hypotonic environment can potentially swell and burst due to the osmotic inflow of water (= <strong>osmotic lysis</strong>).</p>
<p>The function of the peptidoglycan is to protect bacterial cells from osmotic lysis in hypotonic environments. When water enters a bacterial cell, tremendous pressure is exerted on the peptidoglycan as the cytoplasmic compartment increases in volume and the cytoplasmic membrane is pressed against it. Indeed, about 5-20 atmospheres of pressure are generated, a pressure similar to that in an inflated bicycle tire. Nevertheless, the cytoplasmic membrane does not normally rupture. This is because the covalently-bonded, mesh-like nature of the peptidoglycan has great mechanical strength allowing it to withstand the pressure. Peptidoglycan is not completely rigid however; it is elastic and expands slightly as the cell swells.</p>
</div>
<div class="section" id="cytoplasmic-or-plasma-membrane">
<h3>2.2.3 Cytoplasmic or Plasma Membrane<a class="headerlink" href="#cytoplasmic-or-plasma-membrane" title="Permalink to this heading">#</a></h3>
<p>Like the eukaryal plasma membrane, the bacterial cytoplasmic membrane or plasma membrane is a bilayered structure about 10 nm thick composed of <strong>phospholipids</strong> and protein molecules (Fig 17).</p>
<p>Each half of a lipid bilayer is called a leaflet: the outer leaflet of the bilayer usually contacts the environment while the inner leaflet usually contacts the cytoplasm. In each leaflet, the heads of the phospholipids form the surface of the bilayer while the tails form the interior. Embedded in the lipid bilayer are protein molecules which either span the entire bilayer or associate with one or the other of the leaflets. The protein:lipid ratio of the membrane is about 50:50 by weight, but because lipids are much smaller than proteins, lipid molecules outnumber protein molecules by about 30 to 1.</p>
<div class="figure align-default" id="fig17">
<a class="reference internal image-reference" href="_images/2-5.png"><img alt="_images/2-5.png" src="_images/2-5.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 17 </span><span class="caption-text">Plasma membrane composed of phospholipids and proteins.</span><a class="headerlink" href="#fig17" title="Permalink to this image">#</a></p>
</div>
<p><strong>Function(s)</strong>
Like in eukaryotic cells, the plasma membrane is primarily responsible for controlling what goes in and out of the cell. This is highly dependent on <strong>permeability</strong> of the molecule.</p>
<p>Because there are no strong covalent bonds between the lipids and proteins of the cytoplasmic membrane, relatively weak thermally-induced molecular collisions (= <strong>Brownian motion</strong>) occasionally open small gaps between the lipid molecules for extremely short periods of time. In other words, because phospholipids are in constant thermal motion, membranes are not brick walls, they are fluid.</p>
<p>When gaps open between the phospholipids, chemicals can move across (= permeate) the bilayer by diffusion. Diffusion is a random walk process. A chemical may enter the bilayer and as a result of thermal collisions “bump” its way through the bilayer to the other side. Alternatively, as result of random collisions, it may “turn around” and end-up on the same side of the bilayer it started on.</p>
<p>When there is initially a difference in the concentration of a chemical on two sides of a lipid bilayer, chemicals are said to move “down” the concentration gradient from a region of high concentration to a region of low concentration. The phenomenon of moving down a concentration gradient is the result of simultaneous diffusion of chemicals in both directions across the bilayer. Several factors affect the rate at which a chemical diffuses across a phospholipid bilayer. These include size, polar/nonpolar content and charge; the latter two factors are related to water solubility.</p>
</div>
<div class="section" id="appendages">
<h3>2.2.4 Appendages<a class="headerlink" href="#appendages" title="Permalink to this heading">#</a></h3>
<p>Many (<em>not all</em>) prokaryotes are capable of locomotion. The three most common forms of motility are:</p>
<ul class="simple">
<li><p><strong>Swimming</strong>: Movement while suspended in an aqueous environment.</p></li>
<li><p><strong>Twitching</strong>: Jerky movement along a surface.</p></li>
<li><p><strong>Swarming</strong>: Group-based motility that forms unique patterns on a surface.
These forms of motility are often dependent on external appendages such as the tail-like structure, the <strong>flagellum (sing.)</strong> or hair-like structure called <strong>pilus (sing.)</strong>.</p></li>
</ul>
<div class="section" id="flagella">
<h4>Flagella<a class="headerlink" href="#flagella" title="Permalink to this heading">#</a></h4>
<p>The word flagellum is the Latin word for “whip”. The term was originally applied to eukaryal flagella (plur.) which do work by whipping back and forth. However, the filament of the bacterial flagellum is rigid and rotates like a propeller and does not exert a whipping motion like an animal tail.</p>
<p>Bacterial flagella are composed of several different proteins. The flagellar filament is a rigid helical structure about 20 nm in diameter composed of many copies of a single major protein called <strong>flagellin</strong>. The filament is attached via a hook protein to a rod protein which is embedded in the membrane and cell wall by several protein rings (= the basal structure) called the <strong>flagellar motor</strong> (Fig 18).</p>
<div class="figure align-default" id="fig18">
<a class="reference internal image-reference" href="_images/2-6.png"><img alt="_images/2-6.png" src="_images/2-6.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 18 </span><span class="caption-text">Bacterial flagellum and pili appendages.</span><a class="headerlink" href="#fig18" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="pili">
<h4>Pili<a class="headerlink" href="#pili" title="Permalink to this heading">#</a></h4>
<p>Pili (plur.) sometimes called <strong>fimbriae (singular fimbra)</strong> is the name given to certain protein projections from the cell surface (pilus = Latin for “hair”) (Fig 18).</p>
<p>Pili are very diverse. They vary in length (from less than a cell diameter to
many cell diameters), in thickness and in number (from a few to 100s per cell). This structural diversity is linked to their functional diversity.</p>
<p>Twitching motility is the jerky movement of a prokaryote along a surface mediated by the extension and retraction of structure called a <strong>Type IV pilus</strong>. It has been observed in some Proteobacteria, Firmicutes and Cyanobacteria.</p>
</div>
</div>
</div>
<div class="section" id="gram-stain">
<h2>2.3 Gram Stain<a class="headerlink" href="#gram-stain" title="Permalink to this heading">#</a></h2>
<p>Let’s revisit the cell wall (ie. peptidoglycan) and plasma membrane structures. In this chapter, we first introduced the plasma membrane as a bilayer that usually involves one side facing out of the cell and one side facing inside, but we used the word “usually” because for a large group of Bacteria, this may only be the case for one of their plasma membranes. Does this mean that some Bacteria have two plasma membranes? Yes, it does. Bacteria can be grouped into two major categories depending on their cell wall to plasma membrane orientation, <strong>Gram-positive or Gram-negative</strong>. In the lab, when identifying a new or unknown species of Bacteria, one of the first steps researchers take is to perform a Gram-stain to determine which category the unknown species falls into.</p>
<p>The Gram-stain is a procedure developed in the 1890s by Christian Gram. It involves sequential staining of bacterial cells with two stains, one purple and one red: Gram- positive cells stain purple while Gram-negative cells stain pink/red (1). At the time of its development, it was not known why different bacteria stained differently. In the mid 20th century it was discovered that the particular staining reaction was linked to the cell wall structure in some Bacteria.</p>
<div class="section" id="gram-positive-cells">
<h3>2.3.1 Gram Positive Cells<a class="headerlink" href="#gram-positive-cells" title="Permalink to this heading">#</a></h3>
<p>The Gram-positive cell wall consists of relatively thick layer of PG in between the capsule layer, if the species has this, and the plasma membrane (Fig 19). If you studies plant cell structures, this mirrors how those cells are typically structured with a thick cell wall surrounding the plasma membrane.</p>
<p>In addition to PG, Gram-positive cell walls contain polymers of phosphorylated sugar alcohols called <strong>teichoic acids</strong>. If the teichoic acid is anchored to NAM, it is called teichoic acid. If it is anchored to a sugar in the headgroup of a cell membrane phospho- or glycolipid, it is called lipoteichoic acid (LTA) (Fig 2.7). Both types of teichoic acids “poke” through the PG mesh and are exposed on cell surface extending many nanometers into the environment. The function(s) of teichoic acids are not well-understood.</p>
<div class="figure align-default" id="fig19">
<a class="reference internal image-reference" href="_images/2-7.png"><img alt="_images/2-7.png" src="_images/2-7.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 19 </span><span class="caption-text">Organization of the Gram-positive cell wall</span><a class="headerlink" href="#fig19" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="gram-negative-cells">
<h3>2.3.2 Gram Negative Cells<a class="headerlink" href="#gram-negative-cells" title="Permalink to this heading">#</a></h3>
<p>Unlike the Gram-positive bacteria, Gram-negative bacteria have a relatively thin PG layer (ie. cell wall) but to compensate, they have two plasma membranes! One layer sits on the exterior of the cell wall, outer membrane (OM), and one in the interior, inner membrane (IM) (Fig 20).</p>
<p>Like the cytoplasmic membrane, the outer membrane is a lipid bilayer structure about same thickness as the cytoplasmic membrane with embedded proteins that perform a variety of functions.</p>
<p>In Gram-negative Bacteria, the PG and a specific, abundant OM protein interact in a noncovalent manner. This anchors the outer membrane to the peptidoglycan forming the cell wall unit. In some bacteria, an additional protein called <strong>lipoprotein</strong> also functions to anchor the PG to the OM. As its name suggests, lipoprotein is a protein with hydrocarbon tails; the hydrocarbon tails are embedded in outer membrane while the protein portion is covalently bonded to the peptidoglycan.</p>
<p>In terms of composition, the OM exhibits considerable asymmetry. While the inner leaflet is composed of phospholipid, the outer leaflet is dominated by an unique lipid called <strong>lipopolysaccharide (LPS)</strong> (2) (Fig 20). In fact, depending on the bacterium the entire outer leaflet may consist of LPS with phospholipid lacking altogether. As the name suggests, LPS exhibits features of both lipids and polysaccharides. The LPS has been shown to act as a protective layer around the bacteria, preventing the entry of toxins and antibiotics (2).</p>
<div class="figure align-default" id="fig20">
<a class="reference internal image-reference" href="_images/2-8.png"><img alt="_images/2-8.png" src="_images/2-8.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 20 </span><span class="caption-text">Organization of the Gram-negative cell wall</span><a class="headerlink" href="#fig20" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="how-the-procedure-works">
<h3>2.3.3 How the procedure works<a class="headerlink" href="#how-the-procedure-works" title="Permalink to this heading">#</a></h3>
<p>The Gram-stain procedure (Fig 21) relies on differences in the Gram-positive and negative cell walls to stain cells differently (1). Remember that the Gram-positive cell has a much thicker PG (i.e. cell wall) layer whereas the Gram-negative cell has a thinner cell wall but has two plasma membranes (outer and inner).</p>
<div class="figure align-default" id="fig21">
<img alt="_images/2-9.png" src="_images/2-9.png" />
<p class="caption"><span class="caption-number">Fig. 21 </span><span class="caption-text">Organization of the Gram-negative cell wall</span><a class="headerlink" href="#fig21" title="Permalink to this image">#</a></p>
</div>
<ol class="arabic simple">
<li><p>First stain. The first step is stain both types of cells purple using a dye called crystal violet. The dye particles are small enough to penetrate the cell walls of both Gram-positive and Gram-negative cells. At this stage, all cells are stained purple (Fig 21)</p></li>
</ol>
<div class="margin sidebar">
<p class="sidebar-title">Missing the iodine step</p>
<p>An extra step between step 1 and 2 has been purposely omitted. You may encounter it again when you actually do this experiment. As part of this procedure, the second step is to soak the cells in iodine to trap the dye in the cells. For the purposes of this course, you do not need to know how this step works.</p>
</div>
<ol class="arabic simple" start="2">
<li><p>De-stain. The next few steps involve a very quick wash using alcohol. Alcohol is pungent enough to break the cell wall structure down and create pores or holes on the cell wall. Due to the quick treatment and the relatively thick cell wall that Gram-positive cells have, they are not usually significantly affected by this. On the other hand, the Gram-negative cells with a thinner cell wall develop gaps in its cell wall causing the purple dye to wash out.</p></li>
<li><p>Counter stain. Because a non-stained cell is not easily visualized under a microscope. A counter stain called safranin, pink in color, is added to the cells to stain the Gram-negative cells which have now lost the primary stain. Therefore, Gram-positive cells appear purple and Gram-negative cells appear pink (Fig 21).</p></li>
</ol>
</div>
</div>
<div class="section" id="microscopy">
<h2>2.4 Microscopy<a class="headerlink" href="#microscopy" title="Permalink to this heading">#</a></h2>
<div class="section" id="light-microscopy">
<h3>2.4.1 Light Microscopy<a class="headerlink" href="#light-microscopy" title="Permalink to this heading">#</a></h3>
<p>This chapter covers a lot of cell structures but how do scientists go about discovering and recording all these structures? A key technique that goes as far back as the origins of Microbiology is <strong>microscopy</strong>, the technique of using a microscope to capture enlarged images of microscopic structures. The first ever microscope, invented by Antonie Leeuwenhoek, was a <strong>light microscope</strong> (3). The light microscope (Fig 2.10) uses a series of magnifying lenses and light to visualize the specimen. Light microscopy is limited by the size of the specimen it can visualize and often the image is not as high a resolution as other types of microscopy. The advantage of light microscopy is that it can allow you to visualize live specimen.</p>
<div class="figure align-default" id="fig22">
<img alt="_images/2-10.png" src="_images/2-10.png" />
<p class="caption"><span class="caption-number">Fig. 22 </span><span class="caption-text">Compound (light) microscope.</span><a class="headerlink" href="#fig22" title="Permalink to this image">#</a></p>
</div>
<p>Depending on the type of microscope, light micrographs (for example, Fig 23) can:</p>
<ul class="simple">
<li><p>Reveal the shape of bacterial cells but often cannot resolve appendages like flagella or pili</p></li>
<li><p>Have a resolving power of up to 100nm (<em>E. coli</em> is typically 1-2um)</p></li>
<li><p>appear in color (either due to dyes or fluorescence)</p></li>
<li><p>visualize live, active cells</p></li>
</ul>
<div class="figure align-default" id="fig23">
<img alt="_images/2-11.png" src="_images/2-11.png" />
<p class="caption"><span class="caption-number">Fig. 23 </span><span class="caption-text">Gram stain of <em>E. coli</em> (left) and <em>Staphylococcus aureus</em>  (right) using a light microscope.</span><a class="headerlink" href="#fig23" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="electron-microscopy">
<h3>2.4.2 Electron Microscopy<a class="headerlink" href="#electron-microscopy" title="Permalink to this heading">#</a></h3>
<p>Contrary to light microscopy, <strong>electron microscopy</strong> uses electrons instead of light to visualize the specimen (Fig 24). It relies on a computer that reads out the energy omitted by electrons that pulse off of the specimen or through the specimen to create a high-resolution image. There are two types of electron microscopy: <strong>transmission electron microscopy (TEM) and scanning electron microscopy (SEM)</strong>. TEM is used to visualize intracellular structures by magnifying the specimen based on how electron pass through a thinly sliced sample. SEM, on the other hand, is used to visualize the surface of a cell by reading the energy discharge involved in electrons bouncing off the surface of the sample. Electron microscopy is capable of visualizing samples of much smaller specimen than light microscopy (Fig 25). It can generate high resolution 3D black and white renderings of the specimen. The caveat is that it can only be used for dead specimen and cannot be used to visualize live samples.</p>
<div class="figure align-default" id="fig24">
<a class="reference internal image-reference" href="_images/2-12.png"><img alt="_images/2-12.png" src="_images/2-12.png" style="height: 300px;" /></a>
<p class="caption"><span class="caption-number">Fig. 24 </span><span class="caption-text">Electron microscope.</span><a class="headerlink" href="#fig24" title="Permalink to this image">#</a></p>
</div>
<div class="figure align-default" id="fig25">
<a class="reference internal image-reference" href="_images/2-13.png"><img alt="_images/2-13.png" src="_images/2-13.png" style="height: 500px;" /></a>
<p class="caption"><span class="caption-number">Fig. 25 </span><span class="caption-text">Range of different a light versus electron microscope.</span><a class="headerlink" href="#fig25" title="Permalink to this image">#</a></p>
</div>
<p>Electron microscopy can resolve objects up to 0.1nm (an <em>E. coli</em> pilus is typically 0.3-1.0um)</p>
<p>There are two types of electron microscopy that will be introduced in this course. Examples of micrographs are shown in Fig 26.</p>
<ul class="simple">
<li><p>Transmission electron microscopy: imaging intracellular components of the cell. Micrographs show intracellular components or individual cell layers.</p></li>
<li><p>Scanning electron microscopy: imaging surface level structures. Micrographs often appear 3-dimensional and show surface topology.</p></li>
</ul>
<div class="figure align-default" id="fig26">
<img alt="_images/2-14.png" src="_images/2-14.png" />
<p class="caption"><span class="caption-number">Fig. 26 </span><span class="caption-text">Scanning electron micrograph (left) and transmission electron micrograph (4) (right) of <em>E. coli</em>.</span><a class="headerlink" href="#fig26" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="intracellular-structures">
<h2>2.5 Intracellular Structures<a class="headerlink" href="#intracellular-structures" title="Permalink to this heading">#</a></h2>
<p>While Bacteria, Archaea and Eukarya differ with respect to the details of their molecular properties and processes, they are nevertheless composed of the same basic kinds of molecules. This reflects the underlying biochemical unity of all cellular life.</p>
<p>Like eukaryal cells, water constitutes ~70% of the bacterial and archaeal cell wet weight. Indeed, most of the cytoplasm is water which serves a solvent for the biochemical reactions that occur there. The remaining dry weight of a prokaryotic cell is made-up various other chemical constituents.</p>
<p><strong>Proteins</strong> (and some RNAs) serve as the molecular machines of cells. Indeed, almost every process that occurs in a cell is mediated by a specific protein or RNA with a specific function. As components of cell walls, some proteins are also important structural molecules.</p>
<p><strong>DNA and mRNA</strong> serve as informational molecules. DNA carries the information which specifies the sequence of nucleotides in a RNA molecule which in the case of mRNA in turn carries the information that specifies the sequence of amino acids in a polypeptide and thus its function. Some RNAs also serve regulatory functions affecting gene expression.</p>
<p>As constituents of cell membranes and cell walls, <strong>lipids and polysaccharides</strong> serve mainly as the structural molecules of cells; they also used as nutritional storage materials (eg. fat, starch etc).</p>
<div class="section" id="cytosol">
<h3>2.5.1 Cytosol<a class="headerlink" href="#cytosol" title="Permalink to this heading">#</a></h3>
<p>The cytoplasm is both a concentrated solution of protein (principally enzymes) and a dense suspension of ribosomes; mRNA, tRNA, small molecules and inorganic ions are also present (Fig 27). There are typically tens of thousands of ribosomes per prokaryotic cell and this immense number of ribosomes gives the cytoplasm a granular appearance in thin section electron micrographs. The prominence of ribosomes is not too hard to understand.</p>
<p>Because protein accounts for about 50% of the cell dry weight, protein synthesis is the main biosynthetic task of most cells and ribosomes are the protein/rRNA- based, “molecular machines” responsible for protein synthesis.</p>
<div class="figure align-default" id="fig27">
<img alt="_images/2-15.png" src="_images/2-15.png" />
<p class="caption"><span class="caption-number">Fig. 27 </span><span class="caption-text">Intracellular composition.</span><a class="headerlink" href="#fig27" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="nucleoid">
<h3>2.5.2 Nucleoid<a class="headerlink" href="#nucleoid" title="Permalink to this heading">#</a></h3>
<p>The nucleoid is a structure composed principally of DNA and certain proteins. In most prokaryotes, the DNA component of the nucleoid is usually a single, circular DNA molecule called the organism’s <strong>chromosome</strong>. The nucleoid is not surrounded by a membrane, it is in direct contact with the other components of the cytoplasm.</p>
<p>Prokaryotic chromosomes are big relative to the dimensions of the cell they are contained in. For example, the circular chromosome of <em>E. coli</em> strain K-12 has a circumference of 1 mm, yet it is enclosed in a rod-shaped bacterium whose dimensions are only 1 µm by 3 µm. Look at Fig 28. Given this picture, it is perhaps not surprising that cellular rupture leads to the chromosome “exploding” out of its container. In order to “fit” in the cell, DNA needs to be tightly folded and compacted.</p>
<div class="figure align-default" id="fig28">
<a class="reference internal image-reference" href="_images/2-16.png"><img alt="_images/2-16.png" src="_images/2-16.png" style="height: 250px;" /></a>
<p class="caption"><span class="caption-number">Fig. 28 </span><span class="caption-text">DNA leaking out of a ruptured bacterial cell.</span><a class="headerlink" href="#fig28" title="Permalink to this image">#</a></p>
</div>
<p>Bacteria and Archaea synthesize enzymes that twist DNA into supercoils (and others that relax supercoils) (Fig 29). In <strong>supercoiled DNA</strong>, the double helix is itself helically twisted. Supercoiling helps to compact DNA</p>
<div class="figure align-default" id="fig29">
<img alt="_images/2-17.png" src="_images/2-17.png" />
<p class="caption"><span class="caption-number">Fig. 29 </span><span class="caption-text">DNA leaking out of a ruptured bacterial cell.</span><a class="headerlink" href="#fig29" title="Permalink to this image">#</a></p>
</div>
<p>If special methods are used to gently rupture a bacterial cell, some native nucleoid structure can be preserved; these nucleoid-like structures are called folded chromosomes. Folded chromosomes exhibit loops of supercoiled DNA. At the base of each loop are <strong>histone-like anchoring proteins</strong> named for their similarity to histones, the DNA-binding proteins of Eukarya and some Archaea. Histone-like proteins possess a high proportion of amino acids with positively-charged side chains which is important for binding to negatively-charged DNA to prevent the relaxation of supercoiled DNA loops.</p>
</div>
</div>
<div class="section" id="case-study-denatured-lysozyme">
<h2>Case Study: Denatured Lysozyme<a class="headerlink" href="#case-study-denatured-lysozyme" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Ibrahim et al., 1996</strong></p>
<p class="sd-card-text">Ibrahim, HR, Higashiguchi S, Koketsu M, Juneja LR, Kim M, Yamamoto T, Sugimoto Y, Aoki T. 1996. Partially unfolded lysozyme at neutral pH agglutinates and kills Gram-negative and Gram-positive bacteria through membrane damage mechanism. J. Agric. Food Chem. 44:3799-3806</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p>Ibrahim et al. were interested in studying the enzyme lysozyme. Lysozyme is an enzyme that can cleave the bonds in the NAM and NAG subunits of peptidoglycan, the major component that makes up the cell wall. Therefore, it was believed that lysozyme would be a good antimicrobial agent (i.e. it could kill bacteria and possibly archaea). Oddly enough, it was found that lysozyme had very poor antimicrobial activity. It was observed to be able to kill some Gram-positive bacteria but only at really high concentrations. However, Ibrahim et al. observed that lysozyme may be involved in helping our immune system fight bacterial infections. But the lysozyme found during those infections were all <strong>denatured</strong>, which means that the protein was not folded into the structure that we normally expect it to be in order for it to be active. Ibrahim et al. hypothesized that this denatured lysozyme would be more antimicrobial towards both Gram-positive and Gram-negative bacteria than the native form.</p>
<p>In order to test this hypothesis, Ibrahim et al. had to go through the following steps:</p>
<p><strong>Step 1: Confirm that the original observation is true</strong>
Ibrahim et al. first wanted to confirm that the native lysozyme was very poorly antimicrobial and that denatured showed good antimicrobial activity using electron microscopy. The collected the following data:</p>
<div class="figure align-default" id="fig30">
<img alt="_images/2-18.png" src="_images/2-18.png" />
<p class="caption"><span class="caption-number">Fig. 30 </span><span class="caption-text">Electron micrograph of <em>E. coli</em> cells treated with (A) 100ug/mL of native lysozyme, (B) 100ug/mL of denatured lysozyme, and (C) 200ug/mL of denatured lysozyme. (D) Zoomed out version of C. Denatured lysozyme was created by heating the enzyme at 80C for 6 hours.</span><a class="headerlink" href="#fig30" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What technique was used to collect the data?</p></li>
<li><p class="sd-card-text">Is this type of data qualitative or quantitative?</p></li>
<li><p class="sd-card-text">Look at the figure caption, what do each panel show? What are the treatment(s) and what is the control group?</p></li>
<li><p class="sd-card-text">Do the cells look healthy at each panel? What is strange about each treatment?</p></li>
<li><p class="sd-card-text">What is your initial impression of the data?</p></li>
<li><p class="sd-card-text">Why do you think the authors included panel D?</p></li>
<li><p class="sd-card-text">Does the data support the original observation that the native is poorly antimicrobial and the denatured is antimicrobial?</p></li>
<li><p class="sd-card-text">Do you think that controls or treatments are missing to make better comparisons? If so, what would you have included?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 2: Treating bacterial cells with native and denatured lysozyme</strong></p>
<div class="figure align-default" id="fig31">
<img alt="_images/2-19.png" src="_images/2-19.png" />
<p class="caption"><span class="caption-number">Fig. 31 </span><span class="caption-text">Effect of heated lysozyme at different pHs on bacterial growth (CFU per mL) of (A) <em>Staphylococcus aureus</em> and (B) <em>E. coli</em>. Unheated native lysozyme NLz (open symbols) and heated lysozyme HLz (solid symbols). The HLz was heated at 80C for 6 hours.</span><a class="headerlink" href="#fig31" title="Permalink to this image">#</a></p>
</div>
<div class="margin sidebar">
<p class="sidebar-title">Consider</p>
<p>The concept of measuring bacterial load (CFU/mL) will be introduced in Unit 1 Chap 3.</p>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Is <em>E. coli</em> a Gram+ or Gram- species?
Is <em>S. aureus</em> a Gram+ or Gram- species?</p>
<p>Depending on what type of Gram stain each species has, what does that say about their surface structure? What do you expect in terms of resistance or susceptibility to lysozyme being an enzyme that digests peptidoglycan?</p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What is the y-axis showing? What is the x-axis showing?</p></li>
<li><p class="sd-card-text">Is this type of data qualitative or quantitative?</p></li>
<li><p class="sd-card-text">What is the treatment and what is the control group?</p></li>
<li><p class="sd-card-text">What is the difference between panel A and B?</p></li>
<li><p class="sd-card-text">Looking at panel A, what is your impression on how antimicrobial the native and denatured lysozyme are? Is one better than the other? Is it pH dependent?</p></li>
<li><p class="sd-card-text">In panel A, how do you know if the bacteria are growing well or not? What is a potential control that you can include to make this easier to interpret?</p></li>
<li><p class="sd-card-text">Looking at panel B, what is your impression on how antimicrobial the native and denatured lysozyme are? Is one better than the other? Is it pH dependent?</p></li>
<li><p class="sd-card-text">What can you conclude about this data?</p></li>
<li><p class="sd-card-text">Does this data align with authors’ hypothesis? Why or why not?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 3: Testing a potential mechanism of action</strong></p>
<p>Ibrahim et al. wanted to determine if the effects seen in Figure 31 were due to lysozyme’s natural lytic activity (i.e. its ability to cleave the cell wall and cause the cell to burst). They collected the following data.</p>
<div class="figure align-default" id="fig32">
<img alt="_images/2-20.png" src="_images/2-20.png" />
<p class="caption"><span class="caption-number">Fig. 32 </span><span class="caption-text">Effect of heating temperature at pH 6.0 on (A) enzymatic activity and (B) antimicrobial activity of lysozyme against S. aureus and <em>E. coli</em>. (B) Lysozyme was pre-heat at the specified temperature then added to the bacterial cultures. The bacterial cultures were not subjected to the extreme temperatures.</span><a class="headerlink" href="#fig32" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What are the y-axes showing? What is the x-axis showing?</p></li>
<li><p class="sd-card-text">Panel A: What is happening to lytic activity as you heat lysozyme?
a.	At what temperature does activity start to decline?
b.	At what temperature does the enzyme completely lose its lytic activity?</p></li>
<li><p class="sd-card-text">Panel B: What has happened to S. aureus? Does this align with the data in Figure 2?</p></li>
<li><p class="sd-card-text">Panel B: What is happening to E. coli as you add lysozyme subjected to increasing heat?</p></li>
<li><p class="sd-card-text">How does the trend between panel A and B compare?</p></li>
<li><p class="sd-card-text">Based on the collective data, does lytic activity promote increased antimicrobial activity?</p></li>
</ol>
</div>
</div>
</div>
</div>
<div class="section" id="references">
<h2>References<a class="headerlink" href="#references" title="Permalink to this heading">#</a></h2>
<ol class="arabic simple">
<li><p>Tripathi, N. &amp; Sapra, A. Gram Staining. StatPearls [Internet] (StatPearls Publishing, 2021).</p></li>
<li><p>Bertani, B. &amp; Ruiz, N. Function and biogenesis of lipopolysaccharides. EcoSal Plus 8, 10.1128/ecosalplus.ESP-0001–2018 (2018).</p></li>
<li><p>Antonie van Leeuwenhoek | Biography, Discoveries, &amp; Facts | Britannica. <a class="reference external" href="https://www.britannica.com/biography/Antonie-van-Leeuwenhoek">https://www.britannica.com/biography/Antonie-van-Leeuwenhoek</a>.</p></li>
<li><p>Zea, L. et al. Phenotypic Changes Exhibited by E. coli Cultured in Space. Front. Microbiol. 8, (2017).</p></li>
</ol>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="unit1-chap1.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Chapter 1: Introduction to Microbiology and Data Analysis</p>
      </div>
    </a>
    <a class="right-next"
       href="unit1-chap3.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 3: Microbial Growth in the Lab and in Nature</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-shape">2.1 Cell Shape</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#individual-cell-shapes">2.1.1 Individual cell shapes</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#group-cell-shapes">2.1.2 Group cell shapes</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#extracellular-structures">2.2 Extracellular Structures</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#capsule">2.2.1 Capsule</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cell-wall">2.2.2 Cell Wall</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cytoplasmic-or-plasma-membrane">2.2.3 Cytoplasmic or Plasma Membrane</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#appendages">2.2.4 Appendages</a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#flagella">Flagella</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#pili">Pili</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-stain">2.3 Gram Stain</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-positive-cells">2.3.1 Gram Positive Cells</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#gram-negative-cells">2.3.2 Gram Negative Cells</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#how-the-procedure-works">2.3.3 How the procedure works</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#microscopy">2.4 Microscopy</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#light-microscopy">2.4.1 Light Microscopy</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#electron-microscopy">2.4.2 Electron Microscopy</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#intracellular-structures">2.5 Intracellular Structures</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#cytosol">2.5.1 Cytosol</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#nucleoid">2.5.2 Nucleoid</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-denatured-lysozyme">Case Study: Denatured Lysozyme</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By MICB 211 Teaching Team led by Evelyn Sun
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2022.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>