

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Chapter 2: Community Lifestyle of a Biofilm &#8212; MICB 211 Foundations in Microbiology eTextbook</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'unit2-chap2';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Chapter 3: Bacterial Pathogenesis and Virulence" href="unit2-chap3.html" />
    <link rel="prev" title="Chapter 1: The Human Microbiome" href="unit2-chap1.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="intro.html">
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="intro.html">
                    MICB 211 E-Textbook Acknowledgements
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 1</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit1-chap1.html">Chapter 1: Introduction to Microbiology and Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap2.html">Chapter 2: Microbial Cell Structures</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap3.html">Chapter 3: Microbial Growth in the Lab and in Nature</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap4.html">Chapter 4: Gene Expression and Regulation</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap5.html">Chapter 5: Sensing and Responding to the Environment</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 2</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit2-chap1.html">Chapter 1: The Human Microbiome</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Chapter 2: Community Lifestyle of a Biofilm</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap3.html">Chapter 3: Bacterial Pathogenesis and Virulence</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Funit2-chap2.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/unit2-chap2.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Chapter 2: Community Lifestyle of a Biofilm</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bacteria-and-biofilms">2.1 Bacteria and Biofilms</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-a-biofilm">2.1.1 What is a Biofilm?</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#prevalence-of-biofilms">2.1.2 Prevalence of biofilms</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biofilms-in-nature-and-in-human-disease">2.1.3 Biofilms in nature and in human disease</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#treating-biofilms">2.2 Treating Biofilms</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#antimicrobial-resistance-and-persistence-of-biofilms">2.2.1 Antimicrobial resistance and persistence of biofilms</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#new-pharmaceutical-approaches-to-anti-biofilm-therapy">2.2.2 New pharmaceutical approaches to anti-biofilm therapy</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biofilm-regulatory-systems">2.2.3 Biofilm regulatory systems</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-investigating-biofilm-treatment-and-modes-of-study">Case Study: Investigating biofilm treatment and modes of study</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quorum-sensing">2.3 Quorum Sensing</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#quorum-sensing-in-e-coli">2.3.1 Quorum sensing in <em>E. coli</em></a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-motility-and-chemotaxis">2.4 Bacterial Motility and Chemotaxis</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotaxis-vs-biofilm-formation">2.4.1 Chemotaxis vs Biofilm Formation</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#sensing">2.4.2 Sensing</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#responding">2.4.3 Responding</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotaxis-in-e-coli">2.4.4 Chemotaxis in <em>E. coli</em></a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#tumbling-motion">Tumbling Motion</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#running-motion">Running Motion</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-autoaggregation">Case Study: Autoaggregation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <div class="tex2jax_ignore mathjax_ignore section" id="chapter-2-community-lifestyle-of-a-biofilm">
<h1>Chapter 2: Community Lifestyle of a Biofilm<a class="headerlink" href="#chapter-2-community-lifestyle-of-a-biofilm" title="Permalink to this heading">#</a></h1>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Learning Objectives</strong></p>
<p class="sd-card-text">By the end of this chapter, students will be expected to:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">describe characteristics of a biofilm and how it typically forms</p></li>
<li><p class="sd-card-text">distinguish between biofilms and planktonic cultures</p></li>
<li><p class="sd-card-text">describe features of biofilms that contribute to persistence and antimicrobial resistance</p></li>
<li><p class="sd-card-text">describe the molecular mechanisms of quorum sensing and chemotaxis in <em>E. coli</em></p></li>
<li><p class="sd-card-text">Apply your understanding of the <em>E. coli</em> quorum sensing and chemotaxis systems to other bacterial species</p></li>
<li><p class="sd-card-text">explain the role of quorum sensing and chemotaxis in biofilm formation</p></li>
<li><p class="sd-card-text">interpret data collected on biofilms, quorum sensing, and chemotaxis</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="bacteria-and-biofilms">
<h2>2.1 Bacteria and Biofilms<a class="headerlink" href="#bacteria-and-biofilms" title="Permalink to this heading">#</a></h2>
<div class="section" id="what-is-a-biofilm">
<h3>2.1.1 What is a Biofilm?<a class="headerlink" href="#what-is-a-biofilm" title="Permalink to this heading">#</a></h3>
<p><strong>Biofilms</strong> are organized communities of bacteria that are adhered to a surface and surrounded by (enmeshed in) a matrix of <strong>extracellular polymeric substances (EPS)</strong>. The EPS is a slimy, film-like substance produced by the bacteria in the biofilm, and is composed of polysaccharides, proteins and nucleic acids.  A biofilm is often composed of a single bacterial species.  However, some biofilms have been found to contain hundreds of different species of bacteria living together (1).</p>
<p>Biofilm formation occurs as a result of coordinated chemical signaling between cells (called <strong>quorum sensing</strong>) (2). Quorum sensing uses signaling molecules, called <strong>autoinducers</strong>, which are produced by bacteria. The bacteria also possess receptors for these signaling molecules. When elevated numbers of bacteria are present in close proximity to each other, autoinducer concentration increases. Once the binding of autoinducer molecules has exceeded a certain threshold, a signaling cascade is initiated that modulates gene expression and in turn, modulates bacterial physiology to encourage community development. We will explore this in more depth in section 2.3.</p>
<div class="figure align-default" id="fig95">
<img alt="_images/2-110.png" src="_images/2-110.png" />
<p class="caption"><span class="caption-number">Fig. 95 </span><span class="caption-text">Formation of a biofilm.  Biofilm communities can develop within hours. The detachment stage occurs through the loss of clumps of bacteria and biofilm or through the release of individual cells. This allows for propagation of new biofilms at near or distant surfaces (2).</span><a class="headerlink" href="#fig95" title="Permalink to this image">#</a></p>
</div>
<p>The physical biofilm formation process involves:</p>
<ol class="arabic simple">
<li><p>Adherence of bacteria to a surface</p></li>
<li><p>Growth and production of EPS, resulting in much stronger (often irreversible) attachment leading to biofilm maturation.</p></li>
<li><p>Dispersion of single cells (or parts of the biofilm) from the site of a <strong>mature biofilm</strong> (Fig 95).</p></li>
</ol>
<p>The term <strong>planktonic</strong> refers to single bacterial cells that are growing in liquid (i.e., standard laboratory) culture, or to single bacterial cells swimming freely above a biofilm.  Planktonic bacteria can be recruited to a biofilm or may have “seeded” off of an existing biofilm.  Bacteria in a biofilm behave as a group, meaning they sense and respond to stimuli in a coordinated manner.  Some biofilm structures include a network of pores, held together by EPS, that provides a primitive circulatory system. Due to limitations in <span class="math notranslate nohighlight">\(O_2\)</span> and nutrients, bacteria deep inside the biofilm will have a different physiology than the bacteria closer to the surface.</p>
</div>
<div class="section" id="prevalence-of-biofilms">
<h3>2.1.2 Prevalence of biofilms<a class="headerlink" href="#prevalence-of-biofilms" title="Permalink to this heading">#</a></h3>
<p>A review in the Journal of Bacteriology had the following opening paragraph: <em>‘For over a century, microbiologists have studied liquid cultures of bacteria.  In fact, a common criterion for choosing a microorganism to study has been its ability to grow in a suspended, homogeneous culture format, thereby simplifying examination of microbial physiology and genetics.  Although these studies have been tremendously informative, they neglect the observation that many bacteria in the natural environment grow aggregated with each other, with solid surfaces, and at gas-liquid interfaces. There is a growing appreciation that, although clearly worthwhile, studies of standard planktonic cultures provide us with a biased view of microbial life. (3)’</em></p>
<p>It is impossible to know the percent of bacteria in the environment that exist as planktonic versus biofilm.  However, a recent (2017) estimate from the U.S. Centers for Disease Control and Prevention estimated that biofilms account for 65-80% of bacterial infections that physicians encounter (4)!</p>
</div>
<div class="section" id="biofilms-in-nature-and-in-human-disease">
<h3>2.1.3 Biofilms in nature and in human disease<a class="headerlink" href="#biofilms-in-nature-and-in-human-disease" title="Permalink to this heading">#</a></h3>
<p>Biofilms can be commonly found environmentally as well as in association with biological “hosts”. They can be classified as either benign or pathogenic.</p>
<p><strong>Benign environmental biofilms</strong> Examples include slippery coatings on rocks in streams, coatings on ship hulls, etc. Although these biofilms do not generally cause human disease, they are responsible for significant industrial problems such as corrosion.</p>
<p><strong>Pathogenic biofilms on medical implants.</strong> A variety of bacteria, both Gram-negative and Gram-positive, readily form biofilms on many foreign devices implanted into human bodies. These bacteria account for significant human infection and disease. Examples of affected medical devices include:</p>
<ul class="simple">
<li><p>Urinary catheters</p></li>
<li><p>Heart valve implants</p></li>
<li><p>Hemodialysis equipment</p></li>
<li><p>Dental implants</p></li>
</ul>
<p>Pathogenic biofilms may also form on human body surfaces. Bodily surfaces provide an ideal environment for the formation of many different classes of biofilms. Many of these infections are caused by opportunistic pathogens that are also human commensals. Often the biofilm infections result in chronic disease that is difficult to treat with antibiotics. Examples of such infections include:</p>
<ul class="simple">
<li><p>Cystic fibrosis (CF) infections in the lung, caused by <em>Pseudomonas aeruginosa</em></p></li>
<li><p>Tuberculosis infections caused by <em>Mycobacterium tuberculosis</em></p></li>
<li><p>Urinary tract infections caused by <em>Escherichia coli</em></p></li>
<li><p>Ear infections (variety of bacteria)</p></li>
<li><p>Tonsillitis (variety of bacteria)</p></li>
<li><p>Oral biofilms. These biofilms are the primary causative agents of dental caries and gingivitis and are probably the best-studied model of biofilm communities in terms of human infection. Hundreds of species have been identified in oral bacterial communities, which are often very complicated. For instance, secondary colonizers such as <em>Fusobacterium spp.</em> join the <em>Streptococcus spp</em>.</p></li>
</ul>
<p>Environmental biofilms may also impact human health. Some examples include the colonization of hot water systems in hospitals by <em>Mycobacterium avium</em>. Drinking water reservoirs may also be contaminated with <em>Salmonella typhimurium</em>. These ‘benign’ environmental biofilms can attract, recruit, and concentrate pathogenic bacteria that might not form a biofilm on their own. This has many implications, one of which is bioterrorism. For instance, biofilms on copper tubing have been shown to attract and concentrate <em>Bacillus anthracis</em> (anthrax) spores.</p>
</div>
</div>
<div class="section" id="treating-biofilms">
<h2>2.2 Treating Biofilms<a class="headerlink" href="#treating-biofilms" title="Permalink to this heading">#</a></h2>
<div class="section" id="antimicrobial-resistance-and-persistence-of-biofilms">
<h3>2.2.1 Antimicrobial resistance and persistence of biofilms<a class="headerlink" href="#antimicrobial-resistance-and-persistence-of-biofilms" title="Permalink to this heading">#</a></h3>
<p>A general characteristic of biofilm communities is that they can be significantly (i.e., &gt;10,000 times) more resistant to antibiotics and antimicrobial stressors, including those mounted by natural host responses (i.e., the immune system), compared to planktonic bacteria of the same species. Biofilm properties thought to contribute to antimicrobial resistance include:</p>
<ol class="arabic simple">
<li><p>The EPS matrix. The complex EPS layer enmeshing the biofilm can significantly impede the penetration of antimicrobial agents to bacteria buried in the depths of the biofilm.</p></li>
<li><p>Nutrient and O2 availability. Bacteria within a biofilm are subject to nutrient and O2 gradients, meaning bacteria closer to the surface have significantly better access to nutrients and O2 than bacteria on the interior. Consequently, bacteria in the biofilm interior, while alive, are often metabolically inactive. Antibiotics are most effective against metabolically active cells; thus, bacteria in the interior are protected from this type of killing.</p></li>
<li><p>‘Persister’ cells. Sometimes, even though the majority of a biofilm may be killed by antimicrobial therapy, a very small percentage of the population will remain viable despite prolonged exposure to the antimicrobial or even increased dosage. These persisters generally confer no heritable resistance to progeny once the selective pressure is removed; however, this characteristic allows cells in a biofilm to survive for extremely long periods of time, a feature which likely contributes to long-term survival of bacterial species in general.</p></li>
</ol>
</div>
<div class="section" id="new-pharmaceutical-approaches-to-anti-biofilm-therapy">
<h3>2.2.2 New pharmaceutical approaches to anti-biofilm therapy<a class="headerlink" href="#new-pharmaceutical-approaches-to-anti-biofilm-therapy" title="Permalink to this heading">#</a></h3>
<p>While this is a very new area of research, some strategies currently being investigated include:</p>
<ol class="arabic simple">
<li><p>Interfering with EPS synthesis. For instance, coating medical devices with chemicals that hinder matrix formation</p></li>
<li><p>Inhibiting adherence of biofilms to their surface substrate. Identifying chemicals that bind to bacterial cell surfaces will prevent formation of biofilms before they even begin</p></li>
<li><p>Targeting autoinducers. If the bacteria are unable to signal to each other, they will be unable to form or maintain a biofilm. Several strategies are underway to achieve this.</p></li>
</ol>
</div>
<div class="section" id="biofilm-regulatory-systems">
<h3>2.2.3 Biofilm regulatory systems<a class="headerlink" href="#biofilm-regulatory-systems" title="Permalink to this heading">#</a></h3>
<p>In order to form something as complex as a biofilm, bacteria rely on signal transduction systems. As was introduced in Unit 1 Chapter 5 with one- and two-component systems, a signal transduction system involves sensing environmental changes coupled with a response(s). During biofilm formation, two core signal transduction systems play a collective role in mediating biofilm formation, <strong>quorum sensing</strong> (sec 2.3) and <strong>chemotaxis</strong> (sec 2.4), which will be discussed in the next two sections. One of the things we hope that you appreciate is that although these systems may exist in many of the bacterial species that we have studied, how they work may vary greatly due to biological variation and diversity. However, in this course, you will learn how these systems work by studying the model organism, <em>E. coli</em>, as an extensible example of the core components of quorum sensing and chemotaxis.</p>
</div>
</div>
<div class="section" id="case-study-investigating-biofilm-treatment-and-modes-of-study">
<h2>Case Study: Investigating biofilm treatment and modes of study<a class="headerlink" href="#case-study-investigating-biofilm-treatment-and-modes-of-study" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Wang et al., 2021</strong></p>
<p class="sd-card-text">Wang Y, Hoffmann JP, Baker SM, Bentrup KHZ, Wimley WC, Fuselier JA, Bitoun JP, Morici LA.  2021. Inhibition of Streptococcus mutans biofilms with bacterial-derived outer membrane vesicles. BMC Microbiol. 21(1):234. doi: 10.1186/s12866-021-02296-x.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p>Biofilms are complex environments of bacteria and other microbes that require cooperation and communication. A major mode of interbacterial communication is through outer membrane vesicles (OMV). While much of the bacterial interactions lead to a protective extracellular matrix that provides resistance to antibiotic treatment, some bacterial OMVs have been shown to disrupt other bacterial biofilms. Wang et al. hypothesized that <em>Burkholderia thailandensis</em> OMVs will possess antibiofilm activity against the common oral pathogen <em>Streptococcus mutans</em>.</p>
<p>In order to test this hypothesis, Wang et al. measured planktonic response, biofilm biomass, biofilm integrity, and bacterial cell viability via the following experiments:</p>
<p><strong>Step 1: Testing antimicrobial on planktonic growth</strong></p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>What are OMVs? (5)</p>
<p>Gram-negative bacteria are capable of shedding outer membrane vesicles in response to environmental stressors. These vesicles contain many components, including: small molecules, proteins, lipids, polysaccarides, and RNA/DNA encased in a liquid particle. This article states that previous work by their lab reported that non-pathogenic <em>B. thailandenis</em> OMWs possess antimicrobial compounds such as peptidoglycan hydrolases, HMNQ, and long-chain rhamnolipids.</p>
</div>
<p>Wang et al. first tested the <em>B. thailandenis</em> OMW’s effect on <em>S. mutans</em> planktonic growth (ie. non-biofilm cells).</p>
<div class="figure align-default" id="fig96">
<a class="reference internal image-reference" href="_images/2-21.png"><img alt="_images/2-21.png" src="_images/2-21.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 96 </span><span class="caption-text">The antimicrobial activity of B. thailandenis OMVs against <em>S. mutans</em> was evaluated in planktonic cultures. <em>S. mutans</em> cultures were treated with PBS (empty buffer control) and varying concentrations of OMVs. **** indicates significant change in all cultures.</span><a class="headerlink" href="#fig96" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">Questions to consider:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">What is being measured here? Is this assay conducted on biofilm cells or non-biofilm cells?</p></li>
<li><p class="sd-card-text">What is the control group and what are the treatment groups?</p></li>
<li><p class="sd-card-text">How do the treatment groups compare to the control? Is this change significant?</p></li>
<li><p class="sd-card-text">What does this say about the effectiveness of OMVs against planktonic <em>S. mutans</em>?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 2: Antimicrobial activity in planktonic vs biofilm conditions</strong>
Antimicrobial effects of OMVs and gentamicin in both planktonic and biofilm conditions were compared. This was measured by counting colony forming units (CFUs).</p>
<div class="margin sidebar">
<p class="sidebar-title">Gentamicin</p>
<p>Gentamicin is a common antibiotic used in the treatment of general bacterial infections and specifically has shown strong activity against <em>S. mutans</em> strains (6).</p>
</div>
<div class="figure align-default" id="fig97">
<img alt="_images/2-31.png" src="_images/2-31.png" />
<p class="caption"><span class="caption-number">Fig. 97 </span><span class="caption-text"><em>S. mutans</em> (B) biofilm cells and (C) planktonic cells were treated with 50 or 100 µg/mL OMVs, 800 µg/mL gentamicin or PBS for up to 24 h with cell viability monitored. **** indicate significant changes. Panel A was purposely removed.</span><a class="headerlink" href="#fig97" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What are the differences between Figure 97B and C?</p></li>
<li><p class="sd-card-text">What is the purpose of gentamicin in this experiment?</p></li>
<li><p class="sd-card-text">What extra information does Figure 97C tell us about OMV activity when compared to Figure 96?</p></li>
<li><p class="sd-card-text">What information can you gather about OMV treatment when comparing Figure 97B and Figure 97C?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that OMVs from <em>B. thailandensis</em> possess antibiofilm activity against <em>S. mutans</em>?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 3: Further characterizing effects on biofilm formation</strong></p>
<p>Wang et al. Further characterized biofilm formation using fluorescent dyes, a common analysis technique.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>Fluorescent Dyes</strong></p>
<p>Fluorescent dyes are small molecules that absorb and emit light at different wavelengths (7). These are often used to visualize cellular components, by using fluorescent dyes to specifically target cellular components. The dye SYTO 9 binds to nucleic acids of both live and dead cells (8). Propidium iodine (PI), on the other hand, is a nucleic acid dye that cannot permeate the cell membrane (9). Therefore, PI is used to stain dead cells.</p>
</div>
<div class="figure align-default" id="fig98">
<img alt="_images/2-4.png" src="_images/2-4.png" />
<p class="caption"><span class="caption-number">Fig. 98 </span><span class="caption-text">S. mutans biofilms were grown for three days before treatment with (A) PBS, (B) 50 µg/mL, or (C) 100 µg/mL OMVs for 24 h. Following staining with LIVE/DEAD BacLight fluorescent dye (SYTO 9/propidium iodide). Green: total biomass. Red: dead cells. Side bars indicate the thickness of the biofilms. (D) Total biofilm biomass/dead cell biomass and (E) average thickness. ** and **** indicate significance.</span><a class="headerlink" href="#fig98" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">Which boxes represent the STYO 9 stained cultures in Figures 98a-c? The PI stained cultures?</p></li>
<li><p class="sd-card-text">What does SYTO 9 fluorescent dye stain in Figures 98a-c represent? The PI dye? Merged?</p></li>
<li><p class="sd-card-text">What aspect of Figures 98a-c is complementary to Figure 98d? Why?</p></li>
<li><p class="sd-card-text">What aspect of Figures 98a-c is complementary to Figure 98e? Why?</p></li>
<li><p class="sd-card-text">What is the effect of OMVs on <em>S. mutans</em> biofilms? Explain in relation to the parameters measured in this figure.</p></li>
<li><p class="sd-card-text">Are any of the data in this figure redundant with that presented in Figure 97b? Why?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that OMVs from <em>B. thailandensis</em> possess antibiofilm activity against S. mutans?</p></li>
</ol>
</div>
</div>
</div>
</div>
<div class="section" id="quorum-sensing">
<h2>2.3 Quorum Sensing<a class="headerlink" href="#quorum-sensing" title="Permalink to this heading">#</a></h2>
<p>For decades, microbiologists have tended to think of prokaryotic populations as collections of individual organisms reproducing and behaving independently. However, it has become clear that these organisms “talk to” each other using chemicals to coordinate their activities. In order to cooperate, prokaryotes need to be able to sense the presence of others of their own kind that are nearby in the environment. The mechanism to do this is called <strong>quorum sensing</strong> (“quorum” = “sufficient numbers”).</p>
<p>Quorum sensing is a type of environmentally-regulated gene expression in which gene expression is regulated by population density. It is a way prokaryotes sense the presence of others of their own kind and work together. Large changes in gene expression change a cell’s phenotype from individualistic growth to community-based growth.</p>
<p>Quorum sensing is used to ensure that sufficient numbers of a given species are present before initiating some response that requires a certain population density to have an effect. Several different quorum sensing mechanisms of varying complexity based on either one-component or two-component systems have evolved in prokaryotes to regulate gene expression in response to population density. Although they differ in the details of molecular mechanism, they all involve a communication chemical called an <strong>autoinducer (AI)</strong>.</p>
<div class="section" id="quorum-sensing-in-e-coli">
<h3>2.3.1 Quorum sensing in <em>E. coli</em><a class="headerlink" href="#quorum-sensing-in-e-coli" title="Permalink to this heading">#</a></h3>
<p>The Lsr quorum sensing system is one of <em>E. coli’s</em> more extensively studied systems. This system involves two operons as shown in Fig 99 (10,11). One operon contains the <em>lsrK</em> and <em>lsrR</em> genes which encode the two regulators in the system. The other operon consists of several genes, <em>lsrABCD</em>, that encode proteins involved in sensing and transporting the autoinducer into the cell when it reaches the threshold. LsrR being the major <strong>global regulator</strong> of the system is an inhibitor of its own operon and the <em>lsrABCD</em> operon. Interestingly, LsrR, even though it is the main regulator of this quorum sensing system, keeps the quorum sensing system shut off when cell concentration is low. It does so by repressing the expression of both operons which includes its own expression.</p>
<div class="figure align-default" id="fig99">
<img alt="_images/2-51.png" src="_images/2-51.png" />
<p class="caption"><span class="caption-number">Fig. 99 </span><span class="caption-text">Lsr quorum sensing system in <em>E. coli</em>.</span><a class="headerlink" href="#fig99" title="Permalink to this image">#</a></p>
</div>
<p><strong>Low population density.</strong> LuxR is the enzyme responsible for synthesizing the autoinducer, called Autoinducer 2 or AI2, in this system. It does so by converting a molecule called DPD into AI2 which is transported out of the cell by the membrane-bound protein, YdgG. Even when population density is low, the cell continues to secrete AI2 as a way to indicate to other cells that it is present.</p>
<p><strong>High population density.</strong> As the population density increases, since the cells have been secreting AI2, the concentration of AI2 increases as well. Once this concentration reaches a threshold, the LsrABCD complex (membrane-bound) starts to import AI2 from the environment into the cell where LsrB specifically senses external AI2 levels. After AI2 is imported into the cell, AI2 is phosphorylated by LsrK. This phosphorylated form of AI2 binds to LsrR and relieves its repression of the two lsr operons to allow further production of quorum sensing proteins.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>Global Regulator</strong></p>
<p>Some transcriptional regulators are “promiscuous”. It means that they don’t just bind to a single promoter but can regulate the expression of multiple different genes and sometime in multiple different ways (ie. as an enhancer for some and repressor for others). A global regulator is one that regulates the expression of other transcriptional regulator causing a cascade of changes in gene expression of many genes.</p>
</div>
<p>LsrR is a global regulator. This means that it not only regulates its own operon but several other genes in the cell including other regulators. Therefore, it can activate a cascade of changes in gene expression in the cell. These changes often result in the cell undergoing some form of community-based phenotype. LsrR can act as a transcriptional repressor or activator. Binding to AI2-P can change how it binds to certain promoters.</p>
</div>
</div>
<div class="section" id="bacterial-motility-and-chemotaxis">
<h2>2.4 Bacterial Motility and Chemotaxis<a class="headerlink" href="#bacterial-motility-and-chemotaxis" title="Permalink to this heading">#</a></h2>
<div class="section" id="chemotaxis-vs-biofilm-formation">
<h3>2.4.1 Chemotaxis vs Biofilm Formation<a class="headerlink" href="#chemotaxis-vs-biofilm-formation" title="Permalink to this heading">#</a></h3>
<p>Biofilms are a bacteria’s response to persistent environmental changes leading individual bacteria to aggregate in large microbial communities of immobile cells. This requires a transformation from the planktonic bacterial lifestyle which often requires motility to respond to rapid fluctuations in their microenvironments. This responsive and motile lifestyle is dependent on a process called <strong>chemotaxis</strong> (12).</p>
<p>Chemotaxis is a change in motile behaviour as the result of sensing a chemical concentration difference (i.e. a concentration gradient) in the environment. It is an immediate change in behaviour that does not require the synthesis of any new proteins.</p>
<ul class="simple">
<li><p>Positive chemotaxis: the movement toward higher concentrations of chemical substances called <strong>attractants</strong> (eg. nutrients).</p></li>
<li><p>Negative chemotaxis: the movement away from higher concentrations of chemical substances called <strong>repellents</strong> (eg. wastes, antimicrobial compounds).</p></li>
</ul>
</div>
<div class="section" id="sensing">
<h3>2.4.2 Sensing<a class="headerlink" href="#sensing" title="Permalink to this heading">#</a></h3>
<p>In order to exhibit chemotaxis, an organism must have a mechanism to sense when the environmental concentration of an attractant or repellant is increasing or decreasing and be able to respond to the environmental sensation in the appropriate way.</p>
<p>As far as sensing goes, in principle there are at least two ways to sense a chemical concentration gradient in the environment. In spatial sensing, a concentration difference is sensed by comparing the concentration of a chemical at two points in space at the same time using receptors at the front and back of the cell (Fig 100). In temporal sensing, a concentration difference is sensed by comparing the concentration of a chemical at two points in time (Fig 100). Why a prokaryote might use spatial or temporal sensing or both is not clear. Organism size, morphology, kind of motility and speed of motility have all been implicated.</p>
<div class="figure align-default" id="fig100">
<img alt="_images/2-61.png" src="_images/2-61.png" />
<p class="caption"><span class="caption-number">Fig. 100 </span><span class="caption-text">Spatial and temporal sensing used in chemotaxis.</span><a class="headerlink" href="#fig100" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="responding">
<h3>2.4.3 Responding<a class="headerlink" href="#responding" title="Permalink to this heading">#</a></h3>
<p>The smooth forward motion of a swimming organism is called a <strong>run</strong>. A run is typically associated with flagellar rotation in the counter-clockwise rotation. Although we often relate the flagellum to a “tail”-like structure, note that it does not beat or flap like an animal tail but rather rotates using a motor.</p>
<p>As far as <strong>monotrichous (have a single flagellum)</strong> organisms go, some change direction by reversing flagellar rotation (to clockwise). This pulls the cell backward in an erratic way causing a random change in direction for a subsequent run to a twitching motion called <strong>tumbling</strong> (Fig 101). Others change direction by briefly stopping flagellar rotation. It is Brownian motion (random motion) that points them in a random direction for a subsequent run.</p>
<div class="figure align-default" id="fig101">
<img alt="_images/2-71.png" src="_images/2-71.png" />
<p class="caption"><span class="caption-number">Fig. 101 </span><span class="caption-text">Run and tumble in monotrichous cells.</span><a class="headerlink" href="#fig101" title="Permalink to this image">#</a></p>
</div>
<p>In <strong>peritrichous</strong> organisms, multiple flagella are scattered over the cell surface while <strong>lophotrichous</strong> organisms possess a tuft of flagella at a single location. In lophotrichous and peritrichous Bacteria, the rotation of all flagella in one direction causes them to come together in a bundle and act in a coordinated manner resulting in running motion (Fig 102). While a “stop and go” method of changing direction could work, in the organisms studied so far, a change in direction is mediated by a tumble. A tumble results from a switch in the direction of rotation by at least one flagellum. When this occurs, the bundle “flys apart”, the run is disrupted and the bacterium undergoes chaotic 3D whirling motions = a tumble. Following a tumble, a subsequent run occurs in a random direction.</p>
<div class="figure align-default" id="fig102">
<img alt="_images/2-81.png" src="_images/2-81.png" />
<p class="caption"><span class="caption-number">Fig. 102 </span><span class="caption-text">Run and tumble in lophotrichous cells.</span><a class="headerlink" href="#fig102" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="chemotaxis-in-e-coli">
<h3>2.4.4 Chemotaxis in <em>E. coli</em><a class="headerlink" href="#chemotaxis-in-e-coli" title="Permalink to this heading">#</a></h3>
<p>Chemotaxis has been the most well-studied in peritrichously flagellated <em>E. coli</em> (Surprise! Surprise!) which changes direction by tumbling. As a model organism, we want to understand how chemotaxis works in <em>E. coli</em> by considering a default non-chemotactic conditions then considering conditions that may trigger running or tumbling. Although the information in this section is focused on <em>E. coli</em>, note that chemotaxis is a conserved mechanism across bacterial species that does have quite a bit of variability in regards to its overall functionality and the molecular pathways.</p>
<p><strong>Default non-chemotactic conditions</strong></p>
<p>In the absence of a chemical concentration gradient, motility rarely results in net movement in any particular direction. Instead, cells follow a rather random pattern in a motion called <strong>Brownian motion</strong> (Fig 103). Under Brownian motion, there is no set end point or net directionality.</p>
<div class="figure align-default" id="fig103">
<img alt="_images/2-91.png" src="_images/2-91.png" />
<p class="caption"><span class="caption-number">Fig. 103 </span><span class="caption-text">Brownian motion in the absence of any chemical gradient.</span><a class="headerlink" href="#fig103" title="Permalink to this image">#</a></p>
</div>
<p>Let’s now consider a situation where there a chemical gradient present. Depending on the chemical, if it promotes movement towards it, is considered a <strong>chemoattractant</strong>. If it causes the cells to move away from it, it is normally considered a <strong>chemorepellent</strong>. The relative position and type of chemical may promote either tumbling (to change direction) or running. We cannot say that a single type of chemical only promotes one type of reaction (ie. chemoattractants only promote running). It dependent on the location. If a chemoattract is behind a cell, it may tumble first to change direction then run towards it. If it senses the attractant in front of it, it will just keep running.</p>
<div class="section" id="tumbling-motion">
<h4>Tumbling Motion<a class="headerlink" href="#tumbling-motion" title="Permalink to this heading">#</a></h4>
<p>Here we are going to look at a simplified version of the molecular gears that define chemotaxis in <em>E. coli</em> (Fig 2.10). Clustered at the front of the cell, is a series of proteins called chemoreceptors. <em>E. coli</em> has several classes of <strong>chemoreceptors</strong> with Tar and Tsr being the most dominant. Chemoreceptors are bundled in high concentrations in order to be more sensitive to signals (robustness) and to promote signal amplification.</p>
<p>If a chemorepellent, for example an antibiotic, is sensed in front of the cell, the following will occur to cause the cell to tumble and change direction (Fig 104):</p>
<ol class="arabic simple">
<li><p>One of the classes of chemoreceptors will sense the signal and activate CheA which is one of the many response regulators in the pathway. CheA is a kinase, an enzyme capable of phosphorylating other proteins.</p></li>
<li><p>Once activated by the chemoreceptors, CheA will phosphorylate CheB in order to activate its enzymatic activity. CheB is a demethylase (an enzyme that removes methyl (-CH3) groups from other proteins). CheB demethylates CheA, the very protein that just activated it.</p></li>
<li><p>When CheA is demethylated, it is able to phosphorylate a second protein, CheY.</p></li>
<li><p>When CheY is phosphorylated, it changes it conformation so that it can actively bind to the flagella motor switching its default, unbound rotation from counter-clockwise (CCW) to clockwise (CW) causing the cell to tumble and change direction.</p></li>
</ol>
<div class="figure align-default" id="fig104">
<img alt="_images/2-101.png" src="_images/2-101.png" />
<p class="caption"><span class="caption-number">Fig. 104 </span><span class="caption-text">Molecular pathway leading to tumbling in <em>E. coli</em>. Pi or P = phosphate groups, CH3 = methyl group.</span><a class="headerlink" href="#fig104" title="Permalink to this image">#</a></p>
</div>
</div>
<div class="section" id="running-motion">
<h4>Running Motion<a class="headerlink" href="#running-motion" title="Permalink to this heading">#</a></h4>
<p>Let’s consider now that the cell has successfully changed directions and now it wants to put the repellent behind it and run. There are two key enzymes involved in “resetting” the system. Resetting the system means going back to the default rotation of the flagella when it is not bound to CheY-P which is CCW, running rotation (Fig 105):</p>
<ol class="arabic simple">
<li><p>Once the cell senses that it is now facing a direction where the antibiotic concentration is lower, CheA will stop phosphorylating CheB and instead, CheR, a methylase, will start to methylate CheA to CheA-CH3.</p></li>
<li><p>CheA-CH3’s kinase activity is turned off so it no longer phosphorylates CheY. Instead, CheZ is going to dephosphorylate CheY changing its conformation so that it can no longer bind the flagellar motor.</p></li>
<li><p>The free, unbound motor rotates in CCW direction to promote running.</p></li>
</ol>
<div class="figure align-default" id="fig105">
<img alt="_images/2-111.png" src="_images/2-111.png" />
<p class="caption"><span class="caption-number">Fig. 105 </span><span class="caption-text">Molecular pathway leading to running in <em>E. coli</em>.</span><a class="headerlink" href="#fig105" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
</div>
<div class="section" id="case-study-autoaggregation">
<h2>Case Study: Autoaggregation<a class="headerlink" href="#case-study-autoaggregation" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Laganenka et al., 2016</strong></p>
<p class="sd-card-text">Laganenka L, Colin R, Sourjik V. 2016. Chemotaxis towards autoinducer 2 mediates autoaggregation in Escherichia coli. Nature Comm. 7:12984</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p>Autoaggregation is a phenomenon where multiple bacterial cells come together and form a small cluster. The mechanisms of how autoaggregation occurs have been of interest to molecular microbiologists for a while now. In this paper, Laganenka et al. explore the potential that both quorum sensing and chemotaxis may play a role in regulating autoaggregation in <em>E. coli</em>. They hypothesize the <em>E. coli</em> responds to AI2 chemotactically to promote autoaggregation.</p>
<p>Laganenka et al. studied this systematically by first looking into (1) the role of chemotaxis, then (2) quorum sensing, and finally (3) both in mediating autoaggregation.</p>
<p><strong>Step 1: Exploring the role of chemotaxis in autoaggregation</strong></p>
<p>The researchers used knock-out mutants of genes involved in swimming motility, as this is an important phenotype regulated by chemotaxis, and chemotaxis genes on how <em>E. coli</em> cultures autoaggregate. The swimming genes include <em>flu, fliC, and motA</em>. The chemotaxis genes include <em>cheY, tar, tsr</em>.</p>
<div class="figure align-default" id="fig106">
<img alt="_images/2-121.png" src="_images/2-121.png" />
<p class="caption"><span class="caption-number">Fig. 106 </span><span class="caption-text">Aggregation of the wild-type <em>E. coli</em> W3110 (a), and knockouts (b-e). (f) Quantification of the area occupied by aggregates in microscopy images for the wild type and indicated mutants. Error bars indicate s.d. of three independent replicates.</span><a class="headerlink" href="#fig106" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What technique was used to collect data in panels a-e?</p></li>
<li><p class="sd-card-text">Look at the WT first, how does the culture change over time? What are characteristics of autoaggregates?</p></li>
<li><p class="sd-card-text">What does the OD600=6.0 mean? Is 6.0 considered a high OD level?</p></li>
<li><p class="sd-card-text">How do each mutant compare to the WT? If it’s different, what does this say about the importance of that gene in mediating autoaggregation?</p></li>
<li><p class="sd-card-text">Is swimming important for regulating autoaggregation?</p></li>
<li><p class="sd-card-text">Is chemotaxis important for regulating autoaggregation?</p></li>
<li><p class="sd-card-text">What is being measured in panel f?</p></li>
<li><p class="sd-card-text">Does the data in panel f match panels a-e? Are they comparable?</p></li>
<li><p class="sd-card-text">Based on panel f, can you tell which genes are more or less important for regulating autoaggregation?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 2: Exploring the role of quorum sensing in autoaggregation</strong></p>
<div class="figure align-default" id="fig107">
<img alt="_images/2-131.png" src="_images/2-131.png" />
<p class="caption"><span class="caption-number">Fig. 107 </span><span class="caption-text">Activity of the <em>lsr</em> promoter reporting AI-2 levels during growth of the wild-type population as a function of the growth stage of <em>E. coli</em> culture. (b) Aggregation of the wild type at different stages of growth, assayed as in Fig 106. (c) Aggregation of strains defective in production (<em>luxS</em>), sensing and import (<em>lsrB</em>) or only import (<em>lsrC</em>) of AI-2. (d) Effects of indicated concentrations of added DPD on aggregation of the wild-type cells. Error bars in all panels indicate s.d. of three independent replicates.</span><a class="headerlink" href="#fig107" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">For each panel, consider what technique is being used? What is being measured?</p></li>
<li><p class="sd-card-text">For panel A, what phase of growth is AI-2 produced the most?</p></li>
<li><p class="sd-card-text">For panel B, what do the varying OD levels indicate? What does this say about autoaggregation in a more mature versus less mature culture?</p></li>
<li><p class="sd-card-text">Panel C, rank the genes based on their importance to mediating autoaggregation.</p></li>
<li><p class="sd-card-text">Does the data in panel D align with the rest of the data and the authors’ hypothesis?</p></li>
<li><p class="sd-card-text">In general, is quorum sensing important in regulating autoaggregation in E. coli based on the data shown here?</p></li>
</ol>
</div>
</div>
</div>
<p><strong>Step 3: Comparing chemotaxis and QS in autoaggregation</strong></p>
<div class="figure align-default" id="fig108">
<img alt="_images/2-141.png" src="_images/2-141.png" />
<p class="caption"><span class="caption-number">Fig. 108 </span><span class="caption-text">Dynamics of aggregation and dispersal for the wild-type, <em>cheY, lsrB and lsrC</em> mutant strains over 5 h incubation (d) and corresponding levels of AI-2 activity in supernatants quantified using fluorescence. Panels a-c are purposely omitted.</span><a class="headerlink" href="#fig108" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">For each panel, consider what technique is being used? What is being measured?</p></li>
<li><p class="sd-card-text">For panel A, which mutants showed irregular autoaggregation? Consider what you are comparing to. How did the mutation affect autoaggregation? Improved it or inhibited it?</p></li>
<li><p class="sd-card-text">For panel B, how does knocking-out lsrC affect AI2 levels?</p></li>
<li><p class="sd-card-text">Which genes are shown to be important in regulating autoaggregation through AI2 sensing?</p></li>
</ol>
</div>
</div>
</div>
<p></p>
</div>
<div class="section" id="references">
<h2>References:<a class="headerlink" href="#references" title="Permalink to this heading">#</a></h2>
<ol class="arabic simple">
<li><p>Bai, X., Nakatsu, C. H. &amp; Bhunia, A. K. Bacterial Biofilms and Their Implications in Pathogenesis and Food Safety. Foods 10, 2117 (2021).</p></li>
<li><p>What Are Biofilms? <a class="reference external" href="https://www.cs.montana.edu/webworks/projects/stevesbook/contents/chapters/chapter001/section002/black/page001.html">https://www.cs.montana.edu/webworks/projects/stevesbook/contents/chapters/chapter001/section002/black/page001.html</a>.</p></li>
<li><p>Parsek, M. R. &amp; Fuqua, C. Biofilms 2003: Emerging Themes and Challenges in Studies of Surface-Associated Microbial Life. J Bacteriol 186, 4427–4440 (2004).</p></li>
<li><p>Jamal, M. et al. Bacterial biofilm and associated infections. J Chin Med Assoc 81, 7–11 (2018).</p></li>
<li><p>Wang, Y. et al. Inhibition of Streptococcus mutans biofilms with bacterial-derived outer membrane vesicles. BMC Microbiol 21, 234 (2021).</p></li>
<li><p>Baker, C. N. &amp; Thornsberry, C. Antimicrobial Susceptibility of Streptococcus mutans Isolated from Patients with Endocarditis. Antimicrob Agents Chemother 5, 268–271 (1974).</p></li>
<li><p>Fluorescent Dyes in Microscopy - Types, Vs Proteins, Applications Etc. MicroscopeMaster <a class="reference external" href="https://www.microscopemaster.com/fluorescent-dyes.html">https://www.microscopemaster.com/fluorescent-dyes.html</a>.</p></li>
<li><p>SYTO 9 Stain - CA. <a class="reference external" href="https://www.thermofisher.com/ca/en/home/life-science/cell-analysis/fluorophores/syto-9.html">https://www.thermofisher.com/ca/en/home/life-science/cell-analysis/fluorophores/syto-9.html</a>.</p></li>
<li><p>Propidium iodide = 94.0 HPLC 25535-16-4. <a class="reference external" href="http://www.sigmaaldrich.com/">http://www.sigmaaldrich.com/</a>.</p></li>
<li><p>Li, J. et al. Quorum Sensing in Escherichia coli Is Signaled by AI-2/LsrR: Effects on Small RNA and Biofilm Architecture. Journal of Bacteriology 189, 6011–6020 (2007).</p></li>
<li><p>Wang, L., Li, J., March, J. C., Valdes, J. J. &amp; Bentley, W. E. luxS-Dependent Gene Regulation in Escherichia coli K-12 Revealed by Genomic Expression Profiling. J Bacteriol 187, 8350–8360 (2005).</p></li>
<li><p>Huang, Z. et al. Cross Talk between Chemosensory Pathways That Modulate Chemotaxis and Biofilm Formation. mBio 10, e02876-18 (2019).</p></li>
<li><p>Xiong, Q. et al. Quorum Sensing Signal autoinducer-2 promotes root colonization of Bacillus velezensis sqr9 by affecting biofilm formation and motility. Applied Microbiology and Biotechnology, 104(16), 7177–7185 (2020).</p></li>
</ol>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="unit2-chap1.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Chapter 1: The Human Microbiome</p>
      </div>
    </a>
    <a class="right-next"
       href="unit2-chap3.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 3: Bacterial Pathogenesis and Virulence</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bacteria-and-biofilms">2.1 Bacteria and Biofilms</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#what-is-a-biofilm">2.1.1 What is a Biofilm?</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#prevalence-of-biofilms">2.1.2 Prevalence of biofilms</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biofilms-in-nature-and-in-human-disease">2.1.3 Biofilms in nature and in human disease</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#treating-biofilms">2.2 Treating Biofilms</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#antimicrobial-resistance-and-persistence-of-biofilms">2.2.1 Antimicrobial resistance and persistence of biofilms</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#new-pharmaceutical-approaches-to-anti-biofilm-therapy">2.2.2 New pharmaceutical approaches to anti-biofilm therapy</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#biofilm-regulatory-systems">2.2.3 Biofilm regulatory systems</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-investigating-biofilm-treatment-and-modes-of-study">Case Study: Investigating biofilm treatment and modes of study</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#quorum-sensing">2.3 Quorum Sensing</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#quorum-sensing-in-e-coli">2.3.1 Quorum sensing in <em>E. coli</em></a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-motility-and-chemotaxis">2.4 Bacterial Motility and Chemotaxis</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotaxis-vs-biofilm-formation">2.4.1 Chemotaxis vs Biofilm Formation</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#sensing">2.4.2 Sensing</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#responding">2.4.3 Responding</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#chemotaxis-in-e-coli">2.4.4 Chemotaxis in <em>E. coli</em></a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#tumbling-motion">Tumbling Motion</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#running-motion">Running Motion</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-autoaggregation">Case Study: Autoaggregation</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By MICB 211 Teaching Team led by Evelyn Sun
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2022.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>