

<!DOCTYPE html>


<html lang="en" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Chapter 3: Bacterial Pathogenesis and Virulence &#8212; MICB 211 Foundations in Microbiology eTextbook</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "light";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=e353d410970836974a52" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=e353d410970836974a52" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.1.2/css/all.min.css?digest=e353d410970836974a52" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" href="_static/styles/sphinx-book-theme.css?digest=14f4ca6b54d191a8c7657f6c759bf11a5fb86285" type="text/css" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.4045f2051d55cab465a707391d5b2007.min.css" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=e353d410970836974a52" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52" />

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/togglebutton.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script src="_static/scripts/sphinx-book-theme.js?digest=5a5c038af52cf7bc1a1ec88eea08e6366ee68824"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'unit2-chap3';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Chapter 4: Antimicrobials and Resistance" href="unit2-chap4.html" />
    <link rel="prev" title="Chapter 2: Community Lifestyle of a Biofilm" href="unit2-chap2.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <a class="skip-link" href="#main-content">Skip to main content</a>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__primary"
          id="__primary"/>
  <label class="overlay overlay-primary" for="__primary"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          name="__secondary"
          id="__secondary"/>
  <label class="overlay overlay-secondary" for="__secondary"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>
  
    <nav class="bd-header navbar navbar-expand-lg bd-navbar">
    </nav>
  
  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
  

<a class="navbar-brand logo" href="intro.html">
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="Logo image"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="Logo image"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item"><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="intro.html">
                    MICB 211 E-Textbook Acknowledgements
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 1</span></p>
<ul class="nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit1-chap1.html">Chapter 1: Introduction to Microbiology and Data Analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap2.html">Chapter 2: Microbial Cell Structures</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap3.html">Chapter 3: Microbial Growth in the Lab and in Nature</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap4.html">Chapter 4: Gene Expression and Regulation</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit1-chap5.html">Chapter 5: Sensing and Responding to the Environment</a></li>
</ul>
<p aria-level="2" class="caption" role="heading"><span class="caption-text">Unit 2</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="unit2-chap1.html">Chapter 1: The Human Microbiome</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap2.html">Chapter 2: Community Lifestyle of a Biofilm</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Chapter 3: Bacterial Pathogenesis and Virulence</a></li>
<li class="toctree-l1"><a class="reference internal" href="unit2-chap4.html">Chapter 4: Antimicrobials and Resistance</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><label class="sidebar-toggle primary-toggle btn btn-sm" for="__primary" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</label></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/executablebooks/jupyter-book/issues/new?title=Issue%20on%20page%20%2Funit2-chap3.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/unit2-chap3.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>


<script>
document.write(`
  <button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></span>
    <span class="theme-switch" data-mode="dark"><i class="fa-solid fa-moon"></i></span>
    <span class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></span>
  </button>
`);
</script>

<script>
document.write(`
  <button class="btn btn-sm navbar-btn search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
  </button>
`);
</script>
<label class="sidebar-toggle secondary-toggle btn btn-sm" for="__secondary"title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</label>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Chapter 3: Bacterial Pathogenesis and Virulence</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-pathogenicity">3.1 Introduction to Pathogenicity</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#pathogenicity-vs-virulence">3.1.1 Pathogenicity vs. Virulence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-infections">3.1.2 Bacterial Infections</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#infecting-the-host">3.2 Infecting the host</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#mode-of-infection">3.2.1 Mode of infection</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#kochs-postulates">3.2.2 Koch’s Postulates</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#regulation-of-genes-encoding-virulence-factors">3.2.3 Regulation of Genes Encoding Virulence Factors</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virulence-factors-mechanisms-of-pathogenic-bacteria">3.3 Virulence Factors: Mechanisms of Pathogenic Bacteria</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-adherence">3.3.1 Bacterial Adherence</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-neisseria-gonorrhoeae-a-pathogen-that-uses-adherence-and-antigenic-variation-to-enter-host-cells">Case Study: <em>Neisseria gonorrhoeae</em> – a pathogen that uses adherence and antigenic variation to enter host cells</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#capsules">3.3.2 Capsules</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-streptococcus-pneumoniae-a-pathogen-that-uses-a-capsule-to-avoid-host-defenses">Case Study: <em>Streptococcus pneumoniae</em> – A pathogen that uses a capsule to avoid host defenses</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#toxins">3.3.3 Toxins</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-vibrio-cholerae-a-pathogen-that-uses-an-enterotoxin-to-damage-its-host">Case Study: <em>Vibrio cholerae</em> – A pathogen that uses an enterotoxin to damage its host</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article" role="main">
                  
  <div class="tex2jax_ignore mathjax_ignore section" id="chapter-3-bacterial-pathogenesis-and-virulence">
<h1>Chapter 3: Bacterial Pathogenesis and Virulence<a class="headerlink" href="#chapter-3-bacterial-pathogenesis-and-virulence" title="Permalink to this heading">#</a></h1>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Learning Objectives</strong></p>
<p class="sd-card-text">By the end of this chapter, students will be expected to:</p>
<ol class="arabic simple">
<li><p class="sd-card-text">define pathogenicity, virulence, and other key bacterial pathogenesis terms.</p></li>
<li><p class="sd-card-text">describe the general steps involved in bacterial infections.</p></li>
<li><p class="sd-card-text">use Koch’s postulates to predict if an organism is associated with a disease.</p></li>
<li><p class="sd-card-text">associate gene regulation (specifically the two-component system) with virulence and virulence factor expression.</p></li>
<li><p class="sd-card-text">explain the virulence mechanisms involved in adhesion, capsules, and toxins.</p></li>
<li><p class="sd-card-text">interpret data on pathogenicity and virulence.</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="introduction-to-pathogenicity">
<h2>3.1 Introduction to Pathogenicity<a class="headerlink" href="#introduction-to-pathogenicity" title="Permalink to this heading">#</a></h2>
<div class="section" id="pathogenicity-vs-virulence">
<h3>3.1.1 Pathogenicity vs. Virulence<a class="headerlink" href="#pathogenicity-vs-virulence" title="Permalink to this heading">#</a></h3>
<p><strong>Pathogenicity</strong> refers to a bacterium’s ability to cause disease. It is a qualitative concept. A <strong>pathogen</strong> is a bacterium that is capable of harming a healthy host. Note that capable means that it may cause harm to a healthy host, but it does not mean it always will. A healthy host means an individual whose defence mechanisms are not compromised in any substantial fashion. On the other hand, an <strong>opportunistic pathogen</strong> is a bacterium that harms a <strong>compromised host</strong>, i.e., a host whose immune defence system is weakened. For example, <em>Pseudomonas aeruginosa</em> causes few infections in healthy individuals but more than 200,000 infections per year in hospitalized individuals in the U.S.</p>
<p><strong>Virulence</strong> is a measure of the pathogenicity of a microorganism, i.e., it is a quantitative concept.  Virulence can be expressed as the cell number that will elicit a pathogenic response in the host within a given time period. For example, an infection with a relatively small number of highly virulent bacteria (<em>Streptococcus pneumoniae</em>) will kill a certain number of mice, whereas an infection with much more of less virulent bacteria (<em>Salmonella typhimurium</em>) is required to kill the same number of mice (Fig 109) (1).</p>
<div class="figure align-default" id="fig109">
<img alt="_images/3-110.png" src="_images/3-110.png" />
<p class="caption"><span class="caption-number">Fig. 109 </span><span class="caption-text">An example of the differences in microbial virulence of two bacterial pathogens (1).</span><a class="headerlink" href="#fig109" title="Permalink to this image">#</a></p>
</div>
<p>The terms pathogenicity and virulence are related in that pathogens can express a wide range of virulence factors. This is a property of the organism that determines the extent of the disease. The extent of the virulence is usually correlated with the ability of the pathogen to multiply within the host and may be affected by factors such as route of entry and the general health/susceptibility of the host. A microorganism can be defined as being a pathogen and depending upon conditions within the host, may exhibit different levels of virulence.</p>
<p>A highly <strong>virulent organism</strong> causes disease in most individuals with whom it comes in contact. An <strong>avirulent organism</strong> rarely, if ever, causes disease in human beings. Between these two extremes there is a range of virulence possessed by microorganisms.  The virulence of microorganisms is directly attributable to the fact that they possess certain factors (called <strong>virulence factors</strong>) which allow it to enter and colonize in the host, resist the immune defences of the host, and cause damage in the host. A few of these virulence factors will be discussed in this chapter.</p>
</div>
<div class="section" id="bacterial-infections">
<h3>3.1.2 Bacterial Infections<a class="headerlink" href="#bacterial-infections" title="Permalink to this heading">#</a></h3>
<p>Bacterial infections may be classified into the following types:</p>
<ul class="simple">
<li><p>A <strong>primary infection</strong> is an initial, acute infection.</p></li>
<li><p>A <strong>secondary infection</strong> is a later (or second) infection caused by an opportunistic organism after a primary infection has weakened the host defences.</p></li>
<li><p>A <strong>subclinical infection</strong> is one in which the host has no apparent symptoms.</p></li>
<li><p>A <strong>nosocomial infection</strong> is one acquired as a consequence of hospitalization or the process of receiving health care. These are infections that were not present before entering the health care facility.</p></li>
</ul>
<p>Infections may also be differentiated according to the extent of host involvement. This is influenced by the invasiveness of the organism, i.e., the ability of the organism to invade beyond the original site of entry into the host’s body. Some terms describing invasiveness and the spread of disease within a host are listed below.</p>
<ul class="simple">
<li><p>A **local infection*8 is restricted to a relatively small area of the body. Non-invasive organisms usually cause these infections.</p></li>
<li><p>A <strong>systemic or (generalized) infection</strong> is one which is spread throughout the body.  Invasive organisms usually cause these infections.</p></li>
<li><p>A <strong>focal infection</strong> begins in a restricted area and then spreads throughout the body.</p></li>
</ul>
<p>Host susceptibility influences the outcome during infection.  As mentioned earlier, the interaction between microorganisms and the host is affected by the virulence of the microorganisms and the anti-bacterial defences of the host. The susceptibility to infection is increased if the anti-bacterial defences of the host are compromised. In fact, bacteria with low virulence may cause disease in a compromised host.</p>
<p>The host may be more susceptible to infection if there are factors that compromise the immune defences.  These include:</p>
<ul class="simple">
<li><p>Underlying disease or infection (e.g., AIDS, cancer, inherited immunodeficiencies)</p></li>
<li><p>Stress (e.g., exams)</p></li>
<li><p>Poor nutrition (e.g., a diet which leaves out essential vitamins or minerals)</p></li>
<li><p>Age (e.g., the underdeveloped immune defences of the newborn and infant leave them susceptible to infection with microorganisms to which the older child or healthy adult is immune.  Similarly, the degeneration of the immune responses which comes with age results in older people being more susceptible to infection.)</p></li>
<li><p>Immunosuppressive therapy (e.g., such as the therapy used in the treatment of certain cancers or maintenance of transplanted organs)</p></li>
<li><p>Genetics of the host’s defence systems. Individuals in the population respond differently to infection.  Genetic diseases can affect susceptibility to infection (e.g., cystic fibrosis and lung infection).  In addition, some individuals are more resistant or susceptible to infections than others, although the molecular differences are not yet understood.</p></li>
</ul>
</div>
</div>
<div class="section" id="infecting-the-host">
<h2>3.2 Infecting the host<a class="headerlink" href="#infecting-the-host" title="Permalink to this heading">#</a></h2>
<div class="section" id="mode-of-infection">
<h3>3.2.1 Mode of infection<a class="headerlink" href="#mode-of-infection" title="Permalink to this heading">#</a></h3>
<p>In order to infect the human body, a bacterium usually has to proceed through a number of steps (Fig 110).</p>
<ol class="arabic simple">
<li><p>The bacterium has to come in contact with the host’s body.</p></li>
<li><p>The bacterium has to adhere to (and transiently colonize) or invade the host.</p></li>
<li><p>The bacterium has to multiply.</p></li>
<li><p>The bacterium has to evade the host’s innate immune defence system.</p></li>
</ol>
<p>In addition, for the bacterium to cause disease, it must possess chemical/molecular mechanisms whereby it damages the host (e.g., toxins or host damaging enzymes), or it must trigger a chronic inflammatory response.</p>
<div class="figure align-default" id="fig110">
<img alt="_images/3-24.png" src="_images/3-24.png" />
<p class="caption"><span class="caption-number">Fig. 110 </span><span class="caption-text">Steps involved in bacterial infection of a host.</span><a class="headerlink" href="#fig110" title="Permalink to this image">#</a></p>
</div>
<p>Invasive organisms may cause systemic infections, i.e., the infection can be disseminated throughout the body, often via the blood. This is a concern as bacteria are not normally found in the blood. Since blood is considered sterile, the presence of bacteria in the blood is termed <strong>bacteremia</strong>.</p>
</div>
<div class="section" id="kochs-postulates">
<h3>3.2.2 Koch’s Postulates<a class="headerlink" href="#kochs-postulates" title="Permalink to this heading">#</a></h3>
<p>Proving that a specific microorganism causes a specific disease is not a new problem.  Robert Koch, a microbiologist working in the last century, established a set of postulates that allow one to determine whether a relationship exists between a particular organism and a disease.  These postulates have been used as a guide to determine the causes of many important diseases, which led to the development of treatments for the prevention and cure of many infectious diseases. Fig 111 illustrates the traditional steps of Koch’s postulates. Due to advances in science, molecular considerations are now also made in determining whether a microorganism plays a role in a particular disease.</p>
<p>Koch’s postulates have been used to prove the causative relationship between most bacterial pathogens and their respective diseases. Two major impediments can prevent definitive proof of causation using Koch’s postulates. First, certain bacterial pathogens cannot be cultivated <em>in vitro</em>, and second, sometimes there are no animal models for a specific disease.</p>
<div class="figure align-default" id="fig111">
<img alt="_images/3-31.png" src="_images/3-31.png" />
<p class="caption"><span class="caption-number">Fig. 111 </span><span class="caption-text">Koch’s postulates.</span><a class="headerlink" href="#fig111" title="Permalink to this image">#</a></p>
</div>
<p>Having established through Koch’s postulates that a specific pathogen causes a disease, the next question involves how the pathogen causes disease.  Stan Falkow, the father of molecular pathogenesis, developed a newer set of postulates that allows one to prove that a particular virulence factor or gene is involved in the pathogenesis of an organism:</p>
<p>The Molecular Version of Koch’s Postulates</p>
<p>Applying our understanding of molecular genetics to Koch’s postulates expands them:</p>
<ol class="arabic simple">
<li><p>The gene(s)/factor should be present in pathogenic strains of the organism and absent from non-pathogenic strains.</p></li>
<li><p>The molecular disruption, i.e., deletion of the gene(s), should reduce the virulence of the bacterial strain AND introducing the cloned gene(s) into an avirulent strain should render that strain virulent.</p></li>
<li><p>The gene(s) must be expressed at some point during the infectious process in experimentally infected animals.</p></li>
<li><p>Antibody raised against the virulence factor should offer some protection against infection, in an experimentally infected animal.</p></li>
</ol>
</div>
<div class="section" id="regulation-of-genes-encoding-virulence-factors">
<h3>3.2.3 Regulation of Genes Encoding Virulence Factors<a class="headerlink" href="#regulation-of-genes-encoding-virulence-factors" title="Permalink to this heading">#</a></h3>
<p>The disease-causing genes described in the molecular version of Koch’s postulates are referred to as virulence factors. The possession of virulence factors is what allows the pathogenic bacteria to infect normal people and cause disease (Fig 112).</p>
<p>A particular bacterium may live in many different environments, and in turn may need to regulate the expression of components needed for survival in these varying conditions.  Bacteria turn on these different components, which include virulence factors, only when needed.  Bacteria ‘sense’ several environmental factors in regulating the expression of their various virulence factors.</p>
<div class="figure align-default" id="fig112">
<img alt="_images/3-41.png" src="_images/3-41.png" />
<p class="caption"><span class="caption-number">Fig. 112 </span><span class="caption-text">Summary of virulence factors discussed in this chapter.</span><a class="headerlink" href="#fig112" title="Permalink to this image">#</a></p>
</div>
<p>These environmental factors include aspects such as:</p>
<ul class="simple">
<li><p>Temperature</p></li>
<li><p>Nutrients</p></li>
<li><p>Oxygen (O2)</p></li>
<li><p>pH inside the cell</p></li>
<li><p>Iron availability</p></li>
<li><p>Osmolarity</p></li>
</ul>
<p>Common processes regulated at the genetic level include metabolism, response to environmental stresses and conditions, and cell division.</p>
<p>Many virulent bacteria have <strong>‘global’ regulatory systems</strong> to regulate virulence factors (i.e., regulatory systems that control the production of multiple different virulence factors at once).  These systems are co-ordinately regulated by a two-component system (Fig 113) that comprises a sensor protein and a regulator protein.  This is a common system for signal transduction as introduced in Unit 1 Chap 5.</p>
<div class="figure align-default" id="fig113">
<img alt="_images/3-51.png" src="_images/3-51.png" />
<p class="caption"><span class="caption-number">Fig. 113 </span><span class="caption-text">Revisiting the two-component regulatory system for signal transduction.</span><a class="headerlink" href="#fig113" title="Permalink to this image">#</a></p>
</div>
<p>Applying this to pathogens, the sensor (kinase) protein detects changes in the environment that indicate that the bacterium is residing inside a human host and then phosphorylates the response regulator protein.  The phosphorylated regulator protein then transcriptionally activates or represses the virulence factors (e.g., flagella genes, toxin genes, adhesin genes, capsule genes) by binding to a DNA sequence in the regulatory regions of each of the operons for these virulence factors.</p>
<p>In the rest of this chapter, we will look at the production of virulence factors that bacteria use to enter and colonize the host, resist the immune defences of the host, and cause damage in the host. Keep in mind that all of these result from transcription and translation of DNA, thus gene regulation always plays a part!</p>
</div>
</div>
<div class="section" id="virulence-factors-mechanisms-of-pathogenic-bacteria">
<h2>3.3 Virulence Factors: Mechanisms of Pathogenic Bacteria<a class="headerlink" href="#virulence-factors-mechanisms-of-pathogenic-bacteria" title="Permalink to this heading">#</a></h2>
<div class="section" id="bacterial-adherence">
<h3>3.3.1 Bacterial Adherence<a class="headerlink" href="#bacterial-adherence" title="Permalink to this heading">#</a></h3>
<p>The first host barrier for many invading pathogens is usually a mucosal surface, such as in the gut or respiratory tract.  Therefore, most successful pathogens have evolved specific mechanisms to <strong>adhere</strong> to host tissue.  Simple attachment is mediated through a receptor on the host cell surface (e.g., glycoproteins, glycolipids) and an adhesin (e.g., pili, capsule) on the surface of the bacterial cell (Fig 114).</p>
<div class="figure align-default" id="fig114">
<img alt="_images/3-61.png" src="_images/3-61.png" />
<p class="caption"><span class="caption-number">Fig. 114 </span><span class="caption-text">Adherence of pathogens to animal tissues.  Left:  <em>Vibrio cholerae</em> (lacking an outer layer or clycocalyx) adhering to the brush border of rabbit villi (2).  Middle:  Enteropathogenic <em>E. coli</em> adhering to the brush border of calf villi via an extensive glycocalyx (3).  Right:  A shadow-cast electron micrograph showing type P fimbriae of <em>E. coli</em> (4).</span><a class="headerlink" href="#fig114" title="Permalink to this image">#</a></p>
</div>
<p>These mechanisms benefit the pathogen by facilitating colonization, preventing the removal of the bacterium by non-specific host defence systems and by localizing the bacterium to the appropriate tissue. The specificity of the interaction between the host tissue and bacterium is often both host specific (e.g., <em>Neisseria gonorrhoeae</em> will bind to human tissues but not rat tissues) and tissue-type specific (e.g., <em>E. coli</em> will bind to the intestinal lumen, but not the pharynx). One of the known bacterial structures known to mediate adherence to host cells it the pili (plur.). <strong>Pili or pilus</strong> (sing.) were introduced in Unit 1 Chapter 2 as thin hair-like structures that protrude through the cell surface. Pili have a range of different functions including motility, conjugation, and host cell adherence.</p>
<p>The host responds to adherent organisms through:</p>
<ul class="simple">
<li><p>Production of antibodies against bacterial surfaces</p></li>
<li><p>Upregulation of production of antimicrobial peptides by epithelial cells</p></li>
<li><p>Phagocytosis of the adherent organisms</p></li>
<li><p>Shedding/destruction of epithelial cells that become colonized with bacteria.</p></li>
</ul>
</div>
</div>
<div class="section" id="case-study-neisseria-gonorrhoeae-a-pathogen-that-uses-adherence-and-antigenic-variation-to-enter-host-cells">
<h2>Case Study: <em>Neisseria gonorrhoeae</em> – a pathogen that uses adherence and antigenic variation to enter host cells<a class="headerlink" href="#case-study-neisseria-gonorrhoeae-a-pathogen-that-uses-adherence-and-antigenic-variation-to-enter-host-cells" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>He et al., 2021</strong></p>
<p class="sd-card-text">He Y, Zhang S, Zhang Y, Wu B, Xue Y, Ye C, Li Q, Olivia AN, Tembo JM, Chen H, Cai H, Chen T. Distinct patterns of host adherence by Neisseria gonorrhoeae isolated from experimental gonorrhea. Can J Infect Dis Med Microbiol. 2021 May 13;2021:7865405. doi: 10.1155/2021/7865405.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p><em>Neisseria gonorrhoeae</em>, the causative agent of gonorrhea, is a pathogen with a well-known adherence mechanism mediated by pili. Adherence allows the pathogen to penetrate and attach to host mucosal surfaces. It has been observed that during human infection, the genes coding the pili undergo changes through DNA phase and antigenic variation. This variation aids in avoidance of host defenses and ultimately increases adherence and success in infection. In this study, He et al. isolated variants from six individuals infected with the same initial <em>N. gonorrhoeae</em> clone. He et al. hypothesized that variants in pili genes will either increase or decrease adherence abilities and in turn increase or decrease severity of infection.</p>
<p>In order to test this hypothesis, He et al. collected the following data:</p>
<p><strong>Monitoring adherence and internalization of <em>N. gonorrhoeae</em> variants</strong></p>
<p>The original strain (MS11) and 6 variants were monitored for their ability to adhere to host cells 1.5 h, 3 h, and 6 h post 24-hour incubation. The internalized bacteria (bacteria that have penetrated inside host cells) were quantified by first removing extracellular bacteria using antibiotics, then determining CFUs.</p>
<div class="margin sidebar">
<p class="sidebar-title"></p>
<p>Phase and antigenic variation are features often seen in pathogens as a way of increasing pathogenicity or evading the immune response. It occurs when either express the variable protein, one of many antigenic forms, or no protein at all. This is a mode of producing variation without mutations (5).</p>
</div>
<div class="figure align-default" id="fig115">
<img alt="_images/3-71.png" src="_images/3-71.png" />
<p class="caption"><span class="caption-number">Fig. 115 </span><span class="caption-text">The (a) adherence and (b) internalization of reisolates and original strain (MS11) measuring the amount of cells that have adhered or internalized in CFU.  ∗∗∗, ∗∗, and ∗ indicate statistical significance relative to MS11.</span><a class="headerlink" href="#fig115" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">List the adherence capabilities of the studied strains from highest to lowest?</p></li>
<li><p class="sd-card-text">Compare adherence between the three time points, what can you conclude about the role of time during infection?</p></li>
<li><p class="sd-card-text">List the internalization capabilities of the studied strains from highest to lowest?</p></li>
<li><p class="sd-card-text">Do the internalization data support the adherence data, why or why not?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that increased adherence capabilities result in increased infection?</p></li>
</ol>
</div>
</div>
</div>
<div class="section" id="capsules">
<h3>3.3.2 Capsules<a class="headerlink" href="#capsules" title="Permalink to this heading">#</a></h3>
<p>To avoid phagocytosis, many pathogens have evolved surface components that prevent the attachment and engulfment of macrophages and other host cellular immune responses.  This may take the form of membrane bound proteins, slimy polysaccharide <strong>capsules</strong> (Fig 116), or lipopolysaccharides (LPS - Gram-negative bacteria only). These factors prevent the deposition of antibodies and complement on the bacterial surface, avoiding complement-mediated killing and reducing phagocytosis.</p>
<div class="figure align-default" id="fig116">
<img alt="_images/3-81.png" src="_images/3-81.png" />
<p class="caption"><span class="caption-number">Fig. 116 </span><span class="caption-text">An India ink negatively stained preparation of Streptococcus pneumoniae.</span><a class="headerlink" href="#fig116" title="Permalink to this image">#</a></p>
</div>
</div>
</div>
<div class="section" id="case-study-streptococcus-pneumoniae-a-pathogen-that-uses-a-capsule-to-avoid-host-defenses">
<h2>Case Study: <em>Streptococcus pneumoniae</em> – A pathogen that uses a capsule to avoid host defenses<a class="headerlink" href="#case-study-streptococcus-pneumoniae-a-pathogen-that-uses-a-capsule-to-avoid-host-defenses" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: <strong>Hathaway et al., 2016</strong></p>
<p class="sd-card-text">Hathaway LJ, Grandgirard D, Valente LG, Täuber MG, Leib SL. Streptococcus pneumoniae capsule determines disease severity in experimental pneumococcal meningitis. Open Biol. 2016 Mar;6(3):150269. doi: 10.1098/rsob.150269.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p><em>Streptococcus pneumonia</em>, is an encapsulated pathogen that causes several diseases, notably bacterial meningitis in children. There are more than 90 known polysaccharide capsules that determine the serotype (classifying different variations of a particular microbial structure) of the bacteria. While the cell wall structure of S. pneumonia is known to affect disease severity, Hathaway et al. aims to determine if capsule serotype affects disease severity directly. Therefore, Hathaway et al. hypothesized that changing the capsule type would affect disease severity in an infant meningitis rat model.</p>
<p>In order to test this hypothesis, Hathaway et al. performed the following experiments:</p>
<p><strong>Step 1: Determine infectious capabilities of two <em>S. pneumonia</em> serotypes:</strong></p>
<p>The infectious clinical isolate (serotype 6B) was genetically altered to replace the capsule determining genes with that of another isolate (serotype 7F) ensuring capsule composition to be the only variable. Survival curves and clinical scores of infected rats were measured.</p>
<div class="margin sidebar">
<p class="sidebar-title">clinical score</p>
<p>Clinical scores are a standardized method of evaluating disease severity. In this study, the scoring looked at the following parameters: coma, ability to stand upright, and ability to walk. These scores are also used to evaluate the humane endpoint for animal experiments. Upon reaching too low of a score, animal models are sacrificed (7).</p>
</div>
<div class="figure align-default" id="fig117">
<img alt="_images/3-91.png" src="_images/3-91.png" />
<p class="caption"><span class="caption-number">Fig. 117 </span><span class="caption-text">Clinical parameters assessed during acute bacterial meningitis in animals infected by <em>S. pneumoniae</em>. (a) survival of hosts or (b) clinical scores of rats infected with serotype 6B or 7F strains. Higher score means a less severe infection/disease (ie. healthier hosts).</span><a class="headerlink" href="#fig117" title="Permalink to this image">#</a></p>
</div>
<p><strong>Step 2: Monitoring capsule formation in two S. pneumonia serotypes:</strong></p>
<p>The ability of each serotype to form capsules was monitored in both the inoculum before infecting the cell then 21 hours after host infection.</p>
<div class="figure align-default" id="fig118">
<img alt="_images/3-101.png" src="_images/3-101.png" />
<p class="caption"><span class="caption-number">Fig. 118 </span><span class="caption-text">Capsule thickness was determined in serotype 6B (a,c) and serotype 7F (b,d) before inoculation of rats (a,b) and recovered from rats 21 h after infection (c,d).</span><a class="headerlink" href="#fig118" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">What effect does capsule serotype have on disease severity?</p></li>
<li><p class="sd-card-text">Do the results presented in Figure 117b complement that in Figure 117a?</p></li>
<li><p class="sd-card-text">What are the controls in the experiment depicted in Figure 118?</p></li>
<li><p class="sd-card-text">Based on the images in Figure 118, under what conditions are capsules formed?</p></li>
<li><p class="sd-card-text">Does the data in Figure 118 complement that in Figure 117? Explain?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that capsule type affects disease severity?</p></li>
</ol>
</div>
</div>
</div>
<p></p>
<div class="section" id="toxins">
<h3>3.3.3 Toxins<a class="headerlink" href="#toxins" title="Permalink to this heading">#</a></h3>
<p>Toxins are microbial products that can damage the host. Toxins are capable of causing disease, independent of the bacterium.  For example, the pathogenic bacteria can be killed in a patient, but the disease may persist due to the continued action of the toxin.  There are two types of toxins: <strong>exotoxins and endotoxins</strong>  (Table 3.1 and Fig 119).</p>
<p><strong>Table 3.1</strong>.  Properties of exotoxins and endotoxins.</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>Property</p></th>
<th class="head"><p>Exotoxins</p></th>
<th class="head"><p>Endotoxins</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Chemical properties</p></td>
<td><p>Proteins, excreted by certain   gram-positive or gram-negative bacteria; generally, heat-labile</p></td>
<td><p>Lipopolysaccharide-lipoprotein   complexes, released on cell lysis as part of the outer membrane of   gram-negative bacteria; extremely heat-stable</p></td>
</tr>
<tr class="row-odd"><td><p>Mode of action; Symptoms</p></td>
<td><p>Specific; usually binds to   specific cell receptors or structures; either cytotoxin, enterotoxin or neurotoxin   with defined specific action on cells or tissues.</p></td>
<td><p>General; fever, diarrhea, vomiting</p></td>
</tr>
<tr class="row-even"><td><p>Toxicity</p></td>
<td><p>Often highly toxic, sometimes   fatal</p></td>
<td><p>Weakly toxic, rarely fatal</p></td>
</tr>
<tr class="row-odd"><td><p>Immunogenicity response</p></td>
<td><p>Highly immunogenic; stimulate the   production of neutralizing antibody (antitoxin)</p></td>
<td><p>Relatively poor immunogen: immune   response not sufficient to neutralize toxin</p></td>
</tr>
<tr class="row-even"><td><p>Toxoid potential</p></td>
<td><p>Treatment of toxin with   formaldehyde will destroy toxin, but treated toxin (toxoid) remains   immunogenic</p></td>
<td><p>None</p></td>
</tr>
<tr class="row-odd"><td><p>Fever potential</p></td>
<td><p>Does not produce fever in host</p></td>
<td><p>Pyrogenic, often produces fever in   host</p></td>
</tr>
</tbody>
</table>
<div class="figure align-default" id="fig119">
<img alt="_images/3-111.png" src="_images/3-111.png" />
<p class="caption"><span class="caption-number">Fig. 119 </span><span class="caption-text">Exotoxins compared to endotoxins.</span><a class="headerlink" href="#fig119" title="Permalink to this image">#</a></p>
</div>
<p>Exotoxins are secreted proteins made by both Gram-positive and Gram-negative bacteria.  These toxins may enter the blood circulation and spread to parts of the body and cause damage far from the site of the bacterial colonization.  There are three main categories of exotoxins.  Cytolytic toxins enzymatically attack the cell constituents causing cell lysis (e.g., Staphylococcus aureus hemolysin – lyses red blood cells).  Neurotoxins specifically interfere with nerve cell function (e.g., tetanus toxin).  Enterotoxins affect cells lining the gastrointestinal tract, causing massive fluid secretion (diarrhea, e.g., cholera toxin).</p>
<p>Exotoxins are extremely potent molecules.  A small dose can have a significant effect on the host.  Since exotoxins are proteins, they are usually heat labile (i.e., activity destroyed by heating to 60–80ºC) and they are immunogenic (i.e., stimulates the immune system to produce neutralizing antibodies).  Vaccination against toxin-mediated diseases can be done by immunization with toxoids. Toxoids are toxins that have been treated with heat or chemicals to make them non-toxic to the patient, but still capable of stimulating the production of neutralizing antibodies.</p>
<p>Endotoxins are released only when the bacteria die or are digested by phagocytic cells. Generally, the name ‘endotoxin’ is equated with lipopolysaccharide (LPS), which is present in the outer membrane of Gram-negative bacteria.  Gram-positive bacteria produce molecules with similar properties (e.g., lipoteichoic acid, LTA). Since endotoxins are non-protein molecules and are generally heat stable.</p>
<p>Endotoxin activates macrophages, causing release of inflammatory cytokines (e.g., tumor necrosis factor).  In low doses (&lt;1 mg), a patient might experience fever (i.e., endotoxin is pyrogenic), aches and pains.  In higher doses, there is damage to the circulatory system and shock.  Fluids leak out of blood vessels, blood coagulates, blood pressure drops, organs fail and death may result.  This is commonly known as endotoxin shock. Treatment of endotoxin shock primarily involves supportive therapy.  Antibiotics are not used as they can aggravate LPS/endotoxin release.  A recently introduced treatment is Polymyxin B (antibiotic)-coated beads that can help remove endotoxins from the blood. However, its absolute efficacy is not known.  A vaccine cannot be used since endotoxins are non-protein molecules and they are not very immunogenic.</p>
</div>
</div>
<div class="section" id="case-study-vibrio-cholerae-a-pathogen-that-uses-an-enterotoxin-to-damage-its-host">
<h2>Case Study: <em>Vibrio cholerae</em> – A pathogen that uses an enterotoxin to damage its host<a class="headerlink" href="#case-study-vibrio-cholerae-a-pathogen-that-uses-an-enterotoxin-to-damage-its-host" title="Permalink to this heading">#</a></h2>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text">This case study was adapted from: **Takahashi et al., 2021*8</p>
<p class="sd-card-text">Takahashi E, Ochi S, Mizuno T, Morita D, Morita M, Ohnishi M, Koley H, Dutta M, Chowdhury G, Mukhopadhyay AK, Dutta S, Miyoshi SI, Okamoto K. Virulence of cholera toxin gene-positive <em>Vibrio cholerae</em> non-o1/non-o139 strains isolated from environmental water in Kolkata, India. Front Microbiol. 2021 Aug 20;12:726273. doi: 10.3389/fmicb.2021.726273.</p>
</div>
</div>
</div>
<p><strong>Overview</strong></p>
<p><em>Vibrio cholerae</em>, is the pathogen responsible for the acute diarrheal disease, cholera. This disease is the result of the cholera toxin (CT) which is produced by <em>V. cholerae</em> serogroups O1 and O139. Non-O1 and non-O139 strains, referred to as non-agglutinable (NAG), are often non-CT producing and therefore are not the cause of endemic or pandemic outbreaks. Even still, NAG serogroups have been shown to cause infection on occasion. Interestingly, Takahashi et al. identified unique non-O1 and non-O139 <em>V. cholerae</em> that possess the CT gene (ctx). Takahashi et al. hypothesized that the presence of <em>ctx</em> gene will increase pathogenicity in NAG strains.</p>
<p>In order to test this hypothesis, Hathaway et al. performed the following experiments:</p>
<p><strong>Step 1: Determine the ability of each strain to cause disease:</strong></p>
<p>The disease severity was monitored through measurement of fluid accumulation (a marker of disease) in a rabbit ileal loop (ie. small intestine). N16961 is an O1 <em>V. cholerae</em> strain, strain No. A is a non-<em>ctx</em> containing NAG, and strains No. 1-3 are <em>ctx</em> containing NAGs.</p>
<div class="figure align-default" id="fig120">
<img alt="_images/3-121.png" src="_images/3-121.png" />
<p class="caption"><span class="caption-number">Fig. 120 </span><span class="caption-text">(A) Rabbit ileal loop assay with <em>V. cholerae</em> or buffer injected at different locations. PBS is clean buffer solution. N16961 is a toxic strain of <em>V. cholerae</em>. 4 different strains of non-toxic V. cholerae were tested, No 1-3 and A. (B) The volume of fluid accumulated in the loop (mL) and the length of the ileal loop (cm) were measured, and the fluid accumulation ratio, which was the ratio of the volume of fluid accumulated in the intestinal loop to the length of the loop (mL/cm), was calculated.</span><a class="headerlink" href="#fig120" title="Permalink to this image">#</a></p>
</div>
<p><strong>Step 2: Measure the production of Cholera toxin in each V. cholerae strain:</strong></p>
<p>The CT production of each strain was monitored in vitro under two temperature conditions.</p>
<div class="figure align-default" id="fig121">
<img alt="_images/3-131.png" src="_images/3-131.png" />
<p class="caption"><span class="caption-number">Fig. 121 </span><span class="caption-text">Concentration of cholera toxin in the culture supernatant of <em>V. cholerae</em> N16961 and non-toxic strains, No 1-3 grown at 25C and 37C. Use the error bars to determine significance.</span><a class="headerlink" href="#fig121" title="Permalink to this image">#</a></p>
</div>
<div class="sd-col sd-d-flex-row docutils">
<div class="sd-card sd-sphinx-override sd-w-100 sd-shadow-sm docutils">
<div class="sd-card-body docutils">
<p class="sd-card-text"><strong>Questions to consider:</strong></p>
<ol class="arabic simple">
<li><p class="sd-card-text">How would you classify the cholera toxin? endotoxin? exotoxin?</p></li>
<li><p class="sd-card-text">Does the presence of ctx genes result in disease? Why or why not?</p></li>
<li><p class="sd-card-text">Does the presence of ctx genes result in CT production? Why or why not?</p></li>
<li><p class="sd-card-text">What are the controls in Figure 120? Figure 121?</p></li>
<li><p class="sd-card-text">Do the results presented in Figure 121 complement those in Figure 120?</p></li>
<li><p class="sd-card-text">Based on the results presented in Figures 120 and 121, what is required for <em>V. cholerae</em> to result in intestinal disease?</p></li>
<li><p class="sd-card-text">Does this data support the hypothesis that the presence of <em>ctx</em> will increase the pathogenicity in NAG strains?</p></li>
</ol>
</div>
</div>
</div>
</div>
<div class="section" id="references">
<h2>References:<a class="headerlink" href="#references" title="Permalink to this heading">#</a></h2>
<ol class="arabic simple">
<li><p>Madigan, M. Brock biology of microorganisms. (Pearson Prentice Hall, 2006).</p></li>
<li><p>Nelson, E. T., Clements, J. D. &amp; Finkelstein, R. A. Vibrio cholerae adherence and colonization in experimental cholera: electron microscopic studies. Infect Immun 14, 527–547 (1976).</p></li>
<li><p>Acres, S. D. Enterotoxigenic Escherichia coli infections in newborn calves: a review. J Dairy Sci 68, 229–256 (1985).</p></li>
<li><p>Madigan, M., Martinko, J. &amp; Parker, J. B. The Biology of Microorganisms. (Prentice Hall Inc).</p></li>
<li><p>van der Woude, M. W. &amp; Bäumler, A. J. Phase and Antigenic Variation in Bacteria. Clin Microbiol Rev 17, 581–611 (2004).</p></li>
<li><p>Taylor, W. H. &amp; Juni, E. Pathways for biosynthesis of a bacterial capsular polysaccharide I. J Bacteriol 81, 688–693 (1961).</p></li>
<li><p>Liechti, F. D., Grandgirard, D., Leppert, D. &amp; Leib, S. L. Matrix metalloproteinase inhibition lowers mortality and brain injury in experimental pneumococcal meningitis. Infect Immun 82, 1710–1718 (2014).</p></li>
</ol>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
                <footer class="bd-footer-article">
                  
<div class="footer-article-items footer-article__inner">
  
    <div class="footer-article-item"><!-- Previous / next buttons -->
<div class="prev-next-area">
    <a class="left-prev"
       href="unit2-chap2.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Chapter 2: Community Lifestyle of a Biofilm</p>
      </div>
    </a>
    <a class="right-next"
       href="unit2-chap4.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 4: Antimicrobials and Resistance</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div></div>
  
</div>

                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">

  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#introduction-to-pathogenicity">3.1 Introduction to Pathogenicity</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#pathogenicity-vs-virulence">3.1.1 Pathogenicity vs. Virulence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-infections">3.1.2 Bacterial Infections</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#infecting-the-host">3.2 Infecting the host</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#mode-of-infection">3.2.1 Mode of infection</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#kochs-postulates">3.2.2 Koch’s Postulates</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#regulation-of-genes-encoding-virulence-factors">3.2.3 Regulation of Genes Encoding Virulence Factors</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#virulence-factors-mechanisms-of-pathogenic-bacteria">3.3 Virulence Factors: Mechanisms of Pathogenic Bacteria</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#bacterial-adherence">3.3.1 Bacterial Adherence</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-neisseria-gonorrhoeae-a-pathogen-that-uses-adherence-and-antigenic-variation-to-enter-host-cells">Case Study: <em>Neisseria gonorrhoeae</em> – a pathogen that uses adherence and antigenic variation to enter host cells</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#capsules">3.3.2 Capsules</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-streptococcus-pneumoniae-a-pathogen-that-uses-a-capsule-to-avoid-host-defenses">Case Study: <em>Streptococcus pneumoniae</em> – A pathogen that uses a capsule to avoid host defenses</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#toxins">3.3.3 Toxins</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#case-study-vibrio-cholerae-a-pathogen-that-uses-an-enterotoxin-to-damage-its-host">Case Study: <em>Vibrio cholerae</em> – A pathogen that uses an enterotoxin to damage its host</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#references">References:</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By MICB 211 Teaching Team led by Evelyn Sun
</p>

  </div>
  
  <div class="footer-item">
    
  <p class="copyright">
    
      © Copyright 2022.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=e353d410970836974a52"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=e353d410970836974a52"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>